<html>

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=Generator content="Microsoft Word 15 (filtered)">
<title>AQUATOX DOT NET Users Manual</title>
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:Helvetica;
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:Courier;
	panose-1:2 7 4 9 2 2 5 2 4 4;}
@font-face
	{font-family:"Tms Rmn";
	panose-1:2 2 6 3 4 5 5 2 3 4;}
@font-face
	{font-family:Helv;
	panose-1:2 11 6 4 2 2 2 3 2 4;}
@font-face
	{font-family:"New York";
	panose-1:2 4 5 3 6 5 6 2 3 4;}
@font-face
	{font-family:System;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Wingdings;
	panose-1:5 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"MS Mincho";
	panose-1:2 2 6 9 4 2 5 8 3 4;}
@font-face
	{font-family:Batang;
	panose-1:2 3 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimSun;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:PMingLiU;
	panose-1:2 1 6 1 0 1 1 1 1 1;}
@font-face
	{font-family:"MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;}
@font-face
	{font-family:Dotum;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:SimHei;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:MingLiU;
	panose-1:2 1 6 9 0 1 1 1 1 1;}
@font-face
	{font-family:Mincho;
	panose-1:2 2 6 9 4 3 5 8 3 5;}
@font-face
	{font-family:Gulim;
	panose-1:2 11 6 0 0 1 1 1 1 1;}
@font-face
	{font-family:Century;
	panose-1:2 4 6 4 5 5 5 2 3 4;}
@font-face
	{font-family:"Angsana New";
	panose-1:2 2 6 3 5 4 5 2 3 4;}
@font-face
	{font-family:"Cordia New";
	panose-1:2 11 3 4 2 2 2 2 2 4;}
@font-face
	{font-family:Mangal;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Latha;
	panose-1:2 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sylfaen;
	panose-1:1 10 5 2 5 3 6 3 3 3;}
@font-face
	{font-family:Vrinda;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Raavi;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Shruti;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Sendnya;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:Gautami;
	panose-1:2 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:Tunga;
	panose-1:0 0 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Estrangelo Edessa";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Yu Gothic";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:DengXian;
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Calibri Light";
	panose-1:2 15 3 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Palatino Linotype";
	panose-1:2 4 5 2 5 5 5 3 3 4;}
@font-face
	{font-family:Verdana;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"Arial Unicode MS";
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:"Segoe UI Emoji";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"DengXian Light";}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:Cambria;
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:"Cascadia Mono";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:Marlett;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Arial Black";
	panose-1:2 11 10 4 2 1 2 2 2 4;}
@font-face
	{font-family:"Bahnschrift Light";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiLight";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:Bahnschrift;
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiBold";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift Light SemiCondensed";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiLight SemiConde";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiCondensed";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiBold SemiConden";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift Light Condensed";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiLight Condensed";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift Condensed";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Bahnschrift SemiBold Condensed";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:Candara;
	panose-1:2 14 5 2 3 3 3 2 2 4;}
@font-face
	{font-family:"Candara Light";
	panose-1:2 14 5 2 3 3 3 2 2 4;}
@font-face
	{font-family:"Comic Sans MS";
	panose-1:3 15 7 2 3 3 2 2 2 4;}
@font-face
	{font-family:Consolas;
	panose-1:2 11 6 9 2 2 4 3 2 4;}
@font-face
	{font-family:Constantia;
	panose-1:2 3 6 2 5 3 6 3 3 3;}
@font-face
	{font-family:Corbel;
	panose-1:2 11 5 3 2 2 4 2 2 4;}
@font-face
	{font-family:"Corbel Light";
	panose-1:2 11 3 3 2 2 4 2 2 4;}
@font-face
	{font-family:Ebrima;
	panose-1:2 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Franklin Gothic Medium";
	panose-1:2 11 6 3 2 1 2 2 2 4;}
@font-face
	{font-family:Gabriola;
	panose-1:4 4 6 5 5 16 2 2 13 2;}
@font-face
	{font-family:Gadugi;
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:Georgia;
	panose-1:2 4 5 2 5 4 5 2 3 3;}
@font-face
	{font-family:Impact;
	panose-1:2 11 8 6 3 9 2 5 2 4;}
@font-face
	{font-family:"Ink Free";
	panose-1:3 8 4 2 0 5 0 0 0 0;}
@font-face
	{font-family:"Javanese Text";
	panose-1:2 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Leelawadee UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Leelawadee UI Semilight";
	panose-1:2 11 4 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Lucida Console";
	panose-1:2 11 6 9 4 5 4 2 2 4;}
@font-face
	{font-family:"Lucida Sans Unicode";
	panose-1:2 11 6 2 3 5 4 2 2 4;}
@font-face
	{font-family:"Malgun Gothic";
	panose-1:2 11 5 3 2 0 0 2 0 4;}
@font-face
	{font-family:"\@Malgun Gothic";}
@font-face
	{font-family:"Malgun Gothic Semilight";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"\@Malgun Gothic Semilight";}
@font-face
	{font-family:"Microsoft Himalaya";
	panose-1:1 1 1 0 1 1 1 1 1 1;}
@font-face
	{font-family:"Microsoft JhengHei";
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"\@Microsoft JhengHei";}
@font-face
	{font-family:"Microsoft JhengHei UI";
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"\@Microsoft JhengHei UI";}
@font-face
	{font-family:"Microsoft JhengHei Light";
	panose-1:2 11 3 4 3 5 4 4 2 4;}
@font-face
	{font-family:"\@Microsoft JhengHei Light";}
@font-face
	{font-family:"Microsoft JhengHei UI Light";
	panose-1:2 11 3 4 3 5 4 4 2 4;}
@font-face
	{font-family:"\@Microsoft JhengHei UI Light";}
@font-face
	{font-family:"Microsoft New Tai Lue";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Microsoft PhagsPa";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Microsoft Sans Serif";
	panose-1:2 11 6 4 2 2 2 2 2 4;}
@font-face
	{font-family:"Microsoft Tai Le";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Microsoft YaHei";
	panose-1:2 11 5 3 2 2 4 2 2 4;}
@font-face
	{font-family:"\@Microsoft YaHei";}
@font-face
	{font-family:"Microsoft YaHei UI";
	panose-1:2 11 5 3 2 2 4 2 2 4;}
@font-face
	{font-family:"\@Microsoft YaHei UI";}
@font-face
	{font-family:"Microsoft YaHei Light";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"\@Microsoft YaHei Light";}
@font-face
	{font-family:"Microsoft YaHei UI Light";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"\@Microsoft YaHei UI Light";}
@font-face
	{font-family:"Microsoft Yi Baiti";
	panose-1:3 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:MingLiU-ExtB;
	panose-1:2 2 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@MingLiU-ExtB";}
@font-face
	{font-family:PMingLiU-ExtB;
	panose-1:2 2 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@PMingLiU-ExtB";}
@font-face
	{font-family:MingLiU_HKSCS-ExtB;
	panose-1:2 2 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@MingLiU_HKSCS-ExtB";}
@font-face
	{font-family:"Mongolian Baiti";
	panose-1:3 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@MS Gothic";
	panose-1:2 11 6 9 7 2 5 8 2 4;}
@font-face
	{font-family:"MS UI Gothic";
	panose-1:2 11 6 0 7 2 5 8 2 4;}
@font-face
	{font-family:"\@MS UI Gothic";}
@font-face
	{font-family:"MS PGothic";
	panose-1:2 11 6 0 7 2 5 8 2 4;}
@font-face
	{font-family:"\@MS PGothic";}
@font-face
	{font-family:"MV Boli";
	panose-1:2 0 5 0 3 2 0 9 0 0;}
@font-face
	{font-family:"Myanmar Text";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Nirmala UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Nirmala UI Semilight";
	panose-1:2 11 4 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe MDL2 Assets";
	panose-1:5 10 1 2 1 1 1 1 1 1;}
@font-face
	{font-family:"Segoe Print";
	panose-1:2 0 6 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Segoe Script";
	panose-1:3 11 5 4 2 0 0 0 0 3;}
@font-face
	{font-family:"Segoe UI";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI Black";
	panose-1:2 11 10 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI Historic";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI Light";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI Semibold";
	panose-1:2 11 7 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI Semilight";
	panose-1:2 11 4 2 4 2 4 2 2 3;}
@font-face
	{font-family:"Segoe UI Symbol";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
@font-face
	{font-family:"\@SimSun";
	panose-1:2 1 6 0 3 1 1 1 1 1;}
@font-face
	{font-family:NSimSun;
	panose-1:2 1 6 9 3 1 1 1 1 1;}
@font-face
	{font-family:"\@NSimSun";}
@font-face
	{font-family:SimSun-ExtB;
	panose-1:2 1 6 9 6 1 1 1 1 1;}
@font-face
	{font-family:"\@SimSun-ExtB";}
@font-face
	{font-family:"Sitka Small";
	panose-1:2 0 5 5 0 0 0 2 0 4;}
@font-face
	{font-family:"Sitka Text";
	panose-1:2 0 5 5 0 0 0 2 0 4;}
@font-face
	{font-family:"Sitka Subheading";
	panose-1:2 0 5 5 0 0 0 2 0 4;}
@font-face
	{font-family:"Sitka Heading";
	panose-1:2 0 5 5 0 0 0 2 0 4;}
@font-face
	{font-family:"Sitka Display";
	panose-1:2 0 5 5 0 0 0 2 0 4;}
@font-face
	{font-family:"Sitka Banner";
	panose-1:2 0 5 5 0 0 0 2 0 4;}
@font-face
	{font-family:"Trebuchet MS";
	panose-1:2 11 6 3 2 2 2 2 2 4;}
@font-face
	{font-family:Webdings;
	panose-1:5 3 1 2 1 5 9 6 7 3;}
@font-face
	{font-family:"\@Yu Gothic";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Yu Gothic UI";
	panose-1:2 11 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@Yu Gothic UI";}
@font-face
	{font-family:"Yu Gothic UI Semibold";
	panose-1:2 11 7 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@Yu Gothic UI Semibold";}
@font-face
	{font-family:"Yu Gothic Light";
	panose-1:2 11 3 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@Yu Gothic Light";}
@font-face
	{font-family:"Yu Gothic UI Light";
	panose-1:2 11 3 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@Yu Gothic UI Light";}
@font-face
	{font-family:"Yu Gothic Medium";
	panose-1:2 11 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@Yu Gothic Medium";}
@font-face
	{font-family:"Yu Gothic UI Semilight";
	panose-1:2 11 4 0 0 0 0 0 0 0;}
@font-face
	{font-family:"\@Yu Gothic UI Semilight";}
@font-face
	{font-family:"HoloLens MDL2 Assets";
	panose-1:5 10 1 2 1 1 1 1 1 1;}
@font-face
	{font-family:"Agency FB";
	panose-1:2 11 5 3 2 2 2 2 2 4;}
@font-face
	{font-family:Algerian;
	panose-1:4 2 7 5 4 10 2 6 7 2;}
@font-face
	{font-family:"Book Antiqua";
	panose-1:2 4 6 2 5 3 5 3 3 4;}
@font-face
	{font-family:"Arial Narrow";
	panose-1:2 11 6 6 2 2 2 3 2 4;}
@font-face
	{font-family:"Arial Rounded MT Bold";
	panose-1:2 15 7 4 3 5 4 3 2 4;}
@font-face
	{font-family:"Baskerville Old Face";
	panose-1:2 2 6 2 8 5 5 2 3 3;}
@font-face
	{font-family:"Bauhaus 93";
	panose-1:4 3 9 5 2 11 2 2 12 2;}
@font-face
	{font-family:"Bell MT";
	panose-1:2 2 5 3 6 3 5 2 3 3;}
@font-face
	{font-family:"Bernard MT Condensed";
	panose-1:2 5 8 6 6 9 5 2 4 4;}
@font-face
	{font-family:"Bodoni MT";
	panose-1:2 7 6 3 8 6 6 2 2 3;}
@font-face
	{font-family:"Bodoni MT Black";
	panose-1:2 7 10 3 8 6 6 2 2 3;}
@font-face
	{font-family:"Bodoni MT Condensed";
	panose-1:2 7 6 6 8 6 6 2 2 3;}
@font-face
	{font-family:"Bodoni MT Poster Compressed";
	panose-1:2 7 7 6 8 6 1 5 2 4;}
@font-face
	{font-family:"Bookman Old Style";
	panose-1:2 5 6 4 5 5 5 2 2 4;}
@font-face
	{font-family:"Bradley Hand ITC";
	panose-1:3 7 4 2 5 3 2 3 2 3;}
@font-face
	{font-family:"Britannic Bold";
	panose-1:2 11 9 3 6 7 3 2 2 4;}
@font-face
	{font-family:"Berlin Sans FB";
	panose-1:2 14 6 2 2 5 2 2 3 6;}
@font-face
	{font-family:"Berlin Sans FB Demi";
	panose-1:2 14 8 2 2 5 2 2 3 6;}
@font-face
	{font-family:Broadway;
	panose-1:4 4 9 5 8 11 2 2 5 2;}
@font-face
	{font-family:"Brush Script MT";
	panose-1:3 6 8 2 4 4 6 7 3 4;}
@font-face
	{font-family:"Bookshelf Symbol 7";
	panose-1:5 1 1 1 1 1 1 1 1 1;}
@font-face
	{font-family:"Californian FB";
	panose-1:2 7 4 3 6 8 11 3 2 4;}
@font-face
	{font-family:"Calisto MT";
	panose-1:2 4 6 3 5 5 5 3 3 4;}
@font-face
	{font-family:Castellar;
	panose-1:2 10 4 2 6 4 6 1 3 1;}
@font-face
	{font-family:"Century Schoolbook";
	panose-1:2 4 6 4 5 5 5 2 3 4;}
@font-face
	{font-family:Centaur;
	panose-1:2 3 5 4 5 2 5 2 3 4;}
@font-face
	{font-family:Chiller;
	panose-1:4 2 4 4 3 16 7 2 6 2;}
@font-face
	{font-family:"Colonna MT";
	panose-1:4 2 8 5 6 2 2 3 2 3;}
@font-face
	{font-family:"Cooper Black";
	panose-1:2 8 9 4 4 3 11 2 4 4;}
@font-face
	{font-family:"Copperplate Gothic Bold";
	panose-1:2 14 7 5 2 2 6 2 4 4;}
@font-face
	{font-family:"Copperplate Gothic Light";
	panose-1:2 14 5 7 2 2 6 2 4 4;}
@font-face
	{font-family:"Curlz MT";
	panose-1:4 4 4 4 5 7 2 2 2 2;}
@font-face
	{font-family:Dubai;
	panose-1:2 11 5 3 3 4 3 3 2 4;}
@font-face
	{font-family:"Dubai Light";
	panose-1:2 11 3 3 3 4 3 3 2 4;}
@font-face
	{font-family:"Dubai Medium";
	panose-1:2 11 6 3 3 4 3 3 2 4;}
@font-face
	{font-family:Elephant;
	panose-1:2 2 9 4 9 5 5 2 3 3;}
@font-face
	{font-family:"Engravers MT";
	panose-1:2 9 7 7 8 5 5 2 3 4;}
@font-face
	{font-family:"Eras Bold ITC";
	panose-1:2 11 9 7 3 5 4 2 2 4;}
@font-face
	{font-family:"Eras Demi ITC";
	panose-1:2 11 8 5 3 5 4 2 8 4;}
@font-face
	{font-family:"Eras Light ITC";
	panose-1:2 11 4 2 3 5 4 2 8 4;}
@font-face
	{font-family:"Eras Medium ITC";
	panose-1:2 11 6 2 3 5 4 2 8 4;}
@font-face
	{font-family:"Felix Titling";
	panose-1:4 6 5 5 6 2 2 2 10 4;}
@font-face
	{font-family:Forte;
	panose-1:3 6 9 2 4 5 2 7 2 3;}
@font-face
	{font-family:"Franklin Gothic Book";
	panose-1:2 11 5 3 2 1 2 2 2 4;}
@font-face
	{font-family:"Franklin Gothic Demi";
	panose-1:2 11 7 3 2 1 2 2 2 4;}
@font-face
	{font-family:"Franklin Gothic Demi Cond";
	panose-1:2 11 7 6 3 4 2 2 2 4;}
@font-face
	{font-family:"Franklin Gothic Heavy";
	panose-1:2 11 9 3 2 1 2 2 2 4;}
@font-face
	{font-family:"Franklin Gothic Medium Cond";
	panose-1:2 11 6 6 3 4 2 2 2 4;}
@font-face
	{font-family:"Freestyle Script";
	panose-1:3 8 4 2 3 2 5 11 4 4;}
@font-face
	{font-family:"French Script MT";
	panose-1:3 2 4 2 4 6 7 4 6 5;}
@font-face
	{font-family:"Footlight MT Light";
	panose-1:2 4 6 2 6 3 10 2 3 4;}
@font-face
	{font-family:Garamond;
	panose-1:2 2 4 4 3 3 1 1 8 3;}
@font-face
	{font-family:Gigi;
	panose-1:4 4 5 4 6 16 7 2 13 2;}
@font-face
	{font-family:"Gill Sans MT";
	panose-1:2 11 5 2 2 1 4 2 2 3;}
@font-face
	{font-family:"Gill Sans MT Condensed";
	panose-1:2 11 5 6 2 1 4 2 2 3;}
@font-face
	{font-family:"Gill Sans Ultra Bold Condensed";
	panose-1:2 11 10 6 2 1 4 2 2 3;}
@font-face
	{font-family:"Gill Sans Ultra Bold";
	panose-1:2 11 10 2 2 1 4 2 2 3;}
@font-face
	{font-family:"Gloucester MT Extra Condensed";
	panose-1:2 3 8 8 2 6 1 1 1 1;}
@font-face
	{font-family:"Gill Sans MT Ext Condensed Bold";
	panose-1:2 11 9 2 2 1 4 2 2 3;}
@font-face
	{font-family:"Century Gothic";
	panose-1:2 11 5 2 2 2 2 2 2 4;}
@font-face
	{font-family:"Goudy Old Style";
	panose-1:2 2 5 2 5 3 5 2 3 3;}
@font-face
	{font-family:"Goudy Stout";
	panose-1:2 2 9 4 7 3 11 2 4 1;}
@font-face
	{font-family:"Harlow Solid Italic";
	panose-1:4 3 6 4 2 15 2 2 13 2;}
@font-face
	{font-family:Harrington;
	panose-1:4 4 5 5 5 10 2 2 7 2;}
@font-face
	{font-family:Haettenschweiler;
	panose-1:2 11 7 6 4 9 2 6 2 4;}
@font-face
	{font-family:"High Tower Text";
	panose-1:2 4 5 2 5 5 6 3 3 3;}
@font-face
	{font-family:"Imprint MT Shadow";
	panose-1:4 2 6 5 6 3 3 3 2 2;}
@font-face
	{font-family:"Informal Roman";
	panose-1:3 6 4 2 3 4 6 11 2 4;}
@font-face
	{font-family:"Blackadder ITC";
	panose-1:4 2 5 5 5 16 7 2 13 2;}
@font-face
	{font-family:"Edwardian Script ITC";
	panose-1:3 3 3 2 4 7 7 13 8 4;}
@font-face
	{font-family:"Kristen ITC";
	panose-1:3 5 5 2 4 2 2 3 2 2;}
@font-face
	{font-family:Jokerman;
	panose-1:4 9 6 5 6 13 6 2 7 2;}
@font-face
	{font-family:"Juice ITC";
	panose-1:4 4 4 3 4 10 2 2 2 2;}
@font-face
	{font-family:"Kunstler Script";
	panose-1:3 3 4 2 2 6 7 13 13 6;}
@font-face
	{font-family:"Wide Latin";
	panose-1:2 10 10 7 5 5 5 2 4 4;}
@font-face
	{font-family:"Lucida Bright";
	panose-1:2 4 6 2 5 5 5 2 3 4;}
@font-face
	{font-family:"Lucida Calligraphy";
	panose-1:3 1 1 1 1 1 1 1 1 1;}
@font-face
	{font-family:"Lucida Fax";
	panose-1:2 6 6 2 5 5 5 2 2 4;}
@font-face
	{font-family:"Lucida Handwriting";
	panose-1:3 1 1 1 1 1 1 1 1 1;}
@font-face
	{font-family:"Lucida Sans";
	panose-1:2 11 6 2 3 5 4 2 2 4;}
@font-face
	{font-family:"Lucida Sans Typewriter";
	panose-1:2 11 5 9 3 5 4 3 2 4;}
@font-face
	{font-family:Magneto;
	panose-1:4 3 8 5 5 8 2 2 13 2;}
@font-face
	{font-family:"Maiandra GD";
	panose-1:2 14 5 2 3 3 8 2 2 4;}
@font-face
	{font-family:"Matura MT Script Capitals";
	panose-1:3 2 8 2 6 6 2 7 2 2;}
@font-face
	{font-family:Mistral;
	panose-1:3 9 7 2 3 4 7 2 4 3;}
@font-face
	{font-family:"Modern No\. 20";
	panose-1:2 7 7 4 7 5 5 2 3 3;}
@font-face
	{font-family:"Monotype Corsiva";
	panose-1:3 1 1 1 1 2 1 1 1 1;}
@font-face
	{font-family:"MT Extra";
	panose-1:5 5 1 2 1 2 5 2 2 2;}
@font-face
	{font-family:"Niagara Engraved";
	panose-1:4 2 5 2 7 7 3 3 2 2;}
@font-face
	{font-family:"Niagara Solid";
	panose-1:4 2 5 2 7 7 2 2 2 2;}
@font-face
	{font-family:"OCR A Extended";
	panose-1:2 1 5 9 2 1 2 1 3 3;}
@font-face
	{font-family:"Old English Text MT";
	panose-1:3 4 9 2 4 5 8 3 8 6;}
@font-face
	{font-family:Onyx;
	panose-1:4 5 6 2 8 7 2 2 2 3;}
@font-face
	{font-family:"MS Outlook";
	panose-1:5 1 1 0 1 0 0 0 0 0;}
@font-face
	{font-family:"Palace Script MT";
	panose-1:3 3 3 2 2 6 7 12 11 5;}
@font-face
	{font-family:Papyrus;
	panose-1:3 7 5 2 6 5 2 3 2 5;}
@font-face
	{font-family:Parchment;
	panose-1:3 4 6 2 4 7 8 4 8 4;}
@font-face
	{font-family:Perpetua;
	panose-1:2 2 5 2 6 4 1 2 3 3;}
@font-face
	{font-family:"Perpetua Titling MT";
	panose-1:2 2 5 2 6 5 5 2 8 4;}
@font-face
	{font-family:Playbill;
	panose-1:4 5 6 3 10 6 2 2 2 2;}
@font-face
	{font-family:"Poor Richard";
	panose-1:2 8 5 2 5 5 5 2 7 2;}
@font-face
	{font-family:Pristina;
	panose-1:3 6 4 2 4 4 6 8 2 4;}
@font-face
	{font-family:"Rage Italic";
	panose-1:3 7 5 2 4 5 7 7 3 4;}
@font-face
	{font-family:Ravie;
	panose-1:4 4 8 5 5 8 9 2 6 2;}
@font-face
	{font-family:"MS Reference Sans Serif";
	panose-1:2 11 6 4 3 5 4 4 2 4;}
@font-face
	{font-family:"MS Reference Specialty";
	panose-1:5 0 5 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Rockwell Condensed";
	panose-1:2 6 6 3 5 4 5 2 1 4;}
@font-face
	{font-family:Rockwell;
	panose-1:2 6 6 3 2 2 5 2 4 3;}
@font-face
	{font-family:"Rockwell Extra Bold";
	panose-1:2 6 9 3 4 5 5 2 4 3;}
@font-face
	{font-family:"Script MT Bold";
	panose-1:3 4 6 2 4 6 7 8 9 4;}
@font-face
	{font-family:"Showcard Gothic";
	panose-1:4 2 9 4 2 1 2 2 6 4;}
@font-face
	{font-family:"Snap ITC";
	panose-1:4 4 10 7 6 10 2 2 2 2;}
@font-face
	{font-family:Stencil;
	panose-1:4 4 9 5 13 8 2 2 4 4;}
@font-face
	{font-family:"Tw Cen MT";
	panose-1:2 11 6 2 2 1 4 2 6 3;}
@font-face
	{font-family:"Tw Cen MT Condensed";
	panose-1:2 11 6 6 2 1 4 2 2 3;}
@font-face
	{font-family:"Tw Cen MT Condensed Extra Bold";
	panose-1:2 11 8 3 2 2 2 2 2 4;}
@font-face
	{font-family:"Tempus Sans ITC";
	panose-1:4 2 4 4 3 13 7 2 2 2;}
@font-face
	{font-family:"Viner Hand ITC";
	panose-1:3 7 5 2 3 5 2 2 2 3;}
@font-face
	{font-family:Vivaldi;
	panose-1:3 2 6 2 5 5 6 9 8 4;}
@font-face
	{font-family:"Vladimir Script";
	panose-1:3 5 4 2 4 4 7 7 3 5;}
@font-face
	{font-family:"Wingdings 2";
	panose-1:5 2 1 2 1 5 7 7 7 7;}
@font-face
	{font-family:"Wingdings 3";
	panose-1:5 4 1 2 1 8 7 7 7 7;}
@font-face
	{font-family:QuickType;
	panose-1:2 11 6 3 2 0 4 2 2 3;}
@font-face
	{font-family:"QuickType Pi";
	panose-1:2 11 6 0 2 0 0 0 0 1;}
@font-face
	{font-family:"QuickType Condensed";
	panose-1:2 11 5 6 3 4 3 2 2 3;}
@font-face
	{font-family:"QuickType Mono";
	panose-1:2 11 5 9 2 1 4 2 2 3;}
@font-face
	{font-family:"OCR-A II";
	panose-1:2 15 6 9 0 1 4 6 3 7;}
@font-face
	{font-family:"OCR B MT";
	panose-1:2 11 5 9 0 1 2 2 2 3;}
@font-face
	{font-family:"QuickType II Condensed";
	panose-1:2 11 5 6 3 4 3 2 2 3;}
@font-face
	{font-family:"QuickType II Mono";
	panose-1:2 11 5 9 2 1 4 2 2 3;}
@font-face
	{font-family:"QuickType II Pi";
	panose-1:2 11 6 0 2 0 0 0 0 1;}
@font-face
	{font-family:"QuickType II";
	panose-1:2 11 6 3 2 0 4 2 2 3;}
@font-face
	{font-family:"Roboto Bk";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Roboto;
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Roboto Cn";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Roboto Lt";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Roboto Th";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:Lato;
	panose-1:2 15 5 2 2 2 4 3 2 3;}
@font-face
	{font-family:"Lato Light";
	panose-1:2 15 5 2 2 2 4 3 2 3;}
@font-face
	{font-family:"Lato Semibold";
	panose-1:2 15 5 2 2 2 4 3 2 3;}
@font-face
	{font-family:"Cascadia Code ExtraLight";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Code Light";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Code SemiLight";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Code";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Code SemiBold";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Mono ExtraLight";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Mono Light";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Mono SemiLight";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:"Cascadia Mono SemiBold";
	panose-1:2 11 6 9 2 0 0 2 0 4;}
@font-face
	{font-family:Abadi;}
@font-face
	{font-family:"Abadi Extra Light";}
@font-face
	{font-family:Abel;}
@font-face
	{font-family:"Abril Fatface";}
@font-face
	{font-family:"ADLaM Display";}
@font-face
	{font-family:Aharoni;}
@font-face
	{font-family:"Al Fresco";}
@font-face
	{font-family:"Alasassy Caps";}
@font-face
	{font-family:Aldhabi;}
@font-face
	{font-family:Alef;}
@font-face
	{font-family:Aleo;}
@font-face
	{font-family:"Aleo Light";}
@font-face
	{font-family:"Amasis MT Pro";}
@font-face
	{font-family:"Amasis MT Pro Black";}
@font-face
	{font-family:"Amasis MT Pro Light";}
@font-face
	{font-family:"Amasis MT Pro Medium";}
@font-face
	{font-family:"Amatic SC";}
@font-face
	{font-family:AngsanaUPC;}
@font-face
	{font-family:Anton;}
@font-face
	{font-family:Aparajita;}
@font-face
	{font-family:Aptos;}
@font-face
	{font-family:"Aptos Black";}
@font-face
	{font-family:"Aptos Display";}
@font-face
	{font-family:"Aptos ExtraBold";}
@font-face
	{font-family:"Aptos Light";}
@font-face
	{font-family:"Aptos Mono";}
@font-face
	{font-family:"Aptos Narrow";}
@font-face
	{font-family:"Aptos SemiBold";}
@font-face
	{font-family:"Aptos Serif";}
@font-face
	{font-family:"Arabic Typesetting";}
@font-face
	{font-family:"Aref Ruqaa";}
@font-face
	{font-family:"Arial Nova";}
@font-face
	{font-family:"Arial Nova Cond";}
@font-face
	{font-family:"Arial Nova Cond Light";}
@font-face
	{font-family:"Arial Nova Light";}
@font-face
	{font-family:Assistant;}
@font-face
	{font-family:"Assistant ExtraBold";}
@font-face
	{font-family:"Assistant ExtraLight";}
@font-face
	{font-family:"Assistant Light";}
@font-face
	{font-family:"Assistant SemiBold";}
@font-face
	{font-family:Athiti;}
@font-face
	{font-family:"Athiti ExtraLight";}
@font-face
	{font-family:"Athiti Light";}
@font-face
	{font-family:"Athiti Medium";}
@font-face
	{font-family:"Athiti SemiBold";}
@font-face
	{font-family:"Avenir Next LT Pro";}
@font-face
	{font-family:"Avenir Next LT Pro Demi";}
@font-face
	{font-family:"Avenir Next LT Pro Light";}
@font-face
	{font-family:"Baguet Script";}
@font-face
	{font-family:Barlow;}
@font-face
	{font-family:"Barlow Black";}
@font-face
	{font-family:"Barlow Condensed";}
@font-face
	{font-family:"Barlow Condensed Black";}
@font-face
	{font-family:"Barlow Condensed ExtraBold";}
@font-face
	{font-family:"Barlow Condensed ExtraLight";}
@font-face
	{font-family:"Barlow Condensed Light";}
@font-face
	{font-family:"Barlow Condensed Medium";}
@font-face
	{font-family:"Barlow Condensed SemiBold";}
@font-face
	{font-family:"Barlow Condensed Thin";}
@font-face
	{font-family:"Barlow ExtraBold";}
@font-face
	{font-family:"Barlow ExtraLight";}
@font-face
	{font-family:"Barlow Light";}
@font-face
	{font-family:"Barlow Medium";}
@font-face
	{font-family:"Barlow Semi Condensed";}
@font-face
	{font-family:"Barlow Semi Condensed Black";}
@font-face
	{font-family:"Barlow Semi Condensed ExLight";}
@font-face
	{font-family:"Barlow Semi Condensed ExtraBold";}
@font-face
	{font-family:"Barlow Semi Condensed Light";}
@font-face
	{font-family:"Barlow Semi Condensed Medium";}
@font-face
	{font-family:"Barlow Semi Condensed SemiBold";}
@font-face
	{font-family:"Barlow Semi Condensed Thin";}
@font-face
	{font-family:"Barlow SemiBold";}
@font-face
	{font-family:"Barlow Thin";}
@font-face
	{font-family:BatangChe;}
@font-face
	{font-family:"Bebas Neue";}
@font-face
	{font-family:Bembo;}
@font-face
	{font-family:Bierstadt;}
@font-face
	{font-family:"Bierstadt Display";}
@font-face
	{font-family:BierstadtAlt;}
@font-face
	{font-family:"BierstadtAlt2 Cond";}
@font-face
	{font-family:"BierstadtAlt3 Cond";}
@font-face
	{font-family:"BierstadtAlt4 Cond";}
@font-face
	{font-family:BierstadtAlt5;}
@font-face
	{font-family:Biome;}
@font-face
	{font-family:"Biome Light";}
@font-face
	{font-family:"Boucherie Block";}
@font-face
	{font-family:"Boucherie Sans";}
@font-face
	{font-family:"Browallia New";}
@font-face
	{font-family:BrowalliaUPC;}
@font-face
	{font-family:"Buxton Sketch";}
@font-face
	{font-family:"Caveat Brush";}
@font-face
	{font-family:Cavolini;}
@font-face
	{font-family:"Chamberi Super Display";}
@font-face
	{font-family:Charmonman;}
@font-face
	{font-family:"Chilgok Gwon Anja";}
@font-face
	{font-family:"Chilgok Kim Yeongbun";}
@font-face
	{font-family:"Chilgok Lee Jonghui";}
@font-face
	{font-family:"Chilgok Lee Wonsun";}
@font-face
	{font-family:Chonburi;}
@font-face
	{font-family:"Cochocib Script Latin Pro";}
@font-face
	{font-family:"Concert One";}
@font-face
	{font-family:Cond;}
@font-face
	{font-family:Congenial;}
@font-face
	{font-family:"Congenial Black";}
@font-face
	{font-family:"Congenial Light";}
@font-face
	{font-family:"Congenial SemiBold";}
@font-face
	{font-family:"Congenial UltraLight";}
@font-face
	{font-family:Convection;}
@font-face
	{font-family:"Convection Condensed";}
@font-face
	{font-family:"Convection Extra Bold";}
@font-face
	{font-family:"Convection Symbol";}
@font-face
	{font-family:"Convection UI";}
@font-face
	{font-family:CordiaUPC;}
@font-face
	{font-family:Dante;}
@font-face
	{font-family:DaunPenh;}
@font-face
	{font-family:David;}
@font-face
	{font-family:Daytona;}
@font-face
	{font-family:"Daytona Condensed";}
@font-face
	{font-family:"Daytona Condensed Light";}
@font-face
	{font-family:"Daytona Light";}
@font-face
	{font-family:"Didact Gothic";}
@font-face
	{font-family:DilleniaUPC;}
@font-face
	{font-family:"DM Mono";}
@font-face
	{font-family:"DM Mono Light";}
@font-face
	{font-family:"DM Mono Medium";}
@font-face
	{font-family:"DM Sans";}
@font-face
	{font-family:"DM Sans Medium";}
@font-face
	{font-family:"DM Serif Display";}
@font-face
	{font-family:"DM Serif Text";}
@font-face
	{font-family:DokChampa;}
@font-face
	{font-family:Dosis;}
@font-face
	{font-family:"Dosis ExtraBold";}
@font-face
	{font-family:"Dosis ExtraLight";}
@font-face
	{font-family:"Dosis Light";}
@font-face
	{font-family:"Dosis Medium";}
@font-face
	{font-family:"Dosis SemiBold";}
@font-face
	{font-family:DotumChe;}
@font-face
	{font-family:"Dreaming Outloud Pro";}
@font-face
	{font-family:"Dreaming Outloud Script Pro";}
@font-face
	{font-family:"EB Garamond";}
@font-face
	{font-family:"EB Garamond ExtraBold";}
@font-face
	{font-family:"EB Garamond Medium";}
@font-face
	{font-family:"EB Garamond SemiBold";}
@font-face
	{font-family:"Elephant Pro";}
@font-face
	{font-family:EucrosiaUPC;}
@font-face
	{font-family:Euphemia;}
@font-face
	{font-family:Fahkwang;}
@font-face
	{font-family:"Fahkwang ExtraLight";}
@font-face
	{font-family:"Fahkwang Light";}
@font-face
	{font-family:"Fahkwang Medium";}
@font-face
	{font-family:"Fahkwang SemiBold";}
@font-face
	{font-family:"Fairwater Script";}
@font-face
	{font-family:"Fairwater Script Light";}
@font-face
	{font-family:FangSong;}
@font-face
	{font-family:"Fave Script Bold Pro";}
@font-face
	{font-family:"Fira Code";}
@font-face
	{font-family:"Fira Code Light";}
@font-face
	{font-family:"Fira Code Medium";}
@font-face
	{font-family:"Fira Code Retina";}
@font-face
	{font-family:"Fira Mono";}
@font-face
	{font-family:"Fira Mono Medium";}
@font-face
	{font-family:"Fira Sans";}
@font-face
	{font-family:"Fira Sans Black";}
@font-face
	{font-family:"Fira Sans Condensed";}
@font-face
	{font-family:"Fira Sans Condensed Black";}
@font-face
	{font-family:"Fira Sans Condensed ExtraBold";}
@font-face
	{font-family:"Fira Sans Condensed ExtraLight";}
@font-face
	{font-family:"Fira Sans Condensed Light";}
@font-face
	{font-family:"Fira Sans Condensed Medium";}
@font-face
	{font-family:"Fira Sans Condensed SemiBold";}
@font-face
	{font-family:"Fira Sans Condensed Thin";}
@font-face
	{font-family:"Fira Sans Extra Condensed";}
@font-face
	{font-family:"Fira Sans Extra Condensed Black";}
@font-face
	{font-family:"Fira Sans Extra Condensed Light";}
@font-face
	{font-family:"Fira Sans Extra Condensed Thin";}
@font-face
	{font-family:"Fira Sans ExtraBold";}
@font-face
	{font-family:"Fira Sans ExtraLight";}
@font-face
	{font-family:"Fira Sans Light";}
@font-face
	{font-family:"Fira Sans Medium";}
@font-face
	{font-family:"Fira Sans SemiBold";}
@font-face
	{font-family:"Fira Sans Thin";}
@font-face
	{font-family:"Fjalla One";}
@font-face
	{font-family:"Forte Forward";}
@font-face
	{font-family:"Frank Ruhl Libre";}
@font-face
	{font-family:"Frank Ruhl Libre Black";}
@font-face
	{font-family:"Frank Ruhl Libre Light";}
@font-face
	{font-family:"Frank Ruhl Libre Medium";}
@font-face
	{font-family:FrankRuehl;}
@font-face
	{font-family:"Fredoka One";}
@font-face
	{font-family:FreesiaUPC;}
@font-face
	{font-family:Gabriela;}
@font-face
	{font-family:Gaegu;}
@font-face
	{font-family:"Gaegu Light";}
@font-face
	{font-family:"Georgia Pro";}
@font-face
	{font-family:"Georgia Pro Black";}
@font-face
	{font-family:"Georgia Pro Cond";}
@font-face
	{font-family:"Georgia Pro Cond Black";}
@font-face
	{font-family:"Georgia Pro Cond Light";}
@font-face
	{font-family:"Georgia Pro Cond Semibold";}
@font-face
	{font-family:"Georgia Pro Light";}
@font-face
	{font-family:"Georgia Pro Semibold";}
@font-face
	{font-family:"Gill Sans Nova";}
@font-face
	{font-family:"Gill Sans Nova Cond";}
@font-face
	{font-family:"Gill Sans Nova Cond Lt";}
@font-face
	{font-family:"Gill Sans Nova Cond Ultra Bold";}
@font-face
	{font-family:"Gill Sans Nova Cond XBd";}
@font-face
	{font-family:"Gill Sans Nova Light";}
@font-face
	{font-family:"Gill Sans Nova Ultra Bold";}
@font-face
	{font-family:Gisha;}
@font-face
	{font-family:"Goudy Type";}
@font-face
	{font-family:Grandview;}
@font-face
	{font-family:"Grandview Display";}
@font-face
	{font-family:Grotesque;}
@font-face
	{font-family:"Grotesque Light";}
@font-face
	{font-family:GulimChe;}
@font-face
	{font-family:Gungsuh;}
@font-face
	{font-family:GungsuhChe;}
@font-face
	{font-family:"Hadassah Friedlaender";}
@font-face
	{font-family:"Hammersmith One";}
@font-face
	{font-family:Heebo;}
@font-face
	{font-family:"Heebo Black";}
@font-face
	{font-family:"Heebo ExtraBold";}
@font-face
	{font-family:"Heebo Light";}
@font-face
	{font-family:"Heebo Medium";}
@font-face
	{font-family:"Heebo Thin";}
@font-face
	{font-family:HGGothicE;}
@font-face
	{font-family:HGMaruGothicMPRO;}
@font-face
	{font-family:HGMinchoE;}
@font-face
	{font-family:HGPGothicE;}
@font-face
	{font-family:HGPMinchoE;}
@font-face
	{font-family:HGPSoeiKakugothicUB;}
@font-face
	{font-family:HGSGothicE;}
@font-face
	{font-family:HGSMinchoE;}
@font-face
	{font-family:HGSoeiKakugothicUB;}
@font-face
	{font-family:HGSSoeiKakugothicUB;}
@font-face
	{font-family:Hind;}
@font-face
	{font-family:"Hind Colombo";}
@font-face
	{font-family:"Hind Colombo Light";}
@font-face
	{font-family:"Hind Colombo Medium";}
@font-face
	{font-family:"Hind Colombo SemiBold";}
@font-face
	{font-family:"Hind Guntur";}
@font-face
	{font-family:"Hind Guntur Light";}
@font-face
	{font-family:"Hind Guntur Medium";}
@font-face
	{font-family:"Hind Guntur SemiBold";}
@font-face
	{font-family:"Hind Jalandhar";}
@font-face
	{font-family:"Hind Jalandhar Light";}
@font-face
	{font-family:"Hind Jalandhar Medium";}
@font-face
	{font-family:"Hind Jalandhar SemiBold";}
@font-face
	{font-family:"Hind Kochi";}
@font-face
	{font-family:"Hind Kochi Light";}
@font-face
	{font-family:"Hind Kochi Medium";}
@font-face
	{font-family:"Hind Kochi SemiBold";}
@font-face
	{font-family:"Hind Light";}
@font-face
	{font-family:"Hind Madurai";}
@font-face
	{font-family:"Hind Madurai Light";}
@font-face
	{font-family:"Hind Madurai Medium";}
@font-face
	{font-family:"Hind Madurai SemiBold";}
@font-face
	{font-family:"Hind Medium";}
@font-face
	{font-family:"Hind Mysuru";}
@font-face
	{font-family:"Hind Mysuru Light";}
@font-face
	{font-family:"Hind Mysuru Medium";}
@font-face
	{font-family:"Hind Mysuru SemiBold";}
@font-face
	{font-family:"Hind SemiBold";}
@font-face
	{font-family:"Hind Siliguri";}
@font-face
	{font-family:"Hind Siliguri Light";}
@font-face
	{font-family:"Hind Siliguri Medium";}
@font-face
	{font-family:"Hind Siliguri SemiBold";}
@font-face
	{font-family:"Hind Vadodara";}
@font-face
	{font-family:"Hind Vadodara Light";}
@font-face
	{font-family:"Hind Vadodara Medium";}
@font-face
	{font-family:"Hind Vadodara SemiBold";}
@font-face
	{font-family:"IBM Plex Mono";}
@font-face
	{font-family:"IBM Plex Mono ExtraLight";}
@font-face
	{font-family:"IBM Plex Mono Light";}
@font-face
	{font-family:"IBM Plex Mono Medium";}
@font-face
	{font-family:"IBM Plex Mono SemiBold";}
@font-face
	{font-family:"IBM Plex Mono Thin";}
@font-face
	{font-family:"IBM Plex Sans";}
@font-face
	{font-family:"IBM Plex Sans Condensed";}
@font-face
	{font-family:"IBM Plex Sans Condensed Light";}
@font-face
	{font-family:"IBM Plex Sans Condensed Medium";}
@font-face
	{font-family:"IBM Plex Sans Condensed Thin";}
@font-face
	{font-family:"IBM Plex Sans ExtraLight";}
@font-face
	{font-family:"IBM Plex Sans Light";}
@font-face
	{font-family:"IBM Plex Sans Medium";}
@font-face
	{font-family:"IBM Plex Sans SemiBold";}
@font-face
	{font-family:"IBM Plex Sans Thin";}
@font-face
	{font-family:"IBM Plex Serif";}
@font-face
	{font-family:"IBM Plex Serif ExtraLight";}
@font-face
	{font-family:"IBM Plex Serif Light";}
@font-face
	{font-family:"IBM Plex Serif Medium";}
@font-face
	{font-family:"IBM Plex Serif SemiBold";}
@font-face
	{font-family:"IBM Plex Serif Thin";}
@font-face
	{font-family:Inconsolata;}
@font-face
	{font-family:"Inconsolata Black";}
@font-face
	{font-family:"Inconsolata Bold";}
@font-face
	{font-family:"Inconsolata Condensed";}
@font-face
	{font-family:"Inconsolata Condensed Black";}
@font-face
	{font-family:"Inconsolata Condensed Bold";}
@font-face
	{font-family:"Inconsolata Condensed ExtraBold";}
@font-face
	{font-family:"Inconsolata Condensed Light";}
@font-face
	{font-family:"Inconsolata Condensed Medium";}
@font-face
	{font-family:"Inconsolata Condensed SemiBold";}
@font-face
	{font-family:"Inconsolata Expanded";}
@font-face
	{font-family:"Inconsolata Expanded Black";}
@font-face
	{font-family:"Inconsolata Expanded Bold";}
@font-face
	{font-family:"Inconsolata Expanded ExtraBold";}
@font-face
	{font-family:"Inconsolata Expanded ExtraLight";}
@font-face
	{font-family:"Inconsolata Expanded Light";}
@font-face
	{font-family:"Inconsolata Expanded Medium";}
@font-face
	{font-family:"Inconsolata Expanded SemiBold";}
@font-face
	{font-family:"Inconsolata ExtraBold";}
@font-face
	{font-family:"Inconsolata ExtraCondensed";}
@font-face
	{font-family:"Inconsolata ExtraCondensed Bold";}
@font-face
	{font-family:"Inconsolata ExtraExpanded";}
@font-face
	{font-family:"Inconsolata ExtraExpanded Black";}
@font-face
	{font-family:"Inconsolata ExtraExpanded Bold";}
@font-face
	{font-family:"Inconsolata ExtraExpanded Light";}
@font-face
	{font-family:"Inconsolata ExtraLight";}
@font-face
	{font-family:"Inconsolata Light";}
@font-face
	{font-family:"Inconsolata Medium";}
@font-face
	{font-family:"Inconsolata SemiBold";}
@font-face
	{font-family:"Inconsolata SemiCondensed";}
@font-face
	{font-family:"Inconsolata SemiCondensed Black";}
@font-face
	{font-family:"Inconsolata SemiCondensed Bold";}
@font-face
	{font-family:"Inconsolata SemiCondensed Light";}
@font-face
	{font-family:"Inconsolata SemiExpanded";}
@font-face
	{font-family:"Inconsolata SemiExpanded Black";}
@font-face
	{font-family:"Inconsolata SemiExpanded Bold";}
@font-face
	{font-family:"Inconsolata SemiExpanded Light";}
@font-face
	{font-family:"Inconsolata SemiExpanded Medium";}
@font-face
	{font-family:"Inconsolata UltraCondensed";}
@font-face
	{font-family:"Inconsolata UltraCondensed Bold";}
@font-face
	{font-family:"Inconsolata UltraExpanded";}
@font-face
	{font-family:"Inconsolata UltraExpanded Black";}
@font-face
	{font-family:"Inconsolata UltraExpanded Bold";}
@font-face
	{font-family:"Inconsolata UltraExpanded Light";}
@font-face
	{font-family:IrisUPC;}
@font-face
	{font-family:"Iskoola Pota";}
@font-face
	{font-family:Italianno;}
@font-face
	{font-family:JasmineUPC;}
@font-face
	{font-family:"Josefin Sans";}
@font-face
	{font-family:"Josefin Sans Bold";}
@font-face
	{font-family:"Josefin Sans Light";}
@font-face
	{font-family:"Josefin Sans SemiBold";}
@font-face
	{font-family:"Josefin Sans Thin";}
@font-face
	{font-family:"Josefin Slab";}
@font-face
	{font-family:"Josefin Slab Light";}
@font-face
	{font-family:"Josefin Slab SemiBold";}
@font-face
	{font-family:"Josefin Slab Thin";}
@font-face
	{font-family:Jumble;}
@font-face
	{font-family:KaiTi;}
@font-face
	{font-family:Kalinga;}
@font-face
	{font-family:Karla;}
@font-face
	{font-family:"Karla ExtraBold";}
@font-face
	{font-family:"Karla ExtraLight";}
@font-face
	{font-family:"Karla Light";}
@font-face
	{font-family:"Karla Medium";}
@font-face
	{font-family:"Karla Tamil Inclined";}
@font-face
	{font-family:"Karla Tamil Upright";}
@font-face
	{font-family:Kartika;}
@font-face
	{font-family:Kermit;}
@font-face
	{font-family:"Kermit Condensed";}
@font-face
	{font-family:"Kermit Expanded";}
@font-face
	{font-family:"Kermit Extrabold";}
@font-face
	{font-family:"Kermit Extrabold Condensed";}
@font-face
	{font-family:"Kermit Extrabold Expanded";}
@font-face
	{font-family:"Kermit Extralight";}
@font-face
	{font-family:"Kermit Extralight Condensed";}
@font-face
	{font-family:"Kermit Extralight Expanded";}
@font-face
	{font-family:"Kermit Light";}
@font-face
	{font-family:"Kermit Light Condensed";}
@font-face
	{font-family:"Kermit Light Expanded";}
@font-face
	{font-family:"Kermit Semibold";}
@font-face
	{font-family:"Kermit Semibold Condensed";}
@font-face
	{font-family:"Kermit Semibold Expanded";}
@font-face
	{font-family:"Kermit Thin";}
@font-face
	{font-family:"Kermit Thin Condensed";}
@font-face
	{font-family:"Kermit Thin Expanded";}
@font-face
	{font-family:"Khmer UI";}
@font-face
	{font-family:Kigelia;}
@font-face
	{font-family:"Kigelia Arabic";}
@font-face
	{font-family:"Kigelia Arabic Light";}
@font-face
	{font-family:"Kigelia Light";}
@font-face
	{font-family:"Klee One";}
@font-face
	{font-family:"Klee One SemiBold";}
@font-face
	{font-family:KodchiangUPC;}
@font-face
	{font-family:Kokila;}
@font-face
	{font-family:Krub;}
@font-face
	{font-family:"Krub ExtraLight";}
@font-face
	{font-family:"Krub Light";}
@font-face
	{font-family:"Krub Medium";}
@font-face
	{font-family:"Krub SemiBold";}
@font-face
	{font-family:Lalezar;}
@font-face
	{font-family:"Lao UI";}
@font-face
	{font-family:"Lato Black";}
@font-face
	{font-family:"Lato ExtraBold";}
@font-face
	{font-family:"Lato ExtraLight";}
@font-face
	{font-family:"Lato Medium";}
@font-face
	{font-family:"Lato Thin";}
@font-face
	{font-family:Leelawadee;}
@font-face
	{font-family:"Levenim MT";}
@font-face
	{font-family:"Libre Barcode 128";}
@font-face
	{font-family:"Libre Barcode 128 Text";}
@font-face
	{font-family:"Libre Barcode 39";}
@font-face
	{font-family:"Libre Barcode 39 Extended";}
@font-face
	{font-family:"Libre Barcode 39 Extended Text";}
@font-face
	{font-family:"Libre Barcode 39 Text";}
@font-face
	{font-family:"Libre Barcode EAN13 Text";}
@font-face
	{font-family:"Libre Baskerville";}
@font-face
	{font-family:"Libre Franklin";}
@font-face
	{font-family:"Libre Franklin Black";}
@font-face
	{font-family:"Libre Franklin ExtraBold";}
@font-face
	{font-family:"Libre Franklin ExtraLight";}
@font-face
	{font-family:"Libre Franklin Light";}
@font-face
	{font-family:"Libre Franklin Medium";}
@font-face
	{font-family:"Libre Franklin SemiBold";}
@font-face
	{font-family:"Libre Franklin Thin";}
@font-face
	{font-family:Ligconsolata;}
@font-face
	{font-family:LilyUPC;}
@font-face
	{font-family:Livvic;}
@font-face
	{font-family:"Livvic Black";}
@font-face
	{font-family:"Livvic ExtraLight";}
@font-face
	{font-family:"Livvic Light";}
@font-face
	{font-family:"Livvic Medium";}
@font-face
	{font-family:"Livvic SemiBold";}
@font-face
	{font-family:"Livvic Thin";}
@font-face
	{font-family:Lobster;}
@font-face
	{font-family:"Lobster Two";}
@font-face
	{font-family:Lora;}
@font-face
	{font-family:"Lora Medium";}
@font-face
	{font-family:"Mangal Pro";}
@font-face
	{font-family:Meddon;}
@font-face
	{font-family:Meiryo;}
@font-face
	{font-family:"Meiryo UI";}
@font-face
	{font-family:Merriweather;}
@font-face
	{font-family:"Merriweather Black";}
@font-face
	{font-family:"Merriweather Light";}
@font-face
	{font-family:"Merriweather Sans";}
@font-face
	{font-family:"Merriweather Sans ExtraBold";}
@font-face
	{font-family:"Merriweather Sans Light";}
@font-face
	{font-family:"Microsoft GothicNeo";}
@font-face
	{font-family:"Microsoft GothicNeo Light";}
@font-face
	{font-family:"Microsoft Uighur";}
@font-face
	{font-family:MingLiU_HKSCS;}
@font-face
	{font-family:Miriam;}
@font-face
	{font-family:"Miriam Fixed";}
@font-face
	{font-family:"Miriam Libre";}
@font-face
	{font-family:Mitr;}
@font-face
	{font-family:"Mitr ExtraLight";}
@font-face
	{font-family:"Mitr Light";}
@font-face
	{font-family:"Mitr Medium";}
@font-face
	{font-family:"Mitr SemiBold";}
@font-face
	{font-family:"Modern Love";}
@font-face
	{font-family:"Modern Love Caps";}
@font-face
	{font-family:"Modern Love Grunge";}
@font-face
	{font-family:Montserrat;}
@font-face
	{font-family:"Montserrat Black";}
@font-face
	{font-family:"Montserrat ExtraBold";}
@font-face
	{font-family:"Montserrat ExtraLight";}
@font-face
	{font-family:"Montserrat Light";}
@font-face
	{font-family:"Montserrat Medium";}
@font-face
	{font-family:"Montserrat SemiBold";}
@font-face
	{font-family:"Montserrat Thin";}
@font-face
	{font-family:MoolBoran;}
@font-face
	{font-family:"Mr Gabe";}
@font-face
	{font-family:"MS PMincho";}
@font-face
	{font-family:"Mystical Woods Rough Script";}
@font-face
	{font-family:"Mystical Woods Smooth Script";}
@font-face
	{font-family:"Nanum Brush Script";}
@font-face
	{font-family:"Nanum Pen";}
@font-face
	{font-family:NanumGothic;}
@font-face
	{font-family:NanumGothicCoding;}
@font-face
	{font-family:NanumGothicExtraBold;}
@font-face
	{font-family:NanumMyeongjo;}
@font-face
	{font-family:NanumMyeongjoExtraBold;}
@font-face
	{font-family:Narkisim;}
@font-face
	{font-family:"Neue Haas Grotesk Text Pro";}
@font-face
	{font-family:"News Gothic MT";}
@font-face
	{font-family:Nina;}
@font-face
	{font-family:"Nirmala Text";}
@font-face
	{font-family:"Nirmala Text Semilight";}
@font-face
	{font-family:"Nordique Inline";}
@font-face
	{font-family:"Noto Music";}
@font-face
	{font-family:"Noto Nastaliq Urdu";}
@font-face
	{font-family:"Noto Sans";}
@font-face
	{font-family:"Noto Sans AnatoHiero";}
@font-face
	{font-family:"Noto Sans Avestan";}
@font-face
	{font-family:"Noto Sans Bassa Vah";}
@font-face
	{font-family:"Noto Sans Batak";}
@font-face
	{font-family:"Noto Sans Bhaiksuki";}
@font-face
	{font-family:"Noto Sans Brahmi";}
@font-face
	{font-family:"Noto Sans Buginese";}
@font-face
	{font-family:"Noto Sans Buhid";}
@font-face
	{font-family:"Noto Sans Carian";}
@font-face
	{font-family:"Noto Sans CaucAlban";}
@font-face
	{font-family:"Noto Sans Chakma";}
@font-face
	{font-family:"Noto Sans Coptic";}
@font-face
	{font-family:"Noto Sans Cuneiform";}
@font-face
	{font-family:"Noto Sans Cypriot";}
@font-face
	{font-family:"Noto Sans Deseret";}
@font-face
	{font-family:"Noto Sans Devanagari";}
@font-face
	{font-family:"Noto Sans Devanagari Black";}
@font-face
	{font-family:"Noto Sans Devanagari ExtraBold";}
@font-face
	{font-family:"Noto Sans Devanagari ExtraLight";}
@font-face
	{font-family:"Noto Sans Devanagari Light";}
@font-face
	{font-family:"Noto Sans Devanagari Medium";}
@font-face
	{font-family:"Noto Sans Devanagari SemiBold";}
@font-face
	{font-family:"Noto Sans Devanagari Thin";}
@font-face
	{font-family:"Noto Sans Devanagari UI";}
@font-face
	{font-family:"Noto Sans Devanagari UI Black";}
@font-face
	{font-family:"Noto Sans Devanagari UI Light";}
@font-face
	{font-family:"Noto Sans Devanagari UI Medium";}
@font-face
	{font-family:"Noto Sans Devanagari UI Thin";}
@font-face
	{font-family:"Noto Sans Duployan";}
@font-face
	{font-family:"Noto Sans EgyptHiero";}
@font-face
	{font-family:"Noto Sans Elbasan";}
@font-face
	{font-family:"Noto Sans Elymaic";}
@font-face
	{font-family:"Noto Sans Glagolitic";}
@font-face
	{font-family:"Noto Sans Gothic";}
@font-face
	{font-family:"Noto Sans Grantha";}
@font-face
	{font-family:"Noto Sans Gujarati";}
@font-face
	{font-family:"Noto Sans Gujarati Black";}
@font-face
	{font-family:"Noto Sans Gujarati ExtraBold";}
@font-face
	{font-family:"Noto Sans Gujarati ExtraLight";}
@font-face
	{font-family:"Noto Sans Gujarati Light";}
@font-face
	{font-family:"Noto Sans Gujarati Medium";}
@font-face
	{font-family:"Noto Sans Gujarati SemiBold";}
@font-face
	{font-family:"Noto Sans Gujarati Thin";}
@font-face
	{font-family:"Noto Sans Gujarati UI";}
@font-face
	{font-family:"Noto Sans Gujarati UI Black";}
@font-face
	{font-family:"Noto Sans Gujarati UI ExtraBold";}
@font-face
	{font-family:"Noto Sans Gujarati UI Light";}
@font-face
	{font-family:"Noto Sans Gujarati UI Medium";}
@font-face
	{font-family:"Noto Sans Gujarati UI SemiBold";}
@font-face
	{font-family:"Noto Sans Gujarati UI Thin";}
@font-face
	{font-family:"Noto Sans Gunjala Gondi";}
@font-face
	{font-family:"Noto Sans Hanunoo";}
@font-face
	{font-family:"Noto Sans Hatran";}
@font-face
	{font-family:"Noto Sans ImpAramaic";}
@font-face
	{font-family:"Noto Sans Indic Siyaq Numbers";}
@font-face
	{font-family:"Noto Sans InsPahlavi";}
@font-face
	{font-family:"Noto Sans InsParthi";}
@font-face
	{font-family:"Noto Sans Javanese";}
@font-face
	{font-family:"Noto Sans Kaithi";}
@font-face
	{font-family:"Noto Sans Kharoshthi";}
@font-face
	{font-family:"Noto Sans Khojki";}
@font-face
	{font-family:"Noto Sans Khudawadi";}
@font-face
	{font-family:"Noto Sans Lepcha";}
@font-face
	{font-family:"Noto Sans Limbu";}
@font-face
	{font-family:"Noto Sans Linear A";}
@font-face
	{font-family:"Noto Sans Linear B";}
@font-face
	{font-family:"Noto Sans Lycian";}
@font-face
	{font-family:"Noto Sans Lydian";}
@font-face
	{font-family:"Noto Sans Mahajani";}
@font-face
	{font-family:"Noto Sans Mandaic";}
@font-face
	{font-family:"Noto Sans Manichaean";}
@font-face
	{font-family:"Noto Sans Marchen";}
@font-face
	{font-family:"Noto Sans Masaram Gondi";}
@font-face
	{font-family:"Noto Sans Math";}
@font-face
	{font-family:"Noto Sans Mayan Numerals";}
@font-face
	{font-family:"Noto Sans Mende Kikakui";}
@font-face
	{font-family:"Noto Sans Meroitic";}
@font-face
	{font-family:"Noto Sans Miao";}
@font-face
	{font-family:"Noto Sans Modi";}
@font-face
	{font-family:"Noto Sans Mongolian";}
@font-face
	{font-family:"Noto Sans Mro";}
@font-face
	{font-family:"Noto Sans Multani";}
@font-face
	{font-family:"Noto Sans Myanmar";}
@font-face
	{font-family:"Noto Sans Myanmar Blk";}
@font-face
	{font-family:"Noto Sans Myanmar ExtBd";}
@font-face
	{font-family:"Noto Sans Myanmar ExtLt";}
@font-face
	{font-family:"Noto Sans Myanmar Light";}
@font-face
	{font-family:"Noto Sans Myanmar Med";}
@font-face
	{font-family:"Noto Sans Myanmar SemBd";}
@font-face
	{font-family:"Noto Sans Myanmar Thin";}
@font-face
	{font-family:"Noto Sans Myanmar UI";}
@font-face
	{font-family:"Noto Sans Myanmar UI Black";}
@font-face
	{font-family:"Noto Sans Myanmar UI ExtraBold";}
@font-face
	{font-family:"Noto Sans Myanmar UI ExtraLight";}
@font-face
	{font-family:"Noto Sans Myanmar UI Light";}
@font-face
	{font-family:"Noto Sans Myanmar UI Medium";}
@font-face
	{font-family:"Noto Sans Myanmar UI SemiBold";}
@font-face
	{font-family:"Noto Sans Myanmar UI Thin";}
@font-face
	{font-family:"Noto Sans Nabataean";}
@font-face
	{font-family:"Noto Sans Newa";}
@font-face
	{font-family:"Noto Sans NewTaiLue";}
@font-face
	{font-family:"Noto Sans NKo";}
@font-face
	{font-family:"Noto Sans Nushu";}
@font-face
	{font-family:"Noto Sans Ogham";}
@font-face
	{font-family:"Noto Sans Old Italic";}
@font-face
	{font-family:"Noto Sans Old Permic";}
@font-face
	{font-family:"Noto Sans Old Turkic";}
@font-face
	{font-family:"Noto Sans OldHung";}
@font-face
	{font-family:"Noto Sans OldNorArab";}
@font-face
	{font-family:"Noto Sans OldPersian";}
@font-face
	{font-family:"Noto Sans OldSogdian";}
@font-face
	{font-family:"Noto Sans OldSouArab";}
@font-face
	{font-family:"Noto Sans Oriya";}
@font-face
	{font-family:"Noto Sans Oriya Blk";}
@font-face
	{font-family:"Noto Sans Oriya Thin";}
@font-face
	{font-family:"Noto Sans Oriya UI";}
@font-face
	{font-family:"Noto Sans Oriya UI Blk";}
@font-face
	{font-family:"Noto Sans Oriya UI Thin";}
@font-face
	{font-family:"Noto Sans Osage";}
@font-face
	{font-family:"Noto Sans Osmanya";}
@font-face
	{font-family:"Noto Sans Pahawh Hmong";}
@font-face
	{font-family:"Noto Sans Palmyrene";}
@font-face
	{font-family:"Noto Sans PauCinHau";}
@font-face
	{font-family:"Noto Sans PhagsPa";}
@font-face
	{font-family:"Noto Sans Phoenician";}
@font-face
	{font-family:"Noto Sans PsaPahlavi";}
@font-face
	{font-family:"Noto Sans Rejang";}
@font-face
	{font-family:"Noto Sans Runic";}
@font-face
	{font-family:"Noto Sans Samaritan";}
@font-face
	{font-family:"Noto Sans Saurashtra";}
@font-face
	{font-family:"Noto Sans Sharada";}
@font-face
	{font-family:"Noto Sans Shavian";}
@font-face
	{font-family:"Noto Sans Siddham";}
@font-face
	{font-family:"Noto Sans Sogdian";}
@font-face
	{font-family:"Noto Sans Soyombo";}
@font-face
	{font-family:"Noto Sans Syloti Nagri";}
@font-face
	{font-family:"Noto Sans Symbols2";}
@font-face
	{font-family:"Noto Sans Syriac";}
@font-face
	{font-family:"Noto Sans Syriac Black";}
@font-face
	{font-family:"Noto Sans Syriac Thin";}
@font-face
	{font-family:"Noto Sans Tagalog";}
@font-face
	{font-family:"Noto Sans Tagbanwa";}
@font-face
	{font-family:"Noto Sans Tai Le";}
@font-face
	{font-family:"Noto Sans Tai Viet";}
@font-face
	{font-family:"Noto Sans Takri";}
@font-face
	{font-family:"Noto Sans Tamil Supplement";}
@font-face
	{font-family:"Noto Sans Thai Looped Black";}
@font-face
	{font-family:"Noto Sans Thai Looped Bold";}
@font-face
	{font-family:"Noto Sans Thai Looped ExtLight";}
@font-face
	{font-family:"Noto Sans Thai Looped Extrabold";}
@font-face
	{font-family:"Noto Sans Thai Looped Light";}
@font-face
	{font-family:"Noto Sans Thai Looped Medium";}
@font-face
	{font-family:"Noto Sans Thai Looped Regular";}
@font-face
	{font-family:"Noto Sans Thai Looped Semibold";}
@font-face
	{font-family:"Noto Sans Thai Looped Thin";}
@font-face
	{font-family:"Noto Sans Tifinagh";}
@font-face
	{font-family:"Noto Sans Tirhuta";}
@font-face
	{font-family:"Noto Sans Ugaritic";}
@font-face
	{font-family:"Noto Sans Vai";}
@font-face
	{font-family:"Noto Sans Wancho";}
@font-face
	{font-family:"Noto Sans WarangCiti";}
@font-face
	{font-family:"Noto Sans Yi";}
@font-face
	{font-family:"Noto Sans Zanabazar";}
@font-face
	{font-family:"Noto Serif";}
@font-face
	{font-family:"Noto Serif Ahom";}
@font-face
	{font-family:"Noto Serif Balinese";}
@font-face
	{font-family:"Noto Serif Dogra";}
@font-face
	{font-family:"Noto Serif Grantha";}
@font-face
	{font-family:"Noto Serif Myanmar";}
@font-face
	{font-family:"Noto Serif Myanmar Blk";}
@font-face
	{font-family:"Noto Serif Myanmar ExtBd";}
@font-face
	{font-family:"Noto Serif Myanmar ExtLt";}
@font-face
	{font-family:"Noto Serif Myanmar Light";}
@font-face
	{font-family:"Noto Serif Myanmar Med";}
@font-face
	{font-family:"Noto Serif Myanmar SemBd";}
@font-face
	{font-family:"Noto Serif Myanmar Thin";}
@font-face
	{font-family:"Noto Serif Tangut";}
@font-face
	{font-family:"Noto Traditional Nushu";}
@font-face
	{font-family:Nunito;}
@font-face
	{font-family:"Nunito Black";}
@font-face
	{font-family:"Nunito ExtraBold";}
@font-face
	{font-family:"Nunito ExtraLight";}
@font-face
	{font-family:"Nunito Light";}
@font-face
	{font-family:"Nunito Sans";}
@font-face
	{font-family:"Nunito Sans Black";}
@font-face
	{font-family:"Nunito Sans ExtraBold";}
@font-face
	{font-family:"Nunito Sans ExtraLight";}
@font-face
	{font-family:"Nunito Sans Light";}
@font-face
	{font-family:"Nunito Sans SemiBold";}
@font-face
	{font-family:"Nunito SemiBold";}
@font-face
	{font-family:Nyala;}
@font-face
	{font-family:OCRB;}
@font-face
	{font-family:"Open Sans";}
@font-face
	{font-family:"Open Sans ExtraBold";}
@font-face
	{font-family:"Open Sans Light";}
@font-face
	{font-family:"Open Sans SemiBold";}
@font-face
	{font-family:Oranienbaum;}
@font-face
	{font-family:Oswald;}
@font-face
	{font-family:"Oswald ExtraLight";}
@font-face
	{font-family:"Oswald Light";}
@font-face
	{font-family:"Oswald Medium";}
@font-face
	{font-family:"Oswald SemiBold";}
@font-face
	{font-family:Oxygen;}
@font-face
	{font-family:"Oxygen Light";}
@font-face
	{font-family:"Oxygen Mono";}
@font-face
	{font-family:Pacifico;}
@font-face
	{font-family:Palanquin;}
@font-face
	{font-family:"Palanquin ExtraLight";}
@font-face
	{font-family:"Palanquin Light";}
@font-face
	{font-family:"Palanquin Medium";}
@font-face
	{font-family:"Palanquin SemiBold";}
@font-face
	{font-family:"Palanquin Thin";}
@font-face
	{font-family:"Patrick Hand";}
@font-face
	{font-family:"Patrick Hand SC";}
@font-face
	{font-family:"Petit Formal Script";}
@font-face
	{font-family:"Plantagenet Cherokee";}
@font-face
	{font-family:"Playfair Display";}
@font-face
	{font-family:"Playfair Display Black";}
@font-face
	{font-family:"Playfair Display SC";}
@font-face
	{font-family:"Playfair Display SC Black";}
@font-face
	{font-family:"Poiret One";}
@font-face
	{font-family:Poppins;}
@font-face
	{font-family:"Poppins Black";}
@font-face
	{font-family:"Poppins ExtraBold";}
@font-face
	{font-family:"Poppins ExtraLight";}
@font-face
	{font-family:"Poppins Light";}
@font-face
	{font-family:"Poppins Medium";}
@font-face
	{font-family:"Poppins SemiBold";}
@font-face
	{font-family:"Poppins Thin";}
@font-face
	{font-family:Posterama;}
@font-face
	{font-family:Pridi;}
@font-face
	{font-family:"Pridi ExtraLight";}
@font-face
	{font-family:"Pridi Light";}
@font-face
	{font-family:"Pridi Medium";}
@font-face
	{font-family:"Pridi SemiBold";}
@font-face
	{font-family:Prompt;}
@font-face
	{font-family:"Prompt Black";}
@font-face
	{font-family:"Prompt ExtraBold";}
@font-face
	{font-family:"Prompt ExtraLight";}
@font-face
	{font-family:"Prompt Light";}
@font-face
	{font-family:"Prompt Medium";}
@font-face
	{font-family:"Prompt SemiBold";}
@font-face
	{font-family:"Prompt Thin";}
@font-face
	{font-family:"PT Mono";}
@font-face
	{font-family:"PT Sans";}
@font-face
	{font-family:"PT Sans Caption";}
@font-face
	{font-family:"PT Sans Narrow";}
@font-face
	{font-family:"PT Serif";}
@font-face
	{font-family:"PT Serif Caption";}
@font-face
	{font-family:Quattrocento;}
@font-face
	{font-family:"Quattrocento Sans";}
@font-face
	{font-family:Questrial;}
@font-face
	{font-family:"Quire Sans";}
@font-face
	{font-family:"Quire Sans Light";}
@font-face
	{font-family:"Quire Sans Pro Light";}
@font-face
	{font-family:Raleway;}
@font-face
	{font-family:"Raleway Black";}
@font-face
	{font-family:"Raleway ExtraBold";}
@font-face
	{font-family:"Raleway ExtraLight";}
@font-face
	{font-family:"Raleway Light";}
@font-face
	{font-family:"Raleway Medium";}
@font-face
	{font-family:"Raleway SemiBold";}
@font-face
	{font-family:"Raleway Thin";}
@font-face
	{font-family:"Rastanty Cortez";}
@font-face
	{font-family:"Reem Kufi";}
@font-face
	{font-family:"Roboto Black";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Roboto Condensed";}
@font-face
	{font-family:"Roboto Condensed Light";}
@font-face
	{font-family:"Roboto Condensed Medium";}
@font-face
	{font-family:"Roboto Light";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Roboto Medium";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Roboto Mono";}
@font-face
	{font-family:"Roboto Mono Light";}
@font-face
	{font-family:"Roboto Mono Medium";}
@font-face
	{font-family:"Roboto Mono Thin";}
@font-face
	{font-family:"Roboto Serif 20pt";}
@font-face
	{font-family:"Roboto Serif 20pt Black";}
@font-face
	{font-family:"Roboto Serif 20pt ExtraBold";}
@font-face
	{font-family:"Roboto Serif 20pt ExtraLight";}
@font-face
	{font-family:"Roboto Serif 20pt Light";}
@font-face
	{font-family:"Roboto Serif 20pt Medium";}
@font-face
	{font-family:"Roboto Serif 20pt SemiBold";}
@font-face
	{font-family:"Roboto Serif 20pt Thin";}
@font-face
	{font-family:"Roboto Slab";}
@font-face
	{font-family:"Roboto Slab Black";}
@font-face
	{font-family:"Roboto Slab ExtraBold";}
@font-face
	{font-family:"Roboto Slab ExtraLight";}
@font-face
	{font-family:"Roboto Slab Light";}
@font-face
	{font-family:"Roboto Slab Medium";}
@font-face
	{font-family:"Roboto Slab SemiBold";}
@font-face
	{font-family:"Roboto Slab Thin";}
@font-face
	{font-family:"Roboto Thin";
	panose-1:0 0 0 0 0 0 0 0 0 0;}
@font-face
	{font-family:"Rockwell Light";}
@font-face
	{font-family:"Rockwell Nova";}
@font-face
	{font-family:"Rockwell Nova Cond";}
@font-face
	{font-family:"Rockwell Nova Cond Light";}
@font-face
	{font-family:"Rockwell Nova Extra Bold";}
@font-face
	{font-family:"Rockwell Nova Light";}
@font-face
	{font-family:Rod;}
@font-face
	{font-family:"Sabon Next LT";}
@font-face
	{font-family:Sacramento;}
@font-face
	{font-family:Sagona;}
@font-face
	{font-family:"Sagona Book";}
@font-face
	{font-family:"Sagona ExtraLight";}
@font-face
	{font-family:"Sakkal Majalla";}
@font-face
	{font-family:"Sans Serif Collection";}
@font-face
	{font-family:"Sanskrit Text";}
@font-face
	{font-family:Seaford;}
@font-face
	{font-family:"Seaford Display";}
@font-face
	{font-family:"Secular One";}
@font-face
	{font-family:"Segoe Marker";}
@font-face
	{font-family:"Segoe Pro";}
@font-face
	{font-family:"Segoe Pro Black";}
@font-face
	{font-family:"Segoe Pro Cond";}
@font-face
	{font-family:"Segoe Pro Display";}
@font-face
	{font-family:"Segoe Pro Display Light";}
@font-face
	{font-family:"Segoe Pro Display Semibold";}
@font-face
	{font-family:"Segoe Pro Display SemiLight";}
@font-face
	{font-family:"Segoe Pro Light";}
@font-face
	{font-family:"Segoe Pro Semibold";}
@font-face
	{font-family:"Segoe Pro SemiLight";}
@font-face
	{font-family:"Segoe Sans Display";}
@font-face
	{font-family:"Segoe Sans Display Black";}
@font-face
	{font-family:"Segoe Sans Display ExtraBold";}
@font-face
	{font-family:"Segoe Sans Display Hairline";}
@font-face
	{font-family:"Segoe Sans Display Light";}
@font-face
	{font-family:"Segoe Sans Display Semibold";}
@font-face
	{font-family:"Segoe Sans Display Semilight";}
@font-face
	{font-family:"Segoe Sans Display Thin";}
@font-face
	{font-family:"Segoe Sans Small";}
@font-face
	{font-family:"Segoe Sans Small Black";}
@font-face
	{font-family:"Segoe Sans Small ExtraBold";}
@font-face
	{font-family:"Segoe Sans Small Hairline";}
@font-face
	{font-family:"Segoe Sans Small Light";}
@font-face
	{font-family:"Segoe Sans Small Semibold";}
@font-face
	{font-family:"Segoe Sans Small Semilight";}
@font-face
	{font-family:"Segoe Sans Small Thin";}
@font-face
	{font-family:"Segoe Sans Text";}
@font-face
	{font-family:"Segoe Sans Text Black";}
@font-face
	{font-family:"Segoe Sans Text ExtraBold";}
@font-face
	{font-family:"Segoe Sans Text Hairline";}
@font-face
	{font-family:"Segoe Sans Text Light";}
@font-face
	{font-family:"Segoe Sans Text Semibold";}
@font-face
	{font-family:"Segoe Sans Text Semilight";}
@font-face
	{font-family:"Segoe Sans Text Thin";}
@font-face
	{font-family:"Segoe Serif Banner";}
@font-face
	{font-family:"Segoe Serif Banner Light";}
@font-face
	{font-family:"Segoe Serif Banner Semibold";}
@font-face
	{font-family:"Segoe Serif Banner Semilight";}
@font-face
	{font-family:"Segoe Serif Display";}
@font-face
	{font-family:"Segoe Serif Display Light";}
@font-face
	{font-family:"Segoe Serif Display Semibold";}
@font-face
	{font-family:"Segoe Serif Display Semilight";}
@font-face
	{font-family:"Segoe Serif Text";}
@font-face
	{font-family:"Segoe Serif Text Light";}
@font-face
	{font-family:"Segoe Serif Text Semibold";}
@font-face
	{font-family:"Segoe Serif Text Semilight";}
@font-face
	{font-family:"Segoe UI Emoji L";}
@font-face
	{font-family:"Segoe UI Variable Display";}
@font-face
	{font-family:"Segoe UI Variable Display Light";}
@font-face
	{font-family:"Segoe UI Variable Small";}
@font-face
	{font-family:"Segoe UI Variable Small Light";}
@font-face
	{font-family:"Segoe UI Variable Text";}
@font-face
	{font-family:"Segoe UI Variable Text Light";}
@font-face
	{font-family:"Segoe Xbox Symbol";}
@font-face
	{font-family:Selawik;}
@font-face
	{font-family:"Selawik Light";}
@font-face
	{font-family:"Selawik Semibold";}
@font-face
	{font-family:"Shadows Into Light Two";}
@font-face
	{font-family:"Shonar Bangla";}
@font-face
	{font-family:"Simplified Arabic";}
@font-face
	{font-family:"Simplified Arabic Fixed";}
@font-face
	{font-family:"Sitka Banner Semibold";}
@font-face
	{font-family:"Sitka Display Semibold";}
@font-face
	{font-family:"Sitka Heading Semibold";}
@font-face
	{font-family:"Sitka Small Semibold";}
@font-face
	{font-family:"Sitka Subheading Semibold";}
@font-face
	{font-family:"Sitka Text Semibold";}
@font-face
	{font-family:Skeena;}
@font-face
	{font-family:"Skeena Display";}
@font-face
	{font-family:"Source Code Pro";}
@font-face
	{font-family:"Source Code Pro Black";}
@font-face
	{font-family:"Source Code Pro ExtraLight";}
@font-face
	{font-family:"Source Code Pro Light";}
@font-face
	{font-family:"Source Code Pro Medium";}
@font-face
	{font-family:"Source Code Pro Semibold";}
@font-face
	{font-family:"Source Sans Pro";}
@font-face
	{font-family:"Source Sans Pro Black";}
@font-face
	{font-family:"Source Sans Pro ExtraLight";}
@font-face
	{font-family:"Source Sans Pro Light";}
@font-face
	{font-family:"Source Sans Pro SemiBold";}
@font-face
	{font-family:"Source Serif Pro";}
@font-face
	{font-family:"Source Serif Pro Black";}
@font-face
	{font-family:"Source Serif Pro ExtraLight";}
@font-face
	{font-family:"Source Serif Pro Light";}
@font-face
	{font-family:"Source Serif Pro SemiBold";}
@font-face
	{font-family:"Speak Pro";}
@font-face
	{font-family:"Speak Pro Light";}
@font-face
	{font-family:Staatliches;}
@font-face
	{font-family:STCaiyun;}
@font-face
	{font-family:STFangsong;}
@font-face
	{font-family:STHupo;}
@font-face
	{font-family:STKaiti;}
@font-face
	{font-family:STXihei;}
@font-face
	{font-family:STXingkai;}
@font-face
	{font-family:STXinwei;}
@font-face
	{font-family:STZhongsong;}
@font-face
	{font-family:"Suez One";}
@font-face
	{font-family:Tenorite;}
@font-face
	{font-family:"Tenorite Display";}
@font-face
	{font-family:"TH SarabunPSK";}
@font-face
	{font-family:"The Hand";}
@font-face
	{font-family:"The Hand Black";}
@font-face
	{font-family:"The Hand Extrablack";}
@font-face
	{font-family:"The Hand Light";}
@font-face
	{font-family:"The Serif Hand";}
@font-face
	{font-family:"The Serif Hand Black";}
@font-face
	{font-family:"The Serif Hand Extrablack";}
@font-face
	{font-family:"The Serif Hand Light";}
@font-face
	{font-family:"Tisa Offc Serif Pro";}
@font-face
	{font-family:"Tisa Offc Serif Pro Thin";}
@font-face
	{font-family:"Titillium Web";}
@font-face
	{font-family:"Titillium Web Black";}
@font-face
	{font-family:"Titillium Web ExtraLight";}
@font-face
	{font-family:"Titillium Web Light";}
@font-face
	{font-family:"Titillium Web SemiBold";}
@font-face
	{font-family:"Trade Gothic Inline";}
@font-face
	{font-family:"Trade Gothic Next";}
@font-face
	{font-family:"Trade Gothic Next Cond";}
@font-face
	{font-family:"Trade Gothic Next Heavy";}
@font-face
	{font-family:"Trade Gothic Next HvyCd";}
@font-face
	{font-family:"Trade Gothic Next Light";}
@font-face
	{font-family:"Trade Gothic Next Rounded";}
@font-face
	{font-family:"Traditional Arabic";}
@font-face
	{font-family:Trirong;}
@font-face
	{font-family:"Trirong Black";}
@font-face
	{font-family:"Trirong ExtraBold";}
@font-face
	{font-family:"Trirong ExtraLight";}
@font-face
	{font-family:"Trirong Light";}
@font-face
	{font-family:"Trirong Medium";}
@font-face
	{font-family:"Trirong SemiBold";}
@font-face
	{font-family:"Trirong Thin";}
@font-face
	{font-family:Ubuntu;}
@font-face
	{font-family:"Ubuntu Condensed";}
@font-face
	{font-family:"Ubuntu Light";}
@font-face
	{font-family:"Ubuntu Medium";}
@font-face
	{font-family:"Ubuntu Mono";}
@font-face
	{font-family:"UD Digi Kyokasho N-B";}
@font-face
	{font-family:"UD Digi Kyokasho N-R";}
@font-face
	{font-family:"UD Digi Kyokasho NK-B";}
@font-face
	{font-family:"UD Digi Kyokasho NK-R";}
@font-face
	{font-family:"UD Digi Kyokasho NP-B";}
@font-face
	{font-family:"UD Digi Kyokasho NP-R";}
@font-face
	{font-family:Univers;}
@font-face
	{font-family:"Univers Condensed";}
@font-face
	{font-family:"Univers Condensed Light";}
@font-face
	{font-family:"Univers Light";}
@font-face
	{font-family:"Urdu Typesetting";}
@font-face
	{font-family:Utsaah;}
@font-face
	{font-family:Vani;}
@font-face
	{font-family:"Varela Round";}
@font-face
	{font-family:"Verdana Pro";}
@font-face
	{font-family:"Verdana Pro Black";}
@font-face
	{font-family:"Verdana Pro Cond";}
@font-face
	{font-family:"Verdana Pro Cond Black";}
@font-face
	{font-family:"Verdana Pro Cond Light";}
@font-face
	{font-family:"Verdana Pro Cond Semibold";}
@font-face
	{font-family:"Verdana Pro Light";}
@font-face
	{font-family:"Verdana Pro Semibold";}
@font-face
	{font-family:Vijaya;}
@font-face
	{font-family:"Walbaum Display";}
@font-face
	{font-family:"Walbaum Display Heavy";}
@font-face
	{font-family:"Walbaum Display Light";}
@font-face
	{font-family:"Walbaum Display SemiBold";}
@font-face
	{font-family:"Walbaum Heading";}
@font-face
	{font-family:"Walbaum Text";}
@font-face
	{font-family:Wandohope;}
@font-face
	{font-family:"Work Sans";}
@font-face
	{font-family:"Work Sans Black";}
@font-face
	{font-family:"Work Sans ExtraBold";}
@font-face
	{font-family:"Work Sans ExtraLight";}
@font-face
	{font-family:"Work Sans Light";}
@font-face
	{font-family:"Work Sans Medium";}
@font-face
	{font-family:"Work Sans SemiBold";}
@font-face
	{font-family:"Work Sans Thin";}
@font-face
	{font-family:Yesteryear;}
@font-face
	{font-family:"Yu Mincho";}
@font-face
	{font-family:"Yu Mincho Demibold";}
@font-face
	{font-family:"Yu Mincho Light";}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
h1
	{mso-style-link:"Heading 1 Char";
	margin-top:12.0pt;
	margin-right:0in;
	margin-bottom:3.0pt;
	margin-left:0in;
	page-break-after:avoid;
	font-size:16.0pt;
	font-family:"Cambria",serif;
	font-weight:bold;}
h2
	{margin-right:0in;
	margin-left:0in;
	font-size:18.0pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
h3
	{margin-right:0in;
	margin-left:0in;
	font-size:13.5pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
h4
	{margin-right:0in;
	margin-left:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
h5
	{margin-right:0in;
	margin-left:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
p.MsoToc1, li.MsoToc1, div.MsoToc1
	{margin:0in;
	margin-bottom:3.0pt;
	font-size:7.0pt;
	font-family:"Times New Roman",serif;}
p.MsoToc2, li.MsoToc2, div.MsoToc2
	{margin-top:6.0pt;
	margin-right:0in;
	margin-bottom:3.0pt;
	margin-left:12.25pt;
	font-size:11.0pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
p.MsoToc3, li.MsoToc3, div.MsoToc3
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:3.0pt;
	margin-left:20.15pt;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoToc4, li.MsoToc4, div.MsoToc4
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:3.0pt;
	margin-left:30.25pt;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoToc5, li.MsoToc5, div.MsoToc5
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:44.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc6, li.MsoToc6, div.MsoToc6
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:55.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc7, li.MsoToc7, div.MsoToc7
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:66.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc8, li.MsoToc8, div.MsoToc8
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:77.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoToc9, li.MsoToc9, div.MsoToc9
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:5.0pt;
	margin-left:88.0pt;
	line-height:115%;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
p.MsoFootnoteText, li.MsoFootnoteText, div.MsoFootnoteText
	{mso-style-link:"Footnote Text Char";
	margin:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoCommentText, li.MsoCommentText, div.MsoCommentText
	{margin:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoHeader, li.MsoHeader, div.MsoHeader
	{mso-style-link:"Header Char";
	margin:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
p.MsoFooter, li.MsoFooter, div.MsoFooter
	{mso-style-link:"Footer Char";
	margin:0in;
	font-size:12.0pt;
	font-family:"Times New Roman",serif;}
span.MsoFootnoteReference
	{vertical-align:super;}
a:link, span.MsoHyperlink
	{color:blue;
	text-decoration:underline;}
a:visited, span.MsoHyperlinkFollowed
	{color:purple;
	text-decoration:underline;}
p
	{margin-right:0in;
	margin-left:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoCommentSubject, li.MsoCommentSubject, div.MsoCommentSubject
	{margin:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;
	font-weight:bold;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{margin:0in;
	font-size:8.0pt;
	font-family:"Tahoma",sans-serif;}
span.MsoPlaceholderText
	{color:gray;}
p.MsoRMPane, li.MsoRMPane, div.MsoRMPane
	{margin:0in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	font-size:10.0pt;
	font-family:"Arial",sans-serif;}
p.MsoTocHeading, li.MsoTocHeading, div.MsoTocHeading
	{margin-top:24.0pt;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:0in;
	line-height:115%;
	page-break-after:avoid;
	font-size:14.0pt;
	font-family:"Cambria",serif;
	color:#365F91;
	font-weight:bold;}
span.Heading1Char
	{mso-style-name:"Heading 1 Char";
	mso-style-link:"Heading 1";
	font-family:"Cambria",serif;
	font-weight:bold;}
span.HeaderChar
	{mso-style-name:"Header Char";
	mso-style-link:Header;}
span.FooterChar
	{mso-style-name:"Footer Char";
	mso-style-link:Footer;}
span.apple-style-span
	{mso-style-name:apple-style-span;}
span.apple-converted-space
	{mso-style-name:apple-converted-space;}
span.FootnoteTextChar
	{mso-style-name:"Footnote Text Char";
	mso-style-link:"Footnote Text";
	font-family:"Arial",sans-serif;}
span.msoIns
	{mso-style-name:"";
	text-decoration:underline;
	color:teal;}
span.msoDel
	{mso-style-name:"";
	text-decoration:line-through;
	color:red;}
.MsoChpDefault
	{font-size:10.0pt;}
 /* Page Definitions */
 @page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.25in 1.0in 1.25in;}
div.WordSection1
	{page:WordSection1;}
@page WordSection2
	{size:8.5in 11.0in;
	margin:1.0in 1.25in 1.0in 1.25in;}
div.WordSection2
	{page:WordSection2;}
 /* List Definitions */
 ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
html{background-color:#999999;}body{width:6.5in;background-color:#FFFFFF;float:none;margin:auto;padding-right:1in;padding-left:1in;}-->
</style>

</head>

<body lang=EN-US link=blue vlink=purple style='word-wrap:break-word'>

<div class=WordSection1>

<div>

<table cellspacing=0 cellpadding=0 hspace=0 vspace=0 width=557>
 <tr>
  <td valign=top align=left style='padding-top:0in;padding-right:0in;
  padding-bottom:0in;padding-left:0in'>
  <p class=MsoNormal style='background:white'><span style='font-size:8.0pt;
  font-family:"Helvetica",sans-serif;color:black'>United States Office
  of Water (4305) Doc Number <span
  style='background:yellow'>TBD</span></span></p>
  <p class=MsoNormal style='background:white'><span style='font-size:8.0pt;
  font-family:"Helvetica",sans-serif;color:black'>Environmental Protection Draft
  7/11/2024</span></p>
  <p class=MsoNormal style='background:white'><span style='font-size:8.0pt;
  font-family:"Helvetica",sans-serif;color:black'>Agency</span></p>
  <p class=MsoNormal style='line-height:3.1pt;background:white'>
  <table cellpadding=0 cellspacing=0 align=left>
   <tr>
    <td width=182 height=2></td>
   </tr>
   <tr>
    <td></td>
    <td><img width=557 height=4 src="AQUATOX.NET_1.0_UMAN_files/image001.gif"v:shapes="Rectangle_x0020_2"></td>
   </tr>
  </table>
  <span style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>
  <br clear=ALL>
  <p class=MsoNormal align=center style='text-align:center;background:white'><b><span
  style='font-size:28.0pt;font-family:"Helvetica",sans-serif;color:black'><img width=100  src="AQUATOX.NET_1.0_UMAN_files/image002.jpg"> AQUATOX DOT NET </span></b><b><br><span style='font-size:18.0pt;font-family:"Helvetica",sans-serif;
  color:black'>(RELEASE 1.0 beta)</span></b></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><b><span
  style='font-size:22.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><b><span
  style='font-size:22.0pt;font-family:"Helvetica",sans-serif;color:black'>MODELING
  ENVIRONMENTAL FATE</span></b></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><b><span
  style='font-size:22.0pt;font-family:"Helvetica",sans-serif;color:black'>AND
  ECOLOGICAL EFFECTS IN</span></b></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><b><span
  style='font-size:22.0pt;font-family:"Helvetica",sans-serif;color:black'>AQUATIC
  ECOSYSTEMS</span></b></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><span
  style='font-size:18.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><span
  style='font-size:18.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><span
  style='font-size:18.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>
  <p class=MsoNormal align=center style='text-align:center;background:white'><span
  style='color:black'></span><b><span style='font-size:20.0pt;font-family:
  "Helvetica",sans-serif;color:black'>DRAFT VOLUME 1: USERS MANUAL</span></b></p>
  </td>
 </tr>
</table>

</div>

<div>
</div>

<p class=MsoNormal>&nbsp;</p>

<div>
</div>

<p class=MsoNormal><span style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>

<p class=MsoNormal><span style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>

<p class=MsoNormal align=center style='margin-left:0in;text-align:center;
text-indent:0in'><img width=580 height=414 id="Picture 2"
src="AQUATOX.NET_1.0_UMAN_files/image003.jpg"><br clear=all style='page-break-before:
always'>

<p class=MsoNormal><span style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></p>

<span style='font-size:8.0pt;font-family:"Helvetica",sans-serif'><br clear=all
style='page-break-before:always'>
</span>

<p class=MsoNormal><a name="_Toc231869984"><span style='font-size:12.0pt'>&nbsp;</span></a></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='font-size:12.0pt'>&nbsp;</span></p>

<p class=MsoNormal align=center style='text-align:center;background:white'><b><span
style='font-size:26.0pt;font-family:"Helvetica",sans-serif;color:black'>AQUATOX
DOT NET </span></b></p>

<p class=MsoNormal align=center style='text-align:center;background:white'><b><span
style='font-size:19.0pt;font-family:"Helvetica",sans-serif;color:black'>(RELEASE
1.0 beta)</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>MODELING ENVIRONMENTAL FATE</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>AND ECOLOGICAL EFFECTS IN</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>AQUATIC ECOSYSTEMS</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><a
name="_Toc330995651"></a><a name="_Toc330995720"></a><a name="_Toc330995785"></a><a
name="_Toc330996002"></a><a name="_Toc330996575"></a><a name="_Toc330996713"></a><a
name="_Toc330996851"></a><a name="_Toc330998621"></a><a name="_Toc475713507"></a><a
name="_Toc475713652"></a><a name="_Toc475968271"></a><a name="_Toc77252194"></a><a
name="_Toc92890495"></a><a name="_Toc123899615"></a><a name="_Toc124171882"></a><a
name="_Toc124172085"><b><span style='font-size:16.0pt'>DRAFT VOLUME 1: USERS
MANUAL</span></b></a></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:16.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:14.0pt'>Jonathan S. Clough</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b>Date <span
style='background:yellow'>TBD</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='font-size:8.0pt;font-family:"Helvetica",sans-serif'>&nbsp;</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b>&nbsp;</b></p>

<p class=MsoNormal align=center style='text-align:center'><a
name="_Toc330995652"></a><a name="_Toc330995721"></a><a name="_Toc330995786"></a><a
name="_Toc330996003"></a><a name="_Toc330996576"></a><a name="_Toc330996714"></a><a
name="_Toc330996852"></a><a name="_Toc330998622"></a><a name="_Toc475713508"></a><a
name="_Toc475713653"></a><a name="_Toc475968272"></a><a name="_Toc77252195"></a><a
name="_Toc92890496"></a><a name="_Toc123899616"></a><a name="_Toc124171883"></a><a
name="_Toc124172086"><b>U.S. ENVIRONMENTAL PROTECTION AGENCY</b></a><b><br>
<br>
</b></p>

<p class=MsoNormal align=center style='text-align:center'><b>CENTER FOR
EXPOSURE ASSESSMENT MODELING<br>
<br>
</b></p>

<p class=MsoNormal align=center style='text-align:center'><b>ATHENS, GA</b></p>

<p class=MsoNormal> </p>

<span style='font-size:10.0pt;font-family:"Arial",sans-serif'><br clear=all
style='page-break-before:always'>
</span>

<p class=MsoNormal><a name="_Toc330995653"></a><a name="_Toc330995722"></a><a
name="_Toc330995787"></a><a name="_Toc330996004"><b><span style='font-size:
12.0pt'>Disclaimer</span></b></a><b><span style='font-size:12.0pt'><br>
<br>
</span></b></p>

<p class=MsoNormal>This document describes the interface of the aquatic
ecosystem model AQUATOX DOT NET, Release 1.0 beta. Anticipated users of this
document include persons who are interested in using the model, including but
not limited to researchers and regulators. The model described in this
document is not required, and the document does not change any legal requirements
or impose legally binding requirements on EPA, states, tribes or the regulated
community. <span style='background:yellow'>This document is pending approval for
publication by the Office of Science and Technology, Office of Water, U.S.
Environmental Protection Agency.</span> Mention of trade names, commercial
products or organizations does not imply endorsement or recommendation for use.<br>
<br>
</p>

<p class=MsoNormal><a name="_Toc330995654"></a><a name="_Toc330995723"></a><a
name="_Toc330995788"></a><a name="_Toc330996005"><b><span style='font-size:
12.0pt'>Acknowledgements</span></b></a></p>

<p class=MsoNormal><b><span style='font-size:12.0pt'>&nbsp;</span></b></p>

<p class=MsoNormal><span style='color:black'>This model has been developed and
documented by Dr. Richard A. Park of Eco Modeling and by Jonathan S. Clough of
Warren Pinnacle Consulting, Inc. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>AQUATOX DOT NET 1.0 (based on
AQUATOX 3.2) was developed under contract HHSN316201200013W, Task Order EP-G16H-01256
with General Dynamics Information Technology Inc. under guidance and direction
from Rajbir Parmar.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Release 2 of this model underwent independent peer review by
Donald DeAngelis, Robert Pastorok, and Frieda Taub; and Release 3 underwent
peer review by Marty Matlock, Damian Preziosi, and Frieda Taub. Their diligence
is greatly appreciated. </p>

<span style='font-size:12.0pt;font-family:"Arial",sans-serif'><br clear=all
style='page-break-before:always'>
</span>

<p class=MsoNormal><a name="_Toc330995655"></a><a name="_Toc330995724"></a><a
name="_Toc330995789"></a><a name="_Toc330996006"></a><a name="_Toc330996577"></a><a
name="_Toc330996715"></a><a name="_Toc330996853"></a><a name="_Toc330998623"></a><a
name="_Toc475713509"></a><a name="_Toc475713654"></a><a name="_Toc475968273"></a><a
name="_Toc77252196"></a><a name="_Toc92890497"></a><a name="_Toc123899617"></a><a
name="_Toc124171884"></a><a name="_Toc124172087"><b><span style='font-size:
16.0pt'>AQUATOX DOT NET (RELEASE 1.0 beta) VOLUME 1: USERS MANUAL</span></b></a><b><span
style='font-size:16.0pt'><br>
<br>
</span></b></p>

<p class=MsoToc1><a name="_Toc330995656"></a><a name="_Toc330995725"></a><a
name="_Toc330995790"><span style='font-size:14.0pt;font-family:"Arial",sans-serif'>Table
of Contents:</span></a><span style='font-size:14.0pt;font-family:"Arial",sans-serif'><br>
</span></p>

<p class=MsoToc2 style="margin-bottom: 3pt;"><a href="#_Toc124172088">Getting Started<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>1</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172089">AQUATOX: A Brief Overview<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>1</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172090">Installation Considerations<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172091">Loading a Simulation<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172092">The Main window<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>2</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172093">Saving a Simulation<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172094">What is included in an AQUATOX JSON
file?<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172095">Parameters within Database Files vs.
Parameters within in a Simulation<span style='color:windowtext;display:none;
text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>4</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172096">Exploring State Variables<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc4><a href="#_Toc124172097">State Variables<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc4><a href="#_Toc124172098">Modifying the State Variable List<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc4><a href="#_Toc124172099">Initial Conditions and Loadings<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>5</span></a></p>

<p class=MsoToc4><a href="#_Toc124172100">Importing Loadings<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>6</span></a></p>

<p class=MsoToc4><a href="#_Toc124172101">Parameters<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>10</span></a></p>

<p class=MsoToc4><a href="#_Toc124172102">Chemical Properties and Fate Data
Screen<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>11</span></a></p>

<p class=MsoToc4><a href="#_Toc124172103">Chemical Bioaccumulation and Toxicity<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>12</span></a></p>

<p class=MsoToc4><a href="#_Toc124172104">Plant Data Screen<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>13</span></a></p>

<p class=MsoToc4><a href="#_Toc124172105">Animal Data Screen<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>15</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172106">Site Information<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>16</span></a></p>

<p class=MsoToc4><a href="#_Toc124172107">Site Parameters<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>17</span></a></p>

<p class=MsoToc4><a href="#_Toc124172108">Remineralization<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>18</span></a></p>

<p class=MsoToc4><a href="#_Toc124172109">Modeling Shade<span style='color:
windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>19</span></a></p>

<p class=MsoToc4><a href="#_Toc124172110">Velocity<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>19</span></a></p>

<p class=MsoToc4><a href="#_Toc124172111">Mean Depth<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>19</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172112">Setup Parameters<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>19</span></a></p>

<p class=MsoToc4><a href="#_Toc124172113">Rate Output<span style='color:windowtext;
display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>21</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172114">Run Button (running the model)<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>21</span></a></p>

<p class=MsoToc4><a href="#_Toc124172115">Multiple Archived Simulations<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>21</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172116">Output Window<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>22</span></a></p>

<p class=MsoToc4><a href="#_Toc124172117">Viewing Existing Graphs<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>22</span></a></p>

<p class=MsoToc4><a href="#_Toc124172118">Setting Up a New Graph<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>22</span></a></p>

<p class=MsoToc2 style="margin-bottom: 3pt;"><a href="#_Toc124172119">Setting Up Simulation, Single Segment<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>23</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172120">Data Requirements<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>23</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172121">Site Types<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>24</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172122">Starting with a Surrogate Simulation<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>25</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172123">Water Volume Modeling Options<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>25</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172124">Adding a State Variable<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>25</span></a></p>

<p class=MsoToc4><a href="#_Toc124172125">Adding a Chemical<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>25</span></a></p>

<p class=MsoToc4><a href="#_Toc124172126">Adding a Plant<span style='color:
windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>26</span></a></p>

<p class=MsoToc4><a href="#_Toc124172127">Adding an Animal<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>26</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172128">Using Sediment Bed Models and Data
Requirements<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>26</span></a></p>

<p class=MsoToc4><a href="#_Toc124172129">Default Sediment Bed Model<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>27</span></a></p>

<p class=MsoToc4><a href="#_Toc124172130">TSS<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>27</span></a></p>

<p class=MsoToc4><a href="#_Toc124172131">Sediment Diagenesis Model<span
style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>27</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172132">Model Calibration<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>28</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172133">Model Validation<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>29</span></a></p>

<p class=MsoToc2 style="margin-bottom: 3pt;"><a href="#_Toc124172134">Multi-Segment Runs<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>30</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172135">New Simulation Window<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>32</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172136">Select Surrogate Simulation<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>32</span></a></p>

<p class=MsoToc2 style="margin-bottom: 3pt;"><a href="#_Toc124172137">Tutorial<span style='color:windowtext;
display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>33</span></a></p>

<p class=MsoToc3 style="margin-bottom: 3pt;"><a href="#_Toc124172138">Simple Tutorial for a 0-D simulation<span
style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>33</span></a></p>

<p class=MsoToc4><a href="#_Toc124172139">Tutorial-- Step 1:&nbsp; Deleting and
Adding a Plant<span style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>33</span></a></p>

<p class=MsoToc4><a href="#_Toc124172140">Tutorial-- Step 2:&nbsp; Setting an
Initial Condition<span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>33</span></a></p>

<p class=MsoToc4><a href="#_Toc124172141">Tutorial-- Step 3:&nbsp; Viewing
Parameters in a Simulation<span style='color:windowtext;display:none;
text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>34</span></a></p>

<p class=MsoToc4><a href="#_Toc124172142">Tutorial-- Step 4:&nbsp; Viewing
Toxicant Loadings<span style='color:windowtext;display:none;text-decoration:
none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>36</span></a></p>

<p class=MsoToc4><a href="#_Toc124172143">Tutorial-- Step 5:&nbsp; Running the
Simulation<span style='color:windowtext;display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>37</span></a></p>

<p class=MsoToc4><a href="#_Toc124172144">Tutorial-- Step 6:&nbsp; Viewing
Output<span style='color:windowtext;display:none;text-decoration:none'> </span><span
style='color:windowtext;display:none;text-decoration:none'>37</span></a></p>

<p class=MsoToc2 style="margin-bottom: 3pt;"><a href="#_Toc124172145">References<span style='color:windowtext;
display:none;text-decoration:none'>. </span><span
style='color:windowtext;display:none;text-decoration:none'>38</span></a></p>

<p class=MsoNormal>&nbsp;</p>

</div>

<b><span style='font-size:12.0pt;font-family:"Arial",sans-serif'><br clear=all
style='page-break-before:always'>
</span></b>

<div class=WordSection2>

<h2><a name="_Toc124172088"><span style='font-size:14.0pt;color:black'>Getting
Started</span></a></h2>

<h3><a name=Overview1></a><a name="_Toc124172089"><span style='font-size:12.0pt;
color:black'>AQUATOX: A Brief Overview</span></a></h3>

<p><span style='color:black'>AQUATOX is a simulation model for aquatic systems.
AQUATOX predicts the fate of various pollutants, such as nutrients and organic
chemicals, and their effects on the ecosystem, including fish, invertebrates,
and aquatic plants. AQUATOX is a valuable tool for ecologists, biologists,
water quality modelers, and anyone involved in performing ecological risk
assessments for aquatic ecosystems. </span></p>

<p><span style='color:black'>AQUATOX simulates the transfer of biomass, energy
and chemicals from one compartment of the ecosystem to another. It does this by
simultaneously computing each of the most important chemical or biological
processes for each day of the simulation period; therefore, it is known as a
process-based or mechanistic model. AQUATOX can predict not only the
environmental fate of chemicals in aquatic ecosystems, but also their direct
and indirect effects on the resident organisms. Therefore, it has the potential
to establish causal links between chemical water quality and biological
response and aquatic life uses.</span></p>

<p><span style='color:black'>AQUATOX is the only general ecological risk model
that represents the combined environmental fate and effects of conventional
pollutants, such as nutrients and sediments, and toxic chemicals in aquatic
ecosystems. It considers several trophic levels, including attached and
planktonic algae and submerged aquatic vegetation, invertebrates, and forage,
bottom-feeding, and game fish; it also represents associated organic toxicants.
AQUATOX has been implemented for streams, ponds, lakes, estuaries, reservoirs,
and experimental enclosures. </span></p>

<div>

<p class=MsoNormal><span style='color:black'>The fate portion of the model,
which is applicable especially to organic toxicants, includes the following:
partitioning among organisms, suspended and sedimented detritus, suspended and
sedimented inorganic sediments, and water; volatilization; hydrolysis;
photolysis; ionization; and microbial degradation. The effects portion of the
model includes the following: acute toxicity to the various organisms modeled;
and indirect effects such as release of grazing and predation pressure,
increase in detritus and recycling of nutrients from killed organisms,
dissolved oxygen sag due to increased decomposition, and loss of food base for
animals.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>AQUATOX DOT NET is the latest in a
long series of models, starting with the aquatic ecosystem model CLEAN (Park et
al., 1974) and subsequently improved in consultation with numerous researchers
at various European hydrobiological laboratories, resulting in the CLEANER series
(Park et al., 1975, 1979, 1980; Park, 1978; Scavia and Park, 1976) and
LAKETRACE (Collins and Park, 1989). The MACROPHYTE model, developed for the
U.S. Army Corps of Engineers (Collins et al., 1985), provided additional
capability for representing submersed aquatic vegetation. Another series
started with the toxic fate model PEST, developed to complement CLEANER (Park
et al., 1980, 1982), and continued with the TOXTRACE model (Park, 1984) and the
spreadsheet equilibrium fugacity PART model. AQUATOX combined algorithms from
these models with an ecotoxicological construct borrowed from the FGETS model
(Surez and Barber, 1992); and additional code was written as required for a
truly integrative fate and effects model (Park, 1990, 1993). In the late 1990s,
AQUATOX was restructured and linked to Microsoft Windows interfaces to provide
even greater flexibility, capacity for additional compartments, and user
friendliness. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='color:black'>AQUATOX Release 1 was produced in 2002
     and was the first EPA release to run under Windows.</li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 2 was completed in
     2003 and included more state variables and multi-age-class fish along with
     a refined user-interface.</li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 2.1 was completed in
     2005 and included additional chemical modeling options and variable
     stoichiometry among numerous other refinements.</li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 2.2 was completed in 2006
     and included updated simulations and parameter databases along with minor
     interface enhancements.</li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 3 was completed in
     2009 and includes linked segments, simulations of estuaries, dramatically
     improved output capabilities, and many other model improvements. </li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 3.1 was completed in
     2012 and includes a steady-state diagenesis model for sediments, updated
     ICE data (toxicity regressions), modified denitrification code, and many
     interface enhancements.</li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 3.1 plus was completed
     in 2014 and includes the option to model nutrient limitation in plants
     based on internal rather than external nutrient concentrations.</li>
 <li class=MsoNormal style='color:black'>AQUATOX Release 3.2 was completed in
     2017 and includes an update to the database management system, an ASCII
     input/output option, a command-line version, and
     nearshore-marine-environment modeling updates, as discussed in its 3.2 <i>Technical
     Documentation </i>(<span style='color:windowtext'>U.S. Environmental
     Protection Agency. 2018)</span><i>.</i></li>
 <li class=MsoNormal style='color:black'>AQUATOX DOT NET Release 1.0 was
     completed in 2024 AQUATOX code was translated to C# in Microsoft Visual
     Studio. The interface was simplified to some degree, and a new
     multi-segment model capability is available that includes links to
     external model and data sources for stream networks, nutrients and
     sediment loadings, and water flows.</li>
</ul>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h3><a name="_Toc124172090"><span style='font-size:12.0pt'>Installation
Considerations</span></a></h3>

<p class=MsoNormal>AQUATOX DOT NET source code was developed in .NET 6.0. At
this time, the graphical interface for AQUATOX DOT NET is exclusively provided
as a Microsoft Windows Forms application. However, all technical components of
the model are also being developed as EPA HMS web services. </p>

<p class=MsoNormal>For instructions on deployment on a Windows machine, Please
see the Deployment_Notes document that in the base directory of the download
files package.<br>
<br>
</p>

<h3 style='page-break-after:avoid'><a name=splash><span style='font-size:12.0pt;
color:black'>Single-Segment Mode and Multi-Segment Mode</span></a></h3>

<p>At the time of model startup, a user is prompted as to whether to run the
model in single-segment mode or multi-segment mode. </p>

<p>Single-segment mode represents a single well-mixed compartment (0-D) in
which to model the interrelationships between sediments, nutrients, organic chemicals,
plants, and animals. </p>

<p><a href="#Multi_Segment_Runs">Multi-segment mode</a> represents many 0-D segments
linked together that are then run sequentially. In multi-segment mode, data
may be imported from the NHD Plus, for stream-network information, the National
Water Model for water flows and lake/reservoir volumes, and HAWQS/SWAT for nutrients,
sediments, and water flows.</p>

<p>The user interface of multi-segment mode may also be used to automatically
link to external data sources for single HND Plus lake/reservoir segments or to
run a HAWQS model for a single drainage basin (at USGS HUC8-HUC12 to HAWQS
HUC14 resolution.)</p>

<p>By returning to the initial splash screen and pressing one of the buttons
again, Multiple single-segment or multi-segment mode windows may be opened at
the same time. This enables multiple simulations to be running at the same
time, or model outputs may be compared on adjacent windows. </p>

<p class=MsoNormal><a name=LoadingAStudy></a><a name="_Toc124172091"></a><a
name=SingleSegment><b><span style='font-size:14.0pt;color:black'>Single-Segment
Simulations</span></b></a></p>

<h3 style='page-break-after:avoid'><span style='font-size:12.0pt;color:black'>Loading
a Simulation</span></h3>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>For
AQUATOX DOT NET, a JSON text file is the basic unit in AQUATOX; it contains
site data, loadings, and parameter values used in a simulation; and it may
contain results from a prior simulation.&nbsp;&nbsp; Click on the <b>Load </b>button
in the upper left corner of the screen. In the <b>Studies </b>directory there
is a choice of AQUATOX JSON simulation files to load.&nbsp;&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>When a simulation is opened, state
variables are shown within the AQUATOX interface.&nbsp;</span></p>

</div>

<h3><a name="_Toc124172092"></a><a name=MainScreen></a><span style='font-size:
12.0pt;color:black'>The Main window</span></h3>

<div>

<p class=MsoNormal><span style='color:black'>The main window includes the name
of the simulation, the list of state variables used, and buttons from which to
choose various operations.&nbsp;&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Name of Simulation</b> field
(top center) can be edited; it is separate from the name of the JSON file you
loaded, which is displayed in the pathway at the top of the screen.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The window buttons and labels
moving from top to bottom down the left side are as follows.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <b>Load </b>button was
discussed in the previous section, Loading a Simulation.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <b>Save </b>button allows the
user to save the simulation and all archived results to a new JSON file that
can then be loaded with all parameters and driving time series.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='color:black'>Edit Setup </span></b><span
style='color:black'>allows the user to set the dates of the simulation, and to
specify various options such as the saving of biologic rates.&nbsp;&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The<b> Run </b>button starts the
simulation. The date of this simulation run will be saved, and a text
description of the run can be added at the time of integration.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Run Status</b> <b>label</b>
tells how many sets of archived results are included within the simulation at
any time.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>If you hit the <b>Help</b> button
from the Main window, you will jump to this topic in the on-screen version of
this Users Manual.&nbsp;The <b>Help</b> button on other screens links to the
appropriate subject in the on-screen version of this Users Manual. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <b>Output</b> button brings up
an interface to present the results as a series of charts and graphs, or export
archived results to comma separated format.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>At the upper right of the
interface is a panel showing <b>Databases of Parameter Values. </b>These
databases are not attached to the JSON file and editing these databases will
have no effect on the simulation unless a set of parameters is later loaded
from the database into the simulation. More information can be found on this
in </span><a href="#_Databases_vs._Parameters">Parameters within Database
Files vs. Parameters within in a Simulation</a><span style='color:black'>.
There are separate databases for Sites, Animals, Remin. Records (i.e.,
remineralization or organic matter parameters), Chemicals, and Plants,
respectively that can be edited via these buttons and then later imported into
a simulation.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>In the center panel is a group of
buttons to edit <b>model parameters</b>:</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The <b>Site</b> button loads the site
characteristic screen for the current simulation.&nbsp; </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The <b>Org. Matter</b> button loads the
remineralization record for the current simulation that can be edited by the
user.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The<b> Plants</b> button brings up an
editable matrix of all plant parameters in the simulation. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The<b> Chemicals</b> button brings up an
editable matrix of all chemical parameters in the simulation. </span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The<b> Animals</b> button brings up an
editable matrix of all animal parameters in the simulation. </span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The<b> Diagenesis</b> button brings up
an editable set of parameters pertaining to the sediment diagenesis model (when
relevant)</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoListParagraph style='text-indent:-.25in'><span style='font-family:
Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>The</span><b><span style='color:black'>
Food Web</span></b><span style='color:black'>&nbsp;button brings up an editable
matrix of trophic interactions for the simulation.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>To the right of the center panel and
below the Databases of Parameter Values is a list of the state and driving variables
in the simulation.&nbsp; Variables can be added to or deleted from this list
using the <b>Add</b> or <b>Delete</b> buttons, respectively, at the bottom of
the list.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Clicking the Edit button below
the list of state and driving variables, or double-clicking on any of the state
or driving variables in this list will bring up the initial conditions and
loadings associated with each of the selected state variables. For animals,
plants, and chemicals, parameters defining the state variables characteristics
can also be edited from this window. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>You may also Add or Delete
variables from this list. Note that the food web may need to be updated
following this procedure.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='color:black'>Note:</span></b><span
style='color:black'> if no simulation has been loaded, several
buttons/locations of information will not be displayed; only the following can
be seen:</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Load, Save, Help, and Multi-Segment
Runs buttons (on the left side)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Run Status label but without detail (on
the left side)</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Name of Simulation label and empty
field</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Databases of Parameter Values panel
with its five buttons (upper right side)</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h3><a name=SaveSimulation></a><a name="_Toc124172093"><span style='font-size:
12.0pt;color:black'>Saving a Simulation</span></a></h3>

<div>

<p class=MsoNormal><span style='color:black'>To save a simulation to a JSON
file, click on the <b>Save</b> button at the upper left of the window. The
JSON saved will include all model parameters, driving time series, and sets of
model results. The user will be prompted whether to overwrite the original
parameters or save these parameters as a new file.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<p class=MsoNormal><b><span style='font-size:14.0pt;color:black'>Working with
Existing Simulations</span></b></p>

<h3><a name="_Toc124172094"><span style='font-size:12.0pt'>What is included in
an AQUATOX JSON file?</span></a></h3>

<p>The AQUATOX JSON simulation file is the basic unit in which AQUATOX
simulations are loaded and saved.&nbsp; Each JSON simulation file contains the
following items:</p>

<ul type=disc>
 <li class=MsoNormal>List of state variables and driving variables utilized and
     their loadings, parameters, and initial conditions; </li>
 <li class=MsoNormal>Site specific and remineralization parameters; </li>
 <li class=MsoNormal>Model setup information; </li>
 <li class=MsoNormal>Boundary condition loadings for a simulation; </li>
 <li class=MsoNormal>Results from any simulation that may have previously been
     run, including &quot;rates&quot;; </li>
 <li class=MsoNormal>The graph library that has been produced for that
     simulation; </li>
</ul>

<p>Databases of parameters are not attached to the loaded JSON simulation file,
and editing these databases will have no effect on the simulation unless a set
of parameter values is later loaded from the parameter database into the
simulation. These databases of parameters are provided as separate JSON files
that include libraries of subject-specific parameters that can be loaded into a
simulation as needed.</p>

<h3><a name=Libraries></a><a name="_Toc124172095"></a><a
name="_Databases_vs._Parameters"></a><span style='font-size:12.0pt;color:black'>Parameters
within Database Files vs. Parameters within in a Simulation</span></h3>

<div>

<p class=MsoNormal><span style='color:black'>There are two approaches to
working with parameters in AQUATOX..&nbsp; You have a choice of editing specific
(animals, plants, etc.) parameter database JSON files or of opening and editing
a particular simulation file.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The parameter database files are stand-alone
files that do not affect model results unless some or all of their contents are
loaded into a simulation.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Simulation files are
self-contained JSON files with all the information intended for a particular
simulation, including initial conditions, loadings, parameter values, first and
last dates for the simulation, and simulation results.&nbsp; Within the simulation
file, parameter values can be edited, but those changes will apply <i>only</i>
to that simulation.&nbsp; The intent is to be able to archive a model
application so that all assumptions and results are saved for future
reference.&nbsp; This is especially important for regulatory applications that
are subject to later review. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Sets of parameters that will be
used repeatedly should be saved in the appropriate database.&nbsp; Each database
in JSON format includes parameter values for a list of organisms, chemicals, or
sites.&nbsp; Generally, editing of parameters should be done in the database
mode to maintain consistency among studies.&nbsp; In contrast, if a site record
is going to be used only for a single simulation, it may be desirable to create
a unique set of parameters within the simulation.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>It is the users responsibility,
though, to synchronize parameter values among studies.&nbsp; This can be done
by saving a set of parameters to one of the existing databases and then loading
that record to each simulation. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal>See also the section on <a href="#Parameters">parameters</a><span
class=MsoHyperlink>.</span></p>

</div>

<h3><a name="_Toc124172096"><span style='font-size:12.0pt;color:black'>Exploring
State Variables</span></a></h3>

<h4><a name="_Toc124172097"><span style='font-size:11.0pt'>State Variables</span></a></h4>

<div>

<p class=MsoNormal><span style='color:black'>State variables are those
ecosystem components that are being simulated.&nbsp; These include organic
toxicants, nutrients and dissolved gasses, organism and detrital compartments
and their associated toxicants,&nbsp;and other variables traditionally considered
driving variables, such as water inflow, temperature, pH, light, and
wind.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>AQUATOX is a powerful model
because you can add or delete state variables.&nbsp; It is even possible to
remove all biotic components in order to model a tank or other sterile
system.&nbsp; In general, the fewer state variables, the better.&nbsp; In particular,
unnecessary state variables slow down the simulation and create additional
requirements for verification.&nbsp; This is especially true for streams, which
tend to be more dynamic and therefore slower to simulate.&nbsp; Nevertheless,
often it is desirable to model a food web rather than a food chain, for example
to examine the possibility of less tolerant organisms being replaced by more
tolerant organisms as environmental perturbations occur.&nbsp; The choice of
which state variables to model depends to a large extent on the purpose of the
modeling application and the availability of data pertaining to the state
variables.</span></p>

</div>

<h4><a name="_Toc124172098"><span style='font-size:11.0pt'>Modifying the State
Variable List</span></a></h4>

<div>

<p class=MsoNormal><b><span style='color:black'>To Delete</span></b><span
style='color:black'> a state variable, select the variable you wish to delete
from the state variables list (on the opening screen after opening a JSON file)
by clicking on it.&nbsp; To select multiple state variables, hold down the
control key while clicking more than one variable on the list.&nbsp; Then,
click on the <b>delete</b> button and confirm the deletion.&nbsp; There are
several state variables (such as nutrients) that are basic to an AQUATOX
simulation, and that therefore cannot be deleted.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><b><span style='color:black'>To Add </span></b><span
style='color:black'>a state variable, click on the <b>Add </b>button and choose
the variable you wish to add from the dialog box that appears.&nbsp; Note that
the names of the taxonomic groups and ecologic guilds on the main window are
followed by the names of the specific groups in brackets.&nbsp; After clicking
the <b>Add</b> button, you will first be prompted to choose the taxonomic
group, ecologic guild, or chemical compartment you wish to add.&nbsp; Then, you
will be prompted to load the chemical or species-specific <b>parameters</b>
from the appropriate database.</span></p>

</div>

<div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

</div>

<div>

<p class=MsoNormal><span style='color:black'>After animal or plant state
variables have been added or removed,&nbsp;one <strong><span style='font-family:
"Arial",sans-serif'>critical step</span></strong> is to view the </span><strong><span
style='font-family:"Arial",sans-serif'>trophic interactions</span></strong><span
style='color:black'> for the simulation to ensure that the resulting food web
is reasonable.&nbsp; AQUATOX <em><span style='font-family:"Arial",sans-serif'>does</span></em>
have default trophic interactions within each organism, but not knowing which
organisms are going to be included in each simulation, the trophic interaction
records will usually require modification.&nbsp; The best way to access trophic
interactions is by selecting <strong><span style='font-family:"Arial",sans-serif'>&quot;Food
Web&quot;</span></strong> on the Main window.</span></p>

</div>

<h4><a name=ICandLoadings></a><a name="_Toc124172099"><span style='font-size:
11.0pt'>Initial Conditions and Loadings</span></a></h4>

<div>

<p class=MsoNormal><span style='color:black'>Initial values and loadings are
needed for all the state variables or compartments simulated.&nbsp; These are
input on the loadings screen by double-clicking on the name in the state
variables list.&nbsp; If one or more toxicants are modeled, then initial
concentrations associated with the biota can also be specified.&nbsp; Constant
loadings for plants and invertebrates can be considered as seed values,
although care should be taken to use small values or the loadings can dominate
the simulation.&nbsp; Even periphyton and zoobenthos may be maintained through
drift from upstream, and a constant loading is appropriate.&nbsp; Likewise,
macrophytes might die back in winter and sprout from rhizomes; because rhizomes
are not explicitly modeled, a small loading is the mechanism for reestablishing
the population in the simulation when environmental conditions become more
favorable.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Of course, upstream loadings could
be significant inputs to a reach or lake.&nbsp; These might be represented by
constant or dynamic (time-varying) loadings.&nbsp; AQUATOX has a very flexible
interpolation routine to obtain daily values from irregular data points and
even time series occurring or extending outside the simulation period.&nbsp;
Dynamic loadings can be entered directly on the loadings screen, or they can be
entered or obtained offline and imported into the model.&nbsp; Imported data
can be in a variety of formats, which are evident when the &quot;<strong><span
style='font-family:"Arial",sans-serif'>Change</span></strong>&quot; button is
used.&nbsp; Loadings can be altered by means of a multiplier (the <strong><span
style='font-family:"Arial",sans-serif'>&quot;Multiply loading by&quot; </span></strong>button).&nbsp;&nbsp;This
procedure is especially useful for analyzing various loading scenarios.&nbsp;
It is also a way to correct or convert data series.&nbsp; However, ordinarily
the multipliers are set to 1 for the Control simulation, so their use for other
than perturbations is discouraged.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Loadings in &quot;inflow
water&quot; are&nbsp;closely related to the volume of inflow water specified
(or calculated as a result of choices) in the water volume screen.&nbsp;&nbsp;
In other words, loadings in unit per liter of water must have an associated
inflow of water in order to be relevant to the simulation.&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>On the other hand, nonpoint-source
(NPS), and point-source (PS) loadings are&nbsp;input in units of&nbsp;grams per
day and are not affected by the quantity of inflow water.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Loadings associated with direct
precipitation are a function of the site's surface area with units of grams per
meter squared per day.&nbsp; This input field includes both wet and dry
precipitation conditions.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>When working with a system with a relatively
low retention time, such as a segment of a river, loading of floating biotic
state variables such as phytoplankton can be important to characterize properly.
These loadings are generally entered in units of milligrams per liter.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Any of the time-series loadings
may be imported, exported, or cleared&nbsp;using the &quot;<strong><span
style='font-family:"Arial",sans-serif'>Change</span></strong>&quot; button
found directly below their listing. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Also see the section titled &quot;</span><a
href="#Important_Note_about_Dynamic_Loadings">Important Note about Dynamic
Loadings</a><span style='color:black'>&quot; describing how AQUATOX
interpolates between data points. </span></p>

</div>

<h4><a name="Import_and_Export_Loadings"></a><a name="_Toc124172100"><span
style='font-size:11.0pt'>Importing Loadings</span></a></h4>

<p>Loadings and other time series can be imported from&nbsp;Comma Separated
Value (CSV) format files using the Import from File button below the list of
loadings. The date should appear in the first column and the numerical value
in the second column of the file.</p>

<h5><a name="Important_Note_about_Dynamic_Loadings"><span style='color:black'>Important
Note about Dynamic Loadings:</span></a></h5>

<div>

<p class=MsoNormal><span style='color:black'>Dynamic loadings are loadings that
are variable over the simulation's time-period.&nbsp; These loadings are
entered using a list of dates and associated loadings.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>During a simulation, if the date
that is being simulated appears on the input list of dates, the loading is
taken directly from the list.&nbsp; If the execution date of in a simulation
occurs <strong><span style='font-family:"Arial",sans-serif'>between two dates</span></strong>,
interpolation is used to determine the correct loading value.&nbsp; Because of
this interpolation, if the intent is to represent a spike such as from storm
runoff on a particular day, the spike loading should be bracketed by zero
(0&quot;) loadings on the day before and after the spike loading.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>If the current date in a
simulation occurs <strong><span style='font-family:"Arial",sans-serif'>before
the first date or after the last date</span></strong><strong><span
style='font-family:"Arial",sans-serif;font-weight:normal'> of the loading time
series</span></strong>,&nbsp;AQUATOX&nbsp;assumes that the loadings wrap
around with an annual cycle.&nbsp;&nbsp;Specifically, the AQUATOX algorithm
will step towards the input&nbsp;data in one-year increments&nbsp;until the
derived date falls within&nbsp;the&nbsp;input time series.&nbsp; The model will
then interpolate the results, if necessary, and assign&nbsp;the results to the
date being modeled.&nbsp;&nbsp;In this manner, if you had two years of loadings
but ran the model for eight years, the model would repeat the second year
of&nbsp;loadings in an annual cycle for the last&nbsp;seven years of the
simulation.&nbsp; (Note: if a different type of annual cycle or interpolation
is desired this can be derived outside of the AQUATOX interface and then
imported into the model.)</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Exercise caution when modeling
multiple years using loadings data from only one or a few years.&nbsp; Sporadic
loadings, which would be expected in that one particular year, may
inappropriately be repeated in successive years.&nbsp; If you do not wish
loadings to be repeated, enter values (0&quot; or otherwise) for the first and
last days of the simulation.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>If there is only one dynamic
loading point present, this is interpreted in the same manner as a constant
load.</span></p>

</div>

<h5><a name=NutrientLoadings>Nutrient Loadings</a></h5>

<div>

<p class=MsoNormal><span style='color:black'>To access this screen,
double-click on Total Ammonia as N,<b></b> Nitrate as N,<b> </b>or Total
Soluble P<b></b> in the state variables list in the main window.</span></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>The nutrient&nbsp;initial conditions and
loadings&nbsp;screens include the capability to model <strong><span
style='font-family:"Arial",sans-serif'>Total N</span></strong> and <strong><span
style='font-family:"Arial",sans-serif'>Total P</span></strong> along with all
of the&nbsp;items contained on other loadings screens.</p>

</div>

<div>

<p class=MsoNormal>&nbsp;</p>

</div>

<div>

<p class=MsoNormal>When initial conditions, inflows, and other loadings are
entered as Total N, by selecting the Nitrate inputs represent Total N radio
button, model inputs are located in the &quot;Nitrate&quot; initial conditions
and loadings screen, and loadings on the ammonia screen are not relevant.&nbsp;
Total N is assumed to be 12% ammonia for inflow and nonpoint-source loadings
and 15% ammonia for point-source loadings.</p>

</div>

<div>

<p class=MsoNormal>&nbsp;</p>

</div>

<div>

<p class=MsoNormal>When Total N or Total P are used as model inputs, AQUATOX
calculates the dissolved content by subtracting out loading&nbsp;inputs for
suspended and dissolved detritus and suspended algae as discussed in section
5.4 of the Release 3.2 technical documentation (U.S. Environmental Protection
Agency. 2018).&nbsp; </p>

</div>

<h5><a name=SuspDiss></a><a name=DetrScreen><span style='color:black'>Detrital
Initial Conditions and Loadings</span></a></h5>

<div>

<p class=MsoNormal><span style='color:black'>To access this screen,
double-click on the Susp. and dissolved detritus in the state variables list
in the main window.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>A complex loading screen is
necessary for organic matter inputs in the water column.&nbsp; AQUATOX
simulates <b>Organic Matter</b> (dry weight); however, the user can input data
as <b>Organic Carbon</b> or <b>CBOD</b> (Carbonaceous Biochemical Oxygen Demand)
and the model will make the necessary conversions.&nbsp; See the AQUATOX
Release 3.2 technical documentation (U.S. Environmental Protection Agency.
2018) for more information about how the model converts organic carbon and CBOD
loadings into organic matter.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>By selecting the Loading Type
dropdown, the type of variable input may be specified. Organic
matter&nbsp;initial conditions and loadings are divided into four compartments:</span></p>

</div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>particulate refractory detritus; </li>
 <li class=MsoNormal style='color:black'>particulate&nbsp;labile detritus; </li>
 <li class=MsoNormal style='color:black'>dissolved refractory detritus; and </li>
 <li class=MsoNormal style='color:black'>dissolved labile organic matter.</li>
</ul>

<p><span style='color:black'>Initial conditions and loadings are parsed by
specifying <b>% Particulate</b> and <strong><span style='font-family:"Arial",sans-serif'>%
Refractory </span></strong>which can be entered as constant or time-varying
percentages (0100). &nbsp;Loadings of organic matter can be constant or dynamic
(time series) for concentrations in inflowing water (mg/L), and for mass from
point sources and non-point sources (g/d).&nbsp; Toxicants associated with
detritus also can be specified (&#956;g/kg).&nbsp;&nbsp; </span></p>

<p><span style='color:black'>Organic matter loadings in &quot;inflow
water&quot; are&nbsp;closely related to the volume of inflow water specified
(or calculated as a result of choices) in the water volume screen.&nbsp;&nbsp;
In other words, loadings in unit per liter of water must have an associated
inflow of water in order to be relevant to&nbsp;a simulation. </span></p>

<div>

<p class=MsoNormal><span style='color:black'>Separate state variable input
screens are provided for refractory and labile organic matter within the
sediment bed.&nbsp; The initial conditions are given as g/m<sup>2</sup>, and
the loadings are given as mg/L.&nbsp; Associated toxicants are given as &#956;g/kg
(ppb).</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>See section 5.1 of the <a
name="_Hlk94535032">r</a></span>elease 3.2 <span style='color:black'>technical
documentation (U.S. Environmental Protection Agency. 2018) for more
information. Also see the section titled </span><a
href="#Important_Note_about_Dynamic_Loadings">Important Note about Dynamic
Loadings</a><span style='color:black'>.</span></p>

</div>

<h5><a name=Temperature><span style='color:black'>Temperature Data Screen</span></a></h5>

<div>

<p class=MsoNormal><span style='color:black'>To access this screen,
double-click on Temperature<b></b> in the state variables list in the main
window.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The annual mean and range in
temperature from the site parameters screen can be used or a time series can be
enteredin which case make sure that the complete time period being simulated
is covered.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>For more information on
temperature modeling, see section 3.5 of the r</span>elease 3.2 <span
style='color:black'>technical documentation (U.S. Environmental Protection
Agency. 2018). If time series are entered you may wish to see the note about </span><a
href="#Important_Note_about_Dynamic_Loadings">dynamic loadings</a><span
style='color:black'>.</span></p>

</div>

<h5><a name=Wind><span style='color:black'>Wind Loadings Screen</span></a></h5>

<div>

<p class=MsoNormal><span style='color:black'>To access this screen,
double-click on <b>Wind Loading</b> in the state variables list in the main
window.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Within the wind loadings screen,
you can either use a default time series for loadings, a constant wind loading,
or enter time series loadings.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Wind may be an important driving
variable because it determines the stability of blue-green algal blooms,
affects reaeration or oxygen exchange, and controls volatilization of some
organic chemicals. Wind also can affect the degree of mixing in estuaries.
Wind is usually measured at meteorological stations at a height of 10 m from
the ground surface and is expressed as m/s.&nbsp; Wind is&nbsp;less important
for rivers and streams.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>For the default time series, wind
is computed using a complex Fourier series of sines and cosines for a 365-day
repeating period with a user-supplied <b>mean value</b>.&nbsp;&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>For more information on wind, see
section 3.7 of the r</span>elease 3.2 <span style='color:black'>technical
documentation (U.S. Environmental Protection Agency. 2018).</span></p>

</div>

<h5><a name=Light><span style='color:black'>Light Loadings Screen</span></a></h5>

<div>

<p class=MsoNormal><span style='color:black'>To access this screen,
double-click on Light in the state variables list in the main window.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>When entering light data, the user
has three options:&nbsp; constant, time series, or annual mean and range may be
given for light in Langleys/day.&nbsp; If annual mean and range are used, these
parameters must be filled-in on the site parameters screen.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Also, photoperiod can be
auto-calculated from latitude or entered manually.&nbsp; The latter can be
useful when modeling experimental facilities.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>See section 3.6 of the r</span>elease
3.2 <span style='color:black'>technical documentation (U.S. Environmental
Protection Agency. 2018). Also see the section titled </span><a
href="#Important_Note_about_Dynamic_Loadings">Important Note about Dynamic
Loadings</a><span style='color:black'>.</span></p>

</div>

<h5 style='page-break-after:avoid'><a name=pHScreen><span style='color:black'>pH
Screen</span></a></h5>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>To
access this screen, double-click on pH in the state variables list in the
main window.</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>pH is important in AQUATOX for
several reasons. </span></p>

</div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Ionization of ammonia is sensitive to
     pH; </li>
 <li class=MsoNormal style='color:black'>Nitrification rates are sensitive; </li>
 <li class=MsoNormal style='color:black'>Hydrolysis of organic chemicals can be
     sensitive to pH; </li>
 <li class=MsoNormal style='color:black'>Calcite precipitation is predicted as
     a function of pH; </li>
 <li class=MsoNormal style='color:black'>Conversion of refractory detritus to
     labile detritus is affected by pH; and </li>
 <li class=MsoNormal style='color:black'>Decomposition of organic matter is
     affected;</li>
</ul>
<br>

<div>

<p class=MsoNormal><span style='color:black'>A user may input a time series of
pH values here or calculate pH values using a simple semi-empirical formulation
that requires a Mean Total Alkalinity input on this screen. (See section 5.7 of
the technical documentation (U.S. Environmental Protection Agency. 2018) for
more information.)</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>When a time series is utilized,
the rules for dynamic loadings&nbsp;apply.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The pH state variable cannot be
deleted from an AQUATOX simulation.&nbsp; </span></p>

</div>

<h5><a name=Volume><span style='color:black'>Water Volume Data</span></a></h5>

<div>

<p class=MsoNormal><span style='color:black'>To access this screen,
double-click on Water Volume on the state variables list in the main window.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Considerable flexibility exists to
compute or specify water volume.&nbsp; Depending on the method chosen, inflow
or discharge values might be required.&nbsp; The Mannings equation can be used
to compute changing volumes in a stream.&nbsp; The simplest procedure is to
hold volume constant at the initial condition.&nbsp; Volume can also be
computed dynamically using both inflow and discharge, which&nbsp;are input on
this screen. Evaporation can also affect water volumesthis rate can be input
as an annual evaporation rate on the site parameters screen or using a
time-series import available on the site-type screen. When available, a known
time series can&nbsp;be entered or imported.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><b><span style='color:black'>Water Volume Modeling Notes:</span></b></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The initial condition water volume
is determined from the inputs on this screen and not from the&nbsp;</span>site parameters<span
style='color:black'>.&nbsp; However, a &quot;<b>Get Initial Cond.&nbsp;from
Site Data</b>&quot; button exists to allow the initial condition to be copied
from that screen.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Mannings Equation Method </b>(streams
only) requires discharge data.&nbsp; Inflow data and site volume are calculated
using Manning's Equation.&nbsp; Careful attention should be given to the
&quot;Channel Slope&quot; and &quot;Manning's Coefficient&quot; parameters
entered in the &quot;Stream Data&quot; screen (within the site parameters
screen.)</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Keep Constant at Initial
Condition Level </b>method requires inflow data.&nbsp; Discharge is calculated
based on inflow and evaporation.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>If you choose to <b>Calculate
Dynamically, </b>volume is calculated based on inflow, outflow and
evaporation.&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Utilize Known Values Method
</b>requires a time series of known volumes and inflow data.&nbsp; Outflow is
calculated taking evaporation into account.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The differential equation that
calculates the water volume of the system is:</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='color:black'>&#916;Volume / &#916; t&nbsp; =&nbsp; Inflow - Outflow -
Evaporation</span></b></p>

<p class=MsoNormal align=center style='text-align:center'><b><span
style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal><span style='color:black'>where:</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='color:black'>Volume =
site volume in cubic meters,</span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='color:black'>t = time
in days, and </span></p>

<p class=MsoNormal style='text-indent:.25in'><span style='color:black'>Inflow,
Outflow, and Evaporation = fluxes in cubic meters per day.</span></p>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h4><a name=Parameters></a><a name="_Toc124172101"><span style='font-size:11.0pt'>Parameters</span></a></h4>

<div>

<p class=MsoNormal><span style='color:black'>Parameters provide values for
coefficients in the process equations.&nbsp; Although default values are given,
the user has great flexibility in specifying values to represent site-specific
species or groups.&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>There are five <b>databases</b> of
parameter values that may be loaded into a simulation. These databases can be
reached by&nbsp;clicking on the Sites, Animals, Remin., Chems, and Plants
buttons available at the upper right of the main AQUATOX interface.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Site Database </b>contains
parameters for representative sites that could be modeled by AQUATOX.&nbsp;
Within a simulation, the site parameters can be found by clicking the <b>Site </b>button
and then by clicking on the <b>Parameters </b>button that appears.&nbsp; See
also </span><a href="#SiteScreen">Site Data Screen</a><b><span
style='color:black'>.</span></b></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Animal Database</b>
contains parameters for fish and invertebrates that could be associated with a
simulation.&nbsp; Within a simulation, animal parameters may be found by
double-clicking an animal on the state-variable list and then choosing <b>Parameters</b>.
See also: </span><a href="#AnimalDataScreen">Animal Data Screen</a><b><span
style='color:black'>.</span></b></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <b>Remineralization</b> Database
contains parameters about the detritus and nutrients associated with a
site.&nbsp; To find these parameters within a simulation click on the <b>Site </b>button
and then by clicking on the <b>Remineralization </b>button that appears.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Chemical Database</b>
contains parameters for organic chemicals that could be associated with a
simulation.&nbsp; Within a simulation, chemical parameters may be found by
double-clicking a <b>Dissolved org. toxicant</b> on the state-variable list and
then choosing <b>Parameters</b>.&nbsp; See also: </span><a href="#ChemData">Chemical
Data Screen</a><b><span style='color:black'>.</span></b></p>

<p class=MsoNormal><b><span style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal><span style='color:black'>The <b>Plant Database</b> contains
parameters for algae and macrophytes that could be added to a simulation.&nbsp;
Within a simulation, plant parameters may be found by double-clicking a plant
on the state-variable list and then choosing <b>Parameters</b>. See also </span><a
href="#PlantData">Plant Data Screen.</a></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>To add a new database entry, add a
new row to the bottom of the matrix.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>When you are editing the parameters
that apply to a simulation, you can load those parameters from the library file
by clicking on the <b>Load </b>buttons at the top right of the screen.</span></p>

</div>

<h4><a name=ChemData></a><a name="_Toc124172102"><span style='font-size:11.0pt'>Chemical
Properties and Fate Data Screen</span></a></h4>

<div>

<p class=MsoNormal><span style='color:black'>This screen can be accessed by
clicking on the Chemical button on the main window or double-click on Dissolved
org. tox. on the state variables list in the main window.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Required chemical parameters and
units can be found on this screen.&nbsp; These parameters govern chemical fate
and partitioning&nbsp;behavior.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Note that&nbsp;the
organic-sediment/detritus and water partition coefficients can be calculated
dynamically or entered manually by the user.&nbsp; Sorption to inorganic
sediments is relevant only when the multi-layer sediment model is incorporated.
</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Parameters on this screen govern
the chemical fate processes of </span></p>

</div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>ionization, </li>
 <li class=MsoNormal style='color:black'>volatilization, </li>
 <li class=MsoNormal style='color:black'>hydrolysis, </li>
 <li class=MsoNormal style='color:black'>photolysis, </li>
 <li class=MsoNormal style='color:black'>sorption, and</li>
 <li class=MsoNormal style='color:black'>microbial degradation</li>
</ul>
<br>

<div>

<p class=MsoNormal><span style='color:black'>A few of the parameters on this
screen are grayed out as they are not currently utilized by the model (e.g.,
Solubility and Vapor Pressure). These parameters have not been completely
suppressed so that data in the database are not lost and in case these
parameters become useful in the future.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>You may also edit the toxicity
data for the relevant chemical by selecting the Toxicity Data button at the top
right of the screen.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>To maximize comprehension,
parameters on this user interface screen are described with several English
words rather than symbolically.&nbsp; <strong><span style='font-family:"Arial",sans-serif'>Appendix
B</span></strong> of the <strong><span style='font-family:"Arial",sans-serif'>AQUATOX
Technical documentation </span></strong><strong><span style='font-family:"Arial",sans-serif;
font-weight:normal'>(U.S. Environmental Protection Agency. 2018)</span></strong><b>
</b>contains<b> </b>a full description of each of the parameters shown here
under <strong><span style='font-family:"Arial",sans-serif'>&quot;Chemical Parameters&quot;</span></strong>
as well as their manner of referral in the equations of the technical
documentation (often a shorter variable name).&nbsp; In this way, a user can
use this appendix as a reference to search the technical documentation and find
all equations and related parameters.&nbsp; Advanced users can also easily find
the parameters within the AQUATOX source code as the
&quot;internal&quot;&nbsp;variable names are also listed within Appendix B.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>For more information on modeling
organic chemicals, see chapter 8 of the technical documentation (U.S.
Environmental Protection Agency. 2018).&nbsp;</span></p>

</div>

<h4 style='page-break-after:avoid'><a name="_Toc124172103"></a><a name=ChemTox></a><span
style='font-size:11.0pt'>Chemical Bioaccumulation and Toxicity</span></h4>

<p class=MsoNormal><span style='color:black'>This screen is where all of the
chemical toxicity parameters are located.&nbsp; To get to this screen, use the <b>Toxicity
and Bioaccumulation</b> button on the chemical's initial conditions and
loadings screen.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>There are multiple options for
entering uptake rate constant (k1), the elimination rate constant (k2) and the
bioconcentration factor (BCF) or allowing the model to calculate these parameters
(BCF=k1/k2). The radio buttons at the bottom of the animal and plant toxicity
boxes allow the user to select which uptake and elimination model they wish to
use. <br>
<br>
</span></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span style='color:black'>Enter K2, Calc. K1 and BCF:</span></b><span
style='color:black'> This is the default model formulation. The K2
elimination rate must be entered on the matrix, but K1 and BCF will be
calculated using the models equations.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span style='color:black'>Calc. BCF</span></b><span
style='color:black'> (from K1 and K2): K1 and K2 must be entered by the user
and the BCF will be estimated.</span></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span style='color:black'>Calc. K2</span></b><span
style='color:black'> (from K1 and BCF): K1 and BCF must be entered by the user
and the elimination rate (K2) will be estimated.</span></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'><span
style='font-family:Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b><span style='color:black'>Calc. K1</span></b><span
style='color:black'> (from K2 and BCF): K2 and BCF must be entered by the user
and the uptake rate (K1) will be estimated.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>If the user has toxicity data for only
a few species, an extensive&nbsp;library of regressions (Interspecies
Correlation Estimation, or Web-ICE) may be accessed at </span><a
href="https://www3.epa.gov/webice/">https://www3.epa.gov/webice/</a><span
style='color:black'>. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The animal toxicity parameters are
as follows:</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Animal Name</span></b><span
style='color:black'>, must match the &quot;toxicity data&quot; record in the
relevant Animal Data screen.</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>LC50</span></b><span
style='color:black'>, (g/L), external concentration of toxicant at which 50%
of population is killed </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>LC50 exp.
time</span></b><span style='color:black'>, (h), exposure time in toxicity
determination&nbsp; </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><span style='color:black'>[Entered]<b> K2
Elim. Rate Const.</b>, (1/d), elimination rate constant</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><span style='color:black'>[Entered] <strong><span
style='font-family:"Arial",sans-serif'>K1 Uptake Const. </span></strong>(L/kg-d)&nbsp;optional
uptake rate constant (dry weight)</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><span style='color:black'>[Entered] <strong><span
style='font-family:"Arial",sans-serif'>BCF, </span></strong>(L/kg) optional
Bioconcentration Factor (dry weight) </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Bio_rate_const</span></b><span
style='color:black'>, (1/d), Daily rate of biotransformation of this toxicant</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50 growth</span></b><span
style='color:black'>, (g/L), external concentration of toxicant at which there
is a 50% reduction in growth </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Growth exp.
time</span></b><span style='color:black'>, (h), exposure time in toxicity
determination&nbsp; </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50 repro.</span></b><span
style='color:black'>, (g/L), external concentration of toxicant at which there
is a 50% reduction in reproduction </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Repro exp.
time</span></b><span style='color:black'>, (h), exposure time in toxicity
determination&nbsp; </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Mean wet wt.</span></b><span
style='color:black'>, (g), mean wet weight of organism</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Lipid Frac</span></b><span
style='color:black'>, (g lipid/g organism), fraction of lipid in organism (wet
weight)</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Drift Threshold</span></b><span
style='color:black'>, (g/L), concentration at which invertebrate drift is
initiated</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>LC50 Slope, </span></b><span
style='color:black'>(species-specific slope at LC50 multiplied by LC50), LC50
slope factor for the external toxicity model. (Optional: if left blank or set
to zero, the value from the chemicals underlying data is used.)</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50 Growth
Slope </span></b><span style='color:black'>(species-specific slope at EC50
multiplied by EC50)<b> </b>EC50 slope factor for growth effects. (Optional: if
left blank or set to zero, the value from the chemicals underlying data is
used.) </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50 Repro
Slope </span></b><span style='color:black'>(species-specific slope at EC50
multiplied by EC50)<b> </b>EC50 slope factor for reproduction effects. (Optional:
if left blank or set to zero, the value from the chemicals underlying data is
used.)</span></p>

<p class=MsoNormal><span style='color:black'>Note: there are several comment
fields provided for documentation of parameter source citation or other notes.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The plant toxicity parameters are
as follows</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Plant Name</span></b><span
style='color:black'>, must match the &quot;toxicity data&quot; record in the
relevant Plant Data screen.</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50 photo</span></b><span
style='color:black'>, (g/L), external concentration of toxicant at which there
is 50% reduction in photosynthesis </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50 exp.
time</span></b><span style='color:black'>, (h), exposure time in toxicity
determination </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>EC50
dislodge</span></b><span style='color:black'>, (g/L), external concentration
of toxicant at which 50% of algae is dislodged (not applicable for
phytoplankton)</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>K2 Elim.
Rate Const</span></b><span style='color:black'>, (1/d), elimination rate
constant</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><strong><span style='font-family:"Arial",sans-serif;
color:black'>K1 Uptake Const. </span></strong><span style='color:black'>(L/kg-d)&nbsp;optional
uptake rate constant (dry weight)</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><span style='color:black'>[Entered]<b> <strong><span
style='font-family:"Arial",sans-serif'>BCF, </span></strong></b>(L/kg) optional
Bioconcentration Factor (dry weight) </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Biotrnsfm.
rate const.</span></b><span style='color:black'>, (1/d), Biotransformation Rate
</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>LC50</span></b><span
style='color:black'>, (g/L), external concentration of toxicant at which 50%
of population is killed </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>LC50 exp
time</span></b><span style='color:black'>, (h), exposure time in toxicity
determination </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>Lipid frac.</span></b><span
style='color:black'>, (g lipid/g organism), fraction of lipid in organism (wet
weight)</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:6.0pt;
margin-left:.75in;text-indent:-.25in'><b><span style='color:black'>LC50 Slope, </span></b><span
style='color:black'>(species-specific slope at LC50 multiplied by LC50), LC50
slope factor for the external toxicity model. (Optional: If left blank or set
to zero, the value from the chemicals underlying data is used.)</span></p>

<p class=MsoNormal style='margin-left:.75in;text-indent:-.25in'><b><span
style='color:black'>EC50 Photo Slope </span></b><span style='color:black'>(species-specific
slope at EC50 multiplied by EC50)<b> </b>EC50 slope factor for photosynthesis
effects. (Optional: if left blank or set to zero, the value from the
chemicals underlying data is used.) </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<h4 style='page-break-after:avoid'><a name="_Toc124172104"></a><a
name=PlantData></a><span style='font-size:11.0pt'>Plant Data Screen</span></h4>

<p class=MsoNormal>To access this screen, double-click on a plant name in the state
variables list in the main window and choose Parameters or by accessing the
plant library.</p>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Several&nbsp;fields near the top
of the screen require explanation.&nbsp; If you click on the drop-down arrow to
the right of <b>Plant type</b>, you will be given a choice. The choice of <b>Plant
type </b>is important because different types have different physical or
biological processes that apply to them.&nbsp; For instance, phytoplankton are
subject to sinking, but not periphyton, which are attached to a surface.&nbsp;
Conversely periphyton are limited somewhat by very slow current velocity; but
not phytoplankton, which are adapted to still water. If macrophytes are
chosen the species must be distinguished as benthic, rooted floating, or
free-floating. Table 5 in the Technical documentation (U.S. Environmental
Protection Agency. 2018) helps clarify some of the differences between plant
types. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>If a plant is surface floating,
it is assumed reside in the top 0.1 m of the system unless wind distributes it
to the top 3 m based on Langmuir circulation. The averaging depth for surface
floating plants, when calculating plant concentrations (mg/L), is assumed to be
the top three meters to more closely correspond to monitoring data.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>A user must consciously choose to
model internal nutrients by checking that option in the Setup screen. However,
to ease the transition, six parameters that are unique to modeling internal
nutrients are populated with default values taken from the documentation of
WASP7 (Ambrose et al. 2006). </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The effects of modeling internal
nutrients were investigated for all studies supplied as examples with the
AQUATOX installation. As might be expected, the nutrient-poor sites exhibited
the greatest response to luxury uptake of nutrients, but default parameters
cannot be used as-is and require the most recalibration. Where there were
sufficient observed data to evaluate the results, the applicability of the
internal-nutrient submodel is noted. In general, the following sites benefited
from modeling internal nutrients: Cahaba River AL, Evers Reservoir FL, Lake
Onondaga NY, Tenkiller Reservoir OK, and Rum River MN. The following sites
showed a tendency to over-predict algal biomass (and dissolved oxygen): Blue
Earth River MN, Crow Wing River MN, DeGray Reservoir AR, and Lake Jesup
FL. <br>
<br>
</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Although <b>Plant type</b> is
important for determining which processes do or do not apply to the state
variable, the <b>Taxonomic Type</b> field is included as an organizing tool and
does not affect model output.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <b>Toxicity Record</b> field
within this screen links to the &quot;plant name&quot; within each chemical's
toxicity data.&nbsp; In this manner, several plants could link to the same
toxicity record if that is desired.&nbsp; You may select a record from the list
or type a new name if the desired plant name does not appear in the drop-down list.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Phytoplankton and periphyton
compartments may be linked together.&nbsp; When viewed in a simulation, a plant
will have an &quot;<b>Edit All Plant Linkages</b>&quot; button available at the
top of the screen.&nbsp; Periphyton also have a &quot;<b>Periphyton Linkage</b>&quot;
button that allows you to edit the same information but only for the species
shown.&nbsp; It is considerably more powerful and user-friendly to use the
&quot;Edit All&quot; button. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>A few notes regarding the some of
the most&nbsp;important parameters: </span></p>

<div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Photosynthesis and growth rates are
     quite sensitive to the saturating light and&nbsp;P half-saturation
     coefficients along with optimum temperature and the maximum photosynthetic
     rate.</li>
</ul>

</div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>The phytoplankton mortality
     coefficient can be adjusted for a particular site.</li>
</ul>

</div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>The exponential mortality coefficient
     (which increases the mortality for suboptimal conditions) might need to be
     adjusted if blooms crash too quickly or not quickly enough. </li>
</ul>

</div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Occasionally the light extinction
     coefficient may need to be increased if algal growth is too strongthis
     coefficient may be the principal means of negative feedback, and can vary
     among algal groups.</li><br>
 <li class=MsoNormal style='color:black'>The sedimentation rate (phytoplankton
     only) should be small for streams and much larger for lakes.</li><br>
 <li class=MsoNormal style='color:black'>The Critical Force (FCrit. for periphyton
     and macrophytes only) for scour&nbsp;is quite important for periphyton.</li><br>
 <li class=MsoNormal style='color:black'>Salinity effects are relevant only if
     salinity is included within the simulation in which case photosynthesis
     and mortality may be affected. </li>
</ul>
<br>

</div>

</div>

</div>

<div>

<p class=MsoNormal><span style='color:black'>To maximize comprehension,
parameters on this user interface screen are described with several English
words rather than symbolically.&nbsp; <strong><span style='font-family:"Arial",sans-serif'>Appendix
B</span></strong> of the <strong><span style='font-family:"Arial",sans-serif'>AQUATOX
Technical documentation</span></strong> (</span>U.S. Environmental Protection
Agency. 2018) <span style='color:black'>contains a full description of each of
the parameters shown here under <strong><span style='font-family:"Arial",sans-serif'>&quot;Plant
Parameters&quot;</span></strong> as well as their manner of referral in the
equations of the technical documentation (often a shorter variable name).&nbsp;
In this way, a user can use this appendix as a reference to search the technical
documentation and find all equations in which each parameter is utilized.&nbsp;
Advanced users can also easily find the parameters within the AQUATOX source
code as the &quot;internal&quot;&nbsp;variable names are also listed within
Appendix B.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>See section 4.1 and 4.2 of the
technical documentation for extensive discussion of modeling algae and
macrophytes.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h4 style='page-break-after:avoid'><a name=AnimalDataScreen></a><a
name="_Toc124172105"><span style='font-size:11.0pt'>Animal Data Screen</span></a></h4>

<div>

<p class=MsoNormal>This screen displays all of the relevant parameters for the
animal that has been selected. To access this screen, double-click on an
animal name in the state variables list in the main window and choose Parameters.<br>
<br>
</p>

</div>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>The
<b>Toxicity Record</b> field within this screen links to the &quot;animal
name&quot; within each chemical's toxicity data.&nbsp; In this manner, several
animals could link to the same toxicity record if that is desired.&nbsp; You
may select a record from the list or type a new name if the desired animal name
does not appear in the pull-down list. &nbsp;To edit toxicity linkages for all
plants and animals in a simulation simultaneously, click the &quot;<strong><span
style='font-family:"Arial",sans-serif'>Edit All</span></strong>&quot; button.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>AQUATOX&nbsp;can model two size
classes for each fish species.&nbsp; Records for different size classes are
linked by clicking on <b>Size-Class Links (</b>under the Animal Name<b> </b>towards
the top of the screen)<b> </b>and choosing the correct record from the list
given. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>As was the case in the Plant Data
Screen, the choice of <b>Animal type </b>is important because different animal types
have different physical or biological processes that apply to them.&nbsp; For
instance, benthic insects are subject to&nbsp;emergence but other animal types
are not. Table 6 in the technical documentation (U.S. Environmental Protection
Agency. 2018) describes the differences between animal types. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>For &quot;Benthic
Invertebrates,&quot; a benthic designation box is available. This box does not
affect model execution but has effects on the calculation of Biological Metrics.
(See section 4.7 of the technical documentation (U.S. Environmental Protection
Agency. 2018).)</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Although <b>Animal type</b> is
important for determining which processes do or do not apply to the state
variable, the <b>Taxonomic Type or Guild</b> field is included as an organizing
tool and does not affect model output.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Sensitivity of animals to <strong><span
style='font-family:"Arial",sans-serif'>sediments </span></strong>comes into
this parameter screen in&nbsp;several places.&nbsp; The feeding effects are
found in the top of the screen with the feeding parameters.&nbsp;&nbsp;Lethal
effects and percent-embeddedness effects are found towards the bottom of the
screen with the other mortality parameters.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The <strong><span
style='font-family:"Arial",sans-serif'>fraction in water column </span></strong>field
is relevant only to models running the multi-layer sediment model and
determines how much of the organism is exposed to the water column as opposed
to pore waters in the top layer of sediment.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><strong><span style='font-family:"Arial",sans-serif;
color:black'>Bioaccumulation Data </span></strong><span style='color:black'>includes
the lifespan and fraction lipid.&nbsp; Uptake and maximum bioaccumulation of
organic chemicals are sensitive to these parameters.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><strong><span style='font-family:"Arial",sans-serif;
color:black'>Low Oxygen Effects </span></strong><span style='color:black'>parameters
are available for lethality and growth and reproduction effects.&nbsp;&nbsp;<strong><span
style='font-family:"Arial",sans-serif'>Ammonia </span></strong>toxicity
parameters follow below that followed by <strong><span style='font-family:"Arial",sans-serif'>Salinity
</span></strong>effects. (These parameters are not on the chemical toxicity
screen since they are not organic-toxicant specific.)</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>If an organism resides in a
stream, the habitat-type within the stream should be specified using the <b>percent
riffle, percent pool, </b>and<b> percent run</b> parameters. This affects the water
velocity the organism is exposed to and also potentially excludes the organism
from some water segments on the basis of habitat availability.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Fish may spawn automatically using
a formulation based on the <strong><span style='font-family:"Arial",sans-serif'>optimal
temperature </span></strong>parameter or specific spawning dates may be entered
under <strong><span style='font-family:"Arial",sans-serif'>spawning parameters.</span></strong></span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>For fish, maximum consumption and endogenous
(basal) respiration rates can be directly entered into the model (top of
screen) or <strong><span style='font-family:"Arial",sans-serif'>allometric
models</span></strong> can be utilized (bottom of screen).&nbsp; Allometric
models within AQUATOX can calculate consumption and respiration rates
as&nbsp;functions of weight and temperature.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>A few notes regarding the some of
the more&nbsp;important parameters: </span></p>

</div>

<div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Biomass predictions are sensitive to
     &quot;Maximum Consumption&quot; rate and&nbsp;&quot;Endogenous
     Respiration&quot;&nbsp;rates when not calculated based on weight
     (allometics).</li>
</ul>

</div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>The minimum prey for feeding affects
     the efficiency of foraging behavior.</li>
 <li class=MsoNormal style='color:black'>The optimum temperature can have a
     significant effect on biomass computations;</li>
</ul>

</div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Intrinsic mortality rate (mortality
     coefficient) is often a site-specific response and is therefore subject
     to calibration.</li>
</ul>

</div>

<div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Allometric means the&nbsp;change in
     metabolic rate in relation to the size of the organism; allometric
     parameter values can be found in the Wisconsin Bioenergetics Model
     documentation (Hewett and Johnson, 1992; Hanson et al., 1997).&nbsp; </li>
</ul>

</div>

</div>

<p><span style='color:black'>To maximize comprehension, parameters on this user
interface screen are described with English words rather than
symbolically.&nbsp; <strong><span style='font-family:"Arial",sans-serif'>Appendix
B</span></strong> of the <strong><span style='font-family:"Arial",sans-serif'>AQUATOX
Technical documentation</span></strong> (</span>U.S. Environmental Protection
Agency. 2018) <span style='color:black'>contains a full description of each of
the parameters shown here under <strong><span style='font-family:"Arial",sans-serif'>&quot;Animal
Parameters&quot;</span></strong> as well as their manner of referral in the
equations of the technical documentation (often a shorter variable name).&nbsp;
In this way, a user can use this appendix as a reference to search the
technical documentation and find all equations in which each parameter is
utilized.&nbsp; Advanced users can also easily find the parameters within the
AQUATOX source code as the &quot;internal&quot;&nbsp;variable names are also
listed within Appendix B.</span></p>

<p><span style='color:black'>See section 4.3 of the technical documentation for
extensive discussion of modeling animals within AQUATOX.</span></p>

<h5 style='page-break-after:avoid'><a name=TrophicMatrix><span
style='color:black'>Trophic Interaction Matrix Screen:</span></a></h5>

<h5 style='page-break-after:avoid'><span style='color:black;font-weight:normal'>This
screen is accessed from the Food Web button on the Main screen.</span></h5>

<p><span style='color:black'>The matrix enables the user to see all trophic
interactions for all animals in a particular species against all prey
items.&nbsp; Generally, this is a more useful interface for examining and
entering information about the food web.&nbsp; Preferences are normalized to
100% when the model is executed.</span></p>

<h3 style='page-break-after:avoid'><a name="_Toc124172106"><span
style='font-size:12.0pt'>Site Information</span></a></h3>

<p class=MsoNormal>This screen displays all of the relevant parameters for the site.
To access this screen, click on the Site button on the main window.</p>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>At
the upper left is a panel that allows a user to select which site type is being
simulated.&nbsp;&nbsp;Six site types are available: pond, lake, stream (creek
or river), reservoir, and (experimental) enclosure, and marine.&nbsp;&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>For&nbsp;standing water (ponds,
lakes, and reservoirs), site type is not currently a sensitive parameter (i.e.,
these site-types can be used interchangeably). </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Selecting to model a
&quot;stream&quot; has the following effects:</span></p>

</div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>Manning's coefficient may be used to
     model water volume.</li>
 <li class=MsoNormal style='color:black'>Periphyton scour and velocity
     limitation for plants is a function of water velocity.</li>
</ul>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Additionally, a user may edit a <b>site's
parameters</b> from this screen.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Parameters that may be edited on
this screen in a time series or constant manner are accessed by the pull-down
menu labeled Site Time Series at the center top of this screen.</span></p>

</div>

<ul type=disc>
 <li class=MsoNormal style='color:black'>&quot;Water Velocity.&quot;&nbsp; See <span
     style='color:windowtext'><a href="#Velocity">velocity</a></span><strong><span
     style='font-family:"Arial",sans-serif'>.</span></strong> </li>
 <li class=MsoNormal style='color:black'>&quot;Mean Depth.&quot;&nbsp; See <span
     style='color:windowtext'><a href="#MeanDepth">mean depth</a></span>.</li>
 <li class=MsoNormal style='color:black'><b>Evaporation of water</b> in cubic
     meters per day. If this time-series option is selected, then the constant
     evaporation data in the site parameter record is not relevant.</li>
 <li class=MsoNormal style='color:black'>&quot;Fraction of site that is
     shaded.&quot;&nbsp;&nbsp; See <span style='color:windowtext'><a
     href="#Shade">modeling shade</a></span>. </li>
</ul>

<h4 style='page-break-after:avoid'><a name=Site></a><a name="_Toc124172107"></a><a
name=SiteScreen></a><span style='font-size:11.0pt'>Site Parameters</span></h4>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>Each
site can be characterized by a relatively small number of site constants.&nbsp;
These can be seen and edited by clicking on <b>Parameters</b> in the <b>Site
Data</b> window, or they can be loaded from the <b>Database</b>.&nbsp; There is
some redundancy in that <b>Volume</b>, <b>Area</b>, and <b>Mean Depth</b> all
have to be specified.&nbsp; Based on mean and maximum depth, the bathymetry of
the site is computed (see equation 8 in the Technical Documentation, </span>U.S.
Environmental Protection Agency. 2018<span style='color:black'>).&nbsp; Volume
is a state variable and can be computed in a variety of ways (accessible
through the <strong><span style='font-family:"Arial",sans-serif'>volume loading
screen</span></strong>); however, one option is to set it to&nbsp;remain
constant using the value provided in the site screen.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <b>Max. Length</b> is the
distance, usually the long axis, across which wave buildup can occur.&nbsp; This
can also affect phytoplankton retention time in flowing systems as well as the
calculation of cross-section area for velocity calculations. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Likewise, in the <b>Light Screen</b>,
if a user has selected to &quot;use annual mean and range
loadings,&quot;&nbsp;that user must provide data about the <b>Average Light</b>
and the <b>Average Light Range </b>on the Site Data screen<b>, </b>from which
seasonal fluctuations are computed.&nbsp; These are not computed from the
latitude because of local and regional differences in elevation, cloud cover,
and maritime or continental climatic conditions.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><b><span style='color:black'>Latitude</span></b><span
style='color:black'> is used to compute the seasonal variation in day length, although
this can be overridden in the <b>Light Screen</b>.&nbsp; This override can be
useful for laboratory simulations.&nbsp; </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='color:black'>Altitude</span></b><span
style='color:black'> is used in the computation of oxygen saturation. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='color:black'>Enclosure wall area</span></b><span
style='color:black'> is used for experimental enclosures only and affects a
sites morphometry. (Specifically, it increases the fraction of the site area
within the euphotic zone.)</span></p>

<p class=MsoNormal><b><span style='color:black'>&nbsp;</span></b></p>

<p class=MsoNormal><b><span style='color:black'>Baseline Percent Embeddedness</span></b><span
style='color:black'> is the initial condition percent embeddedness for a site.
This value is used to calculate effects when organisms are sensitive to embeddedness,
calculated as a function of TSS. See the Interstitial Sediments portion of
section 4.3 of the Technical Documentation (</span>U.S. Environmental
Protection Agency. 2018)<span style='color:black'>. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <b>Minimum Volume Frac.</b>
multiplied by the initial condition volume for a site represents the minimum
volume that a water body can attain within the simulation. If the water volume
drops below this level, numerical modeling of state variables stops, and the
simulation skips forward to the next time period when the water volume is
calculated to be above the minimum level.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<div>

<p class=MsoNormal><span style='color:black'>Toward the bottom of the site
screen are the parameters for the <strong><span style='font-family:"Arial",sans-serif'>phytoplankton
retention function</span></strong>.&nbsp; Phytoplankton and zooplankton will
quickly wash out of a short reach, but might be able to grow over an extensive
reach of a river, including its tributaries.&nbsp; To solve this problem,
AQUATOX takes into account the Total Length of the river being simulated, as
opposed to the length of the river reach, or Site Length so that
phytoplankton and zooplankton production upstream can be estimated.&nbsp; The
assumption is that conditions upstream are similar to those in the reach being
modeled. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>For estuaries, a series of <strong><span
style='font-family:"Arial",sans-serif'>tidal range parameters </span></strong>is
required (amplitudes and epochs).&nbsp; These&nbsp;are used&nbsp;to calculate
the daily average tidal range for the site, which affects the average depth of
the salt wedge on that day.&nbsp;&nbsp; These parameters are generally
available for download from National Oceanic and Atmospheric Administration (NOAA)
databases.</span></p>

</div>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>If a stream is being simulated
(set in the<b> Site Type</b> panel on Site screen) the <b>Stream Data</b>
button in the upper right is enabled.&nbsp; Clicking this button displays a
series of important stream parameters regarding site morphometry, and habitats
represented.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>To maximize comprehension,
parameters on this user interface screen are described with several English
words rather than symbolically.&nbsp; <strong><span style='font-family:"Arial",sans-serif'>Appendix
B</span></strong> of the <strong><span style='font-family:"Arial",sans-serif'>AQUATOX
Technical documentation</span></strong> contains a full description of each of
the parameters shown here under <strong><span style='font-family:"Arial",sans-serif'>&quot;Site
Parameters&quot;</span></strong> as well as their manner of referral in the
equations of the technical documentation (often a shorter variable name).&nbsp;
In this way, a user can use this appendix as a reference to search the
technical documentation and find all equations in which each parameter is
utilized.&nbsp; Advanced users can also easily find the parameters within the
AQUATOX source code as the &quot;internal&quot;&nbsp;variable names are also
listed within Appendix B.</span></p>

</div>

<h4 style='page-break-after:avoid'><a name=Remineralization></a><a
name="_Toc124172108"><span style='font-size:11.0pt'>Remineralization</span></a></h4>

<p class=MsoNormal>The parameters on the remineralization screen primarily
govern the settling and degradation of organic matter and its nutrient
stoichiometry. These parameters are <span style='color:black'>accessed from
the Remineralization button at the lower left of the screen.</span></p>

<p>Many of the parameters on the remineralization screen can be assumed to
remain constant from one simulation to another.&nbsp; A few highlights may be
worth examination because they can both be site-specific and can affect some
types of model results:</p>

<ul type=disc>
 <li class=MsoNormal>Maximum degradation rates:&nbsp; These rates should be
     modified if site-specific data are available.</li>
 <li class=MsoNormal>Detrital Sedimentation Rate (KSed):&nbsp; This rate governs
     the deposition of organic matter and phytoplankton from the water column
     to sediment (unless the sand-silt-clay model is utilized).</li>
</ul>

<p class=MsoNormal>Because of these differences, there are multiple Remineralization
records included in the model:</p>

<ul type=disc>
 <li class=MsoNormal>Eutrophic -- should be used in highly nutrient-impacted
     high systems; </li>
 <li class=MsoNormal>Default  same as Eutrophic record; </li>
 <li class=MsoNormal>Mesotrophic -- should be used in moderately
     nutrient-impacted systems; and</li>
 <li class=MsoNormal>Florida -- used for Florida lakes; this record is
     different because of the stoichiometry of labile detritus observed in
     Florida.</li>
</ul>

<p class=MsoNormal>If your site has very different characteristics governing
remineralization, you can edit one of the existing records or create your own.<span
style='font-size:11.0pt'> </span></p>

<h4 style='page-break-after:avoid'><a name=Shade></a><a name="_Toc124172109"><span
style='font-size:11.0pt'>Modeling Shade</span></a></h4>

<p class=MsoNormal style='page-break-after:avoid'>Shade can be an important
limitation to light, especially in riparian systems.&nbsp; A user input <b>Fraction
of site that is Shaded (canopy covered)</b> parameter can be entered either as
a constant or as a time-series within the Site input screen.&nbsp; AQUATOX
assumes that 2% of incident radiation penetrates the canopy so a site that has
100% canopy cover still receives 2% light transmission. See section 3.6 of the Technical
documentation (U.S. Environmental Protection Agency. 2018) for&nbsp;more
information on this assumption, the equations utilized, and modeling light in
general.</p>

<p class=MsoNormal>This &quot;fraction shaded&quot; parameter can be left as
zero to simulate no shading effects on light.</p>

<h4 style='page-break-after:avoid'><a name=Velocity></a><a name="_Toc124172110"><span
style='font-size:11.0pt'>Velocity</span></a></h4>

<p>Velocity in cm/s&nbsp;may be calculated by AQUATOX or entered as a time
series.&nbsp; Otherwise, velocity is calculated as a simple function of flow
and cross-sectional area. </p>

<p>Velocity has wide reaching effects including: </p>

<ul type=disc>
 <li class=MsoNormal>deposition of phytoplankton and&nbsp;scour of periphyton; </li>
 <li class=MsoNormal>breakage of macrophytes; </li>
 <li class=MsoNormal>entrainment of zooplankton; </li>
 <li class=MsoNormal>deposition or scour of organic matter; </li>
 <li class=MsoNormal>scour and deposition in the sand-silt-clay model; and </li>
 <li class=MsoNormal>oxygen reaeration. </li>
</ul>

<h4 style='page-break-after:avoid'><a name=MeanDepth></a><a name="_Toc124172111"><span
style='font-size:11.0pt'>Mean Depth</span></a></h4>

<p style='page-break-after:avoid'>Mean depth may be kept constant or entered as
a time series. &nbsp; Mean depth has significant effects on the average light
climate that plants are exposed to (especially periphyton, which reside on the
bottom of the water column.)&nbsp;</p>

<p>As noted on the entry screen, if &quot;Use Bathymetry&quot; is <strong><span
style='font-family:"Arial",sans-serif'>not </span></strong>checked on the Site Parameters
screen, the mean depth calculation is irrelevant.&nbsp; In this case, mean
depth is volume divided by surface area.&nbsp;</p>

<h3 style='page-break-after:avoid'><a name=SetupWindow></a><a
name="_Toc124172112"><span style='font-size:12.0pt'>Setup Parameters</span></a></h3>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>The
Simulation Setup screen specifies important parameters about the simulation
itself (as opposed to any of the components of the simulation.)&nbsp; To access
this screen, click on the <b>Setup</b> button on the main window.</span></p>

</div>

<h5><span style='color:black'>Simulation Time and Relative Error </span></h5>

<div>

<p class=MsoNormal><span style='color:black'>At the top of the Simulation Setup
screen, you can modify the <b>first and last days of the simulation</b>.&nbsp;&nbsp;&nbsp;
This defines the simulation period. </span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Default model behavior is to use a
variable step size RungeKutta method as described in section 2.1 of the technical
documentation (U.S. Environmental Protection Agency. 2018). The <b>Relative
Error</b> is the acceptable error in the simulation; if it is not achieved in a
particular time step, the variable RungeKutta routine decreases the step size
and tries again.&nbsp; If the relative error is too large, the results may be
erroneous; if it is too small, the run time may be too long. Usually a value
between 0.005 and 0.0005 is appropriate, but you might wish to experiment for a
particular application.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>A <b>fixed step size</b> may also
be selected with time step varying from one tenth to one hundredth of a day.
This may be useful when a user is precisely comparing control and perturbed
runs and so wishes to have their step-sizes be precisely aligned. More
information about this can be found in section 2.1. of the technical
documentation (U.S. Environmental Protection Agency. 2018).</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='color:black'>Output Storage Options </span></b></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>A user may set the native model
time step to one day or one hour.&nbsp; If a daily simulation is utilized,
average light conditions are&nbsp;utilized throughout the day.&nbsp; If an
hourly simulation is selected, solar radiation is calculated as variable during
the course of each day.&nbsp; Rather than calculating daily average oxygen
concentrations, if an hourly simulation is selected, AQUATOX will simulate
hourly average oxygen concentrations within the water column.&nbsp; These
concentrations will be based on the hourly light climate (and optional hourly
oxygen loadings).&nbsp; These hourly predictions will then&nbsp;be used to
calculate lethal and non-lethal effects due to low oxygen.&nbsp; The user can
output hourly values of DO or set a larger&nbsp;<strong><span style='font-family:
"Arial",sans-serif'>Data Storage Step</span></strong> and examine the minimum
and maximum predictions over that time interval. Hourly inflow loadings can be
input for all nutrients, carbon dioxide, oxygen, inorganic suspended sediments,
TSS, light, and organic matter.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The <strong><span
style='font-family:"Arial",sans-serif'>Data Storage Step</span></strong>
represents the time period over which results are averaged.&nbsp; This
obviously can have a significant effect on the amount of output that is
produced by the model.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The default method of averaging
results is by <strong><span style='font-family:"Arial",sans-serif'>Trapezoidal
Integration</span></strong>, which calculates the average value that has
occurred since the last data storage step. &nbsp;A user may also bypass
integration altogether uncheck this box (output predictions that occur exactly
at a given time-step).</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h5 style='margin:0in'><span style='color:black'>Biota Modeling Options </span></h5>

<div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Selecting the option to <b>Model
Nutrients Internally </b>will populate the model with additional state
variables that track changes to internal nutrient concentrations (N and P) in
non-rooted plants. See <i>Internal Nutrients</i> in section 4.1 of the AQUATOX
Technical documentation (U.S. Environmental Protection Agency. 2018) for more
information about this optional model. Concentrations of internal nutrients
will be output in units of &#956;g/L or the N to org and P to org ratios
may be viewed as they change over time. Some additional plant parameters are
required, see the section on the </span><a href="#PlantData">Plant Data Screen</a><span
style='color:black'> in this document.</span></p>

<p class=MsoNormal><span style='color:black'></span></p>

<p class=MsoNormal><span style='color:black'>By default, nitrogen fixation in
blue-green algae is assumed to occur when nitrogen concentrations are less than
half of the <i>KN</i> value (the half-saturation constant for nitrogen).&nbsp;
AQUATOX also provides an option to trigger nitrogen fixation as a function of
an editable inorganic N to inorganic P ratio which&nbsp;may be selected and
specified on this screen.</span></p>

</div>

</div>

<h5><span style='color:black'>Chemical Options </span></h5>

<p class=MsoNormal><span style='color:black'>This panel is accessed using the collapsible
&quot;show&quot; or &quot;hide&quot; button.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<div>

<div>

<p class=MsoNormal><span style='color:black'>If you wish to compute
steady-state Bioaccumulation Factors (BAFs), you might want to <b>keep the
freely dissolved toxicant constant</b>; this was done in an application
concerning PCBs in Lake Ontario (see Validation Studies document).&nbsp; Otherwise,
toxicant mass is balanced on the basis of loading, partitioning, washout, and
other toxicant loss terms.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The chemical concentration in
bottom-sediment organic matter can become a driving variable when <b>Tox. in
Sed. Detr. is driving var</b>. is checked. This allows the user to specify a
time-series of sediment chemical concentrations to drive the model and examine
the extent of bioaccumulation and effects on the ecosystem.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Toxic effects may be calculated
based on <strong><span style='font-family:"Arial",sans-serif'>internal or
external concentrations.&nbsp; </span></strong>See Chapter 9 of the Technical
documentation (U.S. Environmental Protection Agency. 2018) for more information
about the consequences of this choice.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>When calculating effects based on
external concentrations, internal concentrations may be less important.&nbsp;
For this reason, a user may wish to <strong><span style='font-family:"Arial",sans-serif'>estimate
toxicant uptake based on BCF</span></strong> rather than using the default
kinetic model of toxicant uptake.&nbsp; In many cases this will speed up
simulation run time significantly.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>If <b>T1 is an aggregate of all
other toxicants in study</b> is checked, then the organic chemical in
compartment T1 is used to aggregate all of the other chemicals being modeled as
discussed in section 8.14 of the Release 3.2 <i>Technical documentation (U.S.
Environmental Protection Agency. 2018).</i></span></p>

</div>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h4 style='margin:0in;page-break-after:avoid'><a name=RateOutput></a><a
name="_Toc124172113"><span style='font-size:11.0pt'>Rate Output</span></a><span
style='font-size:11.0pt'> </span></h4>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>In the Setup Screen, choose the <b>Save
Derivative Rates</b> radio button. Each element of a selected state variable's
derivative will be integrated and saved along with the results for the
simulation.&nbsp; They can then be graphed or viewed in tabular format.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Units for rates are
&quot;percent&quot; which is short for &quot;percent&nbsp;of state variable
concentration per day.&quot; The only exception is for limitations to
photosynthesis (variables ending with _LIM) which are expressed on a scale of
0.0 to 1.0 with 0.0 representing complete limitation of photosynthesis and 1.0
representing no limitation to photosynthesis.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Two light limitation outputs are
available to signal the occurrence of photoinhibition (too much light) as
opposed to insufficient light. When low-light limitation causes light
conditions to be sub-optimal then the LowLt_LIM is equal to the overall light
limitation and the &quot;high-light limitation&quot; is set to 1.0. When
photoinhibition is occurring then the HighLt_LIM is equal to the overall light
limitation and &quot;low light limitation&quot; is set to 1.0.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h3 style='margin:0in'><a name="_Toc124172114"><span style='font-size:12.0pt'>Run
Button (running the model)</span></a></h3>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>To run an AQUATOX simulation, click the &quot;Run&quot;
button in the main window. </p>

<p style='margin:0in'>The model will run for the amount of time specified in
the setup screen.&nbsp; When the model has completed you can use the output
button to view the model results.</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="_Toc124172115"><span
style='font-size:11.0pt'>Multiple Archived Simulations</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>A powerful feature of AQUATOX is that it can run paired
simulations for perturbed and control conditions.&nbsp; </p>

<p style='margin:0in'>&nbsp;</p>

<h3 style='margin:0in;page-break-after:avoid'><a name=OutputScreen></a><a
name="_Toc124172116"><span style='font-size:12.0pt;color:black'>Output Window</span></a></h3>

<p class=MsoNormal style='page-break-after:avoid'><a name="_Toc330995804">&nbsp;</a></p>

<p class=MsoNormal>The Output window is accessed from the Output button on
the <b>main window.</b></p>

<p class=MsoNormal><b>&nbsp;</b></p>

<p class=MsoNormal>At the upper left of the window, the user may select which
set of Saved<b> Simulation Results </b>that will be graphed or exported. A set
of archived results sets may be deleted by selecting them from the pull-down
box and then selecting <b>Delete. </b>A full set of archived state variable
results may also be exported to comma-separated value output using the <b>Save
CSV </b>button. The results of an individual graph can be exported using the <b>CSV
</b>button to the right of the screen.<b><br>
<br>
</b></p>

<h4 style='margin:0in;page-break-after:avoid'><a name="_Toc124172117"><span
style='font-size:11.0pt'>Viewing Existing Graphs</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>To determine which graphs have been produced for the
given archived simulation, click on the drop-down box under <b>Saved Graphs</b>.&nbsp;
If there is more than one graph included, a drop-down list of graphs will be
shown.&nbsp; The user can select a graph within this list and it will update on
the chart canvas below<u>. </u></p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>To add a new graph to the list of graphs in the graph
library select the &quot;<b>New Graph</b>&quot; button.&nbsp; To edit the
existing graph, select the &quot;<b>Edit</b>&quot; button.&nbsp; The selected
graph may also be deleted using the <b>Delete</b> button.</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Once a graph is created you may choose to <b>Draw a box
to zoom. </b>In this case, clicking and dragging a box on the graph will zoom
into a particular set of dates or values. Alternatively, <b>clicking on the
graph to show date and value</b> will display the time and value if a line on
the graph is clicked on. You may toggle a log scale at the lower right
(excluding graphs with zero values or negative values, of course).</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="New_Graph"></a><a
name="_Toc124172118"><span style='font-size:11.0pt'>Setting Up a New Graph</span></a></h4>

<div>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>When the &quot;<b>New</b>&quot; button is pressed, a user
<a name="Modifying_a_Graph">will get to the <b>Graph Setup</b> window.</a> <span
style='color:black'>By clicking on the &quot;<b>Edit</b>&quot; button above a
graph, one can also get to this screen.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>At the top of this window is the
capability to edit the graph's name.&nbsp; This affects how the graph appears
in the drop-down list.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Next, choose the output to place
on each axis.&nbsp; There are potentially hundreds of output variables
associated with each simulation.&nbsp; The filter tools help the user pick out
relevant variables from this considerable list. (See&nbsp;the topic </span><a
href="#Selecting_Output">Selecting from a List of Output</a><span
style='color:black'>.)</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>State variables are organized in
order of trophic level, starting with organic matter and working upward through
plants, invertebrates, and fish.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><b><span style='color:black'>Types of AQUATOX Output </span></b><span
style='color:black'>(in order of output list)</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Concentrations of State Variables</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>toxicants in water</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>nutrients and gasses</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>organic matter, plants, invertebrates,
fish</span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Physical Characteristic State Variables</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>water volume, temperature, wind, light,
pH</span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Mass of Toxicants within State
Variables (normalized to water volume)</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>T1-T20 in organic matter, plants,
invertebrates, and fish</span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Sediment diagenesis state variables</span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>Toxicant PPB</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>T1-T20 (PPB) in organic matter, plants,
invertebrates, and fish</span></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>State Variable Rates</span></p>

<p class=MsoNormal style='margin-left:1.0in;text-indent:-.25in'><span
style='color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>These include limitations to photosynthesis</span></p>

<p class=MsoNormal style='margin-left:.5in'><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>The user can graph variables on
either one or two&nbsp;Y axes.&nbsp; Use the button under the results list to
toggle between one and two&nbsp;Y axes.&nbsp; Use the &quot;<b>&lt;</b>&quot;
and &quot;<b>&gt;</b>&quot; buttons to move results into and out of a
particular axis's results.&nbsp; The &quot;<b>&gt;&gt;</b>&quot; button moves
all variables into the selected axis.&nbsp; Note that all variables on the same
axis should have the same units.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<h5 style='margin:0in;page-break-after:avoid'><a name="Selecting_Output"><span
style='color:black'>Selecting from a List of Output</span></a></h5>

<div>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The list of outputs is initially
populated with state variable results. These can be limited to <b>Animals</b>
and <b>Plants </b>variables by selecting those check boxes.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>When <b>Include Rates</b> is
selected, there are potentially hundreds of output variables associated with
each simulation, as individual components of each state variables derivatives
are output.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>To help the user pick out relevant
variables to be graphed, the &quot;<b>Has Substring</b>&quot; edit box is
available at the top of the screen.&nbsp; Typing a string in the box provided
limits the list of variables to those with names that include the string.&nbsp;
For example:</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Type &quot;peri&quot; to find all periphyton variables</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Type &quot;mg/L&quot; to find all variables with units of mg/L.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Type &quot;ppb&quot; to find all parts per billion outputs.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Type &quot;detr&quot; to find all results that pertain to detritus.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
Type &quot;T1&quot; to find all results that pertain to the first organic
toxicant.</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

</div>

<div>

<p class=MsoNormal><span style='color:black'>Use the &quot;<b>&lt;</b>&quot;
and &quot;<b>&gt;</b>&quot; buttons to move results into and out of the
&quot;Available Results&quot; and &quot;Results to Display&quot; columns.&nbsp;
Using &quot;<b>&lt;&lt;</b>&quot; and &quot;<b>&gt;&gt;</b>&quot; moves all
results into or out of the relevant column.</span></p>

</div>

<div>

<p class=MsoNormal>&nbsp;</p>

</div>

<h2 style='margin:0in;page-break-after:avoid'><a name="_Toc124172119"><span
style='font-size:14.0pt;color:black'>Setting Up Simulation, Single Segment</span></a></h2>

<p style='margin:0in;page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>Setting up a single segment
simulation generally has the following types of steps:</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='color:black'>Locate a surrogate simulation that can
     be modified. </li>
 <li class=MsoNormal style='color:black'>Produce a reasonable model of water
     volume for your site. </li>
 <li class=MsoNormal style='color:black'>Choose&nbsp;appropriate biotic state
     variables for your site. </li>
 <li class=MsoNormal style='color:black'>Examine the trophic interactions
     matrix to ensure that a reasonable food-web has been constructed. </li>
 <li class=MsoNormal style='color:black'>Use best available data to set initial
     conditions for nutrients, biota, sediment, and chemicals (optional). </li>
 <li class=MsoNormal style='color:black'>Use data to determine the external
     boundary condition loads for nutrients and, optionally, chemicals. </li>
 <li class=MsoNormal style='color:black'>Test the model and calibrate against
     observed chlorophyll <em><span style='font-family:"Arial",sans-serif'>a,</span></em>&nbsp;biomass,
     and nutrients. </li>
 <li class=MsoNormal style='color:black'>Examine&nbsp;the effects of changing
     nutrient, sediment, or organic toxicant conditions.<br>
     <br>
 </li>
</ul>

<h3 style='margin:0in;page-break-after:avoid'><a name="_Toc124172120"><span
style='font-size:12.0pt'>Data Requirements</span></a></h3>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>Data requirements in AQUATOX depend considerably on the
site-type that is being modeled and the goal of the modeling study.&nbsp; A
modeling study to assess the effects of changing nutrient levels would benefit
from an excellent accounting of nutrients in the water column and
boundary-condition loadings of nutrients.&nbsp; On the other hand, a study that
is primarily about chemical effects might not require such detailed nutrient
information.&nbsp; In this case, a user might&nbsp;wish to produce the required
nutrient environment for a stable food-web and then&nbsp;detailed information
about chemical loadings can be applied to the simulation to assess the effects.</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Precise accounting of time-series loadings is especially
important in rivers and streams.&nbsp; Due to low retention time, inflow
loadings can be the dominant&nbsp;factor affecting water column
conditions.&nbsp; For standing water, initial conditions&nbsp;might be&nbsp;more
important than a perfect representation of daily loadings depending on both
simulation time and retention time.</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Below is a list of data requirements put together for a
single-segment simulation project.&nbsp; This list&nbsp;is not necessarily
exhaustive but contains what are generally the most important time-series
inputs.&nbsp;<br>
<br>
</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>Total N or Nitrate and Ammonia (inflow water, point
     source, non-point source) </li>
 <li class=MsoNormal>Total P&nbsp;or&nbsp;Bioavailable P, (inflow water, point
     source, non-point source) </li>
 <li class=MsoNormal>Chemical loadings (inflow water, point source, non-point
     source).</li>
 <li class=MsoNormal>Oxygen in inflow water </li>
 <li class=MsoNormal>TSS value in water </li>
 <li class=MsoNormal>Light&nbsp;Loadings (can be calculated based on latitude) </li>
 <li class=MsoNormal>CBOD or Suspended Detritus Concs (organic matter inflow,
     point source, non-point source). </li>
 <li class=MsoNormal>Water Volume,&nbsp;Inflows, or Outflows (some flexibility
     here) </li>
 <li class=MsoNormal>Water Temperature (mean ranges can be used) </li>
 <li class=MsoNormal>Water Mean Depth (optional)</li>
</ul>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Not all&nbsp;loadings need to be precise.&nbsp; However,
it is useful to have at least some data for each of the various categories
(nutrients, oxygen, organics, inorganic sediments or TSS, etc.)&nbsp;&nbsp; </p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>A user will need to define the biotic state variables in
each segment as well.&nbsp; Default parameterizations may be used for various
organisms unless there's a pressing need to use your own site-specific
parameters.<br>
<br>
If a chemical is modeled, requirements include initial concentrations in biota,
loadings from various sources, and toxicity data if you wish to model effects.</p>

<p style='margin:0in'>&nbsp;</p>

<h3 style='margin:0in;page-break-after:avoid'><a name="_Toc124172121"><span
style='font-size:12.0pt'>Site Types</span></a></h3>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>The site type may be set in the site data screen. </p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>The following site-types are included:</p>

<p style='margin:0in'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal><strong><span style='font-family:"Arial",sans-serif'>Pond,
     Lake, Reservoir:&nbsp;</span></strong> These standing-water types may
     currently be used interchangeably but different site-types
     are&nbsp;provided for user clarification. </li>
 <li class=MsoNormal><strong><span style='font-family:"Arial",sans-serif'>Stream:</span></strong>
     &nbsp;&nbsp;Assumed to be&nbsp;moving water, streams have the following
     additional characteristics </li>
 <ul style='margin-top:0in' type=circle>
  <li class=MsoNormal>Manning's coefficient may be used to model water volume; </li>
  <li class=MsoNormal>periphyton scour and velocity limitation for plants is a
      function of water velocity; </li>
  <li class=MsoNormal>oxygen reaeration is a function of stream velocity.</li>
 </ul>
 <li class=MsoNormal><strong><span style='font-family:"Arial",sans-serif'>Enclosure:</span></strong>
     &nbsp; An artificial&nbsp;experimental&nbsp;enclosure within&nbsp;a pond
     or lake.&nbsp; As noted in the technical documentation (U.S. Environmental
     Protection Agency. 2018), the littoral fraction is modified based on the
     area of the&nbsp;enclosure's wall. </li>
 <li class=MsoNormal><b>Marine: </b>Selecting this option primarily has
     effects on nitrification, denitrification, reaeration, and water-freezing
     temperature. See section 3.9 of the Release 3.2 <i>Technical
     documentation (U.S. Environmental Protection Agency. 2018).</i></li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<h3 style='margin:0in;page-break-after:avoid'><a name="_Toc124172122"><span
style='font-size:12.0pt'>Starting with a Surrogate Simulation</span></a><span
style='font-size:12.0pt'> </span></h3>

<div>

<p class=MsoNormal style='page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal>When applying AQUATOX to a new site it is usually most
efficient to find a surrogate site that best matches the characteristics of the
site to be modeled.&nbsp; The user can then modify that site's characteristics
so that it matches the modeled site with respect to site morphometry, nutrients,
organic matter, suspended sediments, biota, and organic chemicals (if
relevant). </p>

</div>

<div>

<p class=MsoNormal>&nbsp;</p>

</div>

<div>

<p class=MsoNormal>To assist in this process the file &quot;<strong><span
style='font-family:"Arial",sans-serif'>Study_descriptions_AQUATOX DOT NET.pdf&quot;</span></strong>
has been added to the STUDIES directory and describes the characteristics of
each of the example model applications included with the model.</p>

<p class=MsoNormal>&nbsp;</p>

</div>

<p style='margin:0in'>After the surrogate simulation has been loaded, the user
should save the JSON file to a new name and then examine the following
components for compatibility with the site being modeled</p>

<p style='margin:0in'>&nbsp;</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>Physical characteristics (site parameters)</li>
 <li class=MsoNormal>Water volume setup (water volume loadings and initial
     condition screen.)</li>
 <li class=MsoNormal>Nutrients, organic loadings, and turbidity in the loadings
     and initial conditions screens</li>
 <li class=MsoNormal>Simulation time and other model setup</li>
 <li class=MsoNormal>Water temperature settings</li>
 <li class=MsoNormal>Biotic compartments included and food web (trophic
     interaction matrix)</li>
</ul>

<p style='margin:0in'>&nbsp;</p>

<h3 style='margin:0in;page-break-after:avoid'><a name="_Toc124172123"><span
style='font-size:12.0pt'>Water Volume Modeling Options</span></a></h3>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>As noted in the Water Volume Data screen, there are many
options as to how to compute or specify water volume; each requires a different
set of input data.&nbsp; Often, the&nbsp;selected volume-modeling option is a
function of the available data for&nbsp;the site being modeled.</p>

<p style='margin:0in'>Time series&nbsp;of stream volumes are quite rare
whereas&nbsp;discharge data are more often available.&nbsp; For this reason,
the Mannings Equation, which derives river volumes based on discharge, slope,
and width data,&nbsp;is often used for streams.</p>

<p style='margin:0in'>Standing water volumes are sometimes assumed to be kept
constant; alternatively, time series of known volumes may be used.&nbsp; See
the <a href="#Volume">Water Volume Data</a> section for more information about
modeling water volume.</p>

<p style='margin:0in'>&nbsp;</p>

<h3 style='margin:0in;page-break-after:avoid'><a name="Adding_State_Variable"></a><a
name="_Toc124172124"><span style='font-size:12.0pt'>Adding a State Variable</span></a></h3>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>Chemicals and biotic state variables can be added to the
model through the main interface screen (by clicking the &quot;Add&quot; button
under the list of state variables, which brings up the Insert State Variable screen
and its list of available state variable compartments).&nbsp; </p>

<p style='margin:0in'>If you are modeling an animal, chemical, or plant state
variable for which there are no database parameters available, you will first
need to create a new set of parameters for your state variable in the relevant
AQUATOX database. Often it is best to use an existing database record most
similar to your new addition and modify as necessary. See the section on <a
href="#Parameters">parameters</a><i> </i>for more instructions on adding a new
database entry.</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="_Toc124172125"><span
style='font-size:11.0pt'>Adding a Chemical</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>To add a chemical in AQUATOX&nbsp;select one of the 20
available toxicant&nbsp;compartments (usually &quot;Dissolved org. tox 1&quot;)
on the Insert State Variable screen.&nbsp; You will then be prompted as to
which chemical to load from the chemical database.&nbsp; Loading this chemical
brings in parameters for the chemical database as well as animal and plant <strong><span
style='font-family:"Arial",sans-serif'>toxicity databases</span></strong>.&nbsp;
</p>

<p style='margin:0in'>To properly model toxicity for the chemical, each and
every animal and plant in the simulation must be properly linked to one of the
rows in these toxicity databases.&nbsp; If any plant or animal lacks the link
to toxicity data an error message with be returned. In the animal and plant parameters,
the toxicity linkages may be entered individually by choosing the appropriate
group in the dropdown in the Toxicity Record field or &quot;Edit All&quot;
may be selected, allowing you to link these organisms all at once.</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="_Toc124172126"><span
style='font-size:11.0pt'>Adding a Plant</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>To add a plant in AQUATOX, select one of the available
plant compartments (Diatoms, Greens, Blue-greens,&nbsp;Other Algae, or
Macrophytes) on the Insert State Variable screen and then choose the plant
data you wish to load from the plant&nbsp;database.&nbsp; &nbsp;Initial
conditions for the plant and any external loadings must then be
specified.&nbsp; A &quot;seed&quot;&nbsp;loading may be specified (generally as
a constant load) to prevent permanent extinction from occurring.&nbsp; Seed
loadings are generally very small loadings (e.g., 1e-5 mg/L) that allow for
reintroduction of an organism after environmental conditions have improved to
allow that organism to be viable in the system again. </p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="_Toc124172127"><span
style='font-size:11.0pt'>Adding an Animal</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>To add an animal in AQUATOX, select one of the available
animal compartments (Shredders, Sediment Feeders, Suspended Feeders, Clams,
Grazers, Snails, Predatory Invertebrates, Large and Small Forage Fish, Large
and&nbsp;Small Bottom Fish, and Large and Small Game Fish) on the Insert State
Variable screen and then choose the animal data you wish to load from the animal&nbsp;database.&nbsp;&nbsp;&nbsp;Initial
conditions for the animal, any external loadings, and trophic interactions
within the food web must also be specified.&nbsp; Size-class fish may be
specified using the&nbsp;<strong><span style='font-family:"Arial",sans-serif'>Animal
Linkages</span></strong> interface. &nbsp;A &quot;seed&quot;&nbsp;loading may
be specified as a constant load to prevent permanent extinction from
occurring.&nbsp; </p>

<p style='margin:0in'>&nbsp;</p>

<h3 style='margin:0in;page-break-after:avoid'><a name="_Toc124172128"><span
style='font-size:12.0pt'>Using Sediment Bed Models and Data Requirements</span></a></h3>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>Inorganic sediments can have significant effects on light
climate and inorganic sediment effects on biota can also be explicitly
modeled. AQUATOX DOT NET contains two options for inorganic sediment
submodels:</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in;
text-indent:-.25in'>1.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;
</span>TSS Model: The TSS option is a simple model based on a regression
relationship between sediment deposition and total suspended sediments. This
option also ensures that the effects of inorganic sediments on the light
climate of the water column are properly represented. This approach should be
used when the only inorganic sediment data available are TSS. This option also
assumes that TSS does not undergo any degradation or other reactions while
within the AQUATOX reach. Add the &quot;TSS&quot; state variable to use this
option. <br>
<br>
</p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in;
text-indent:-.25in'>2.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;
</span>Sediment Diagenesis Model: The sediment diagenesis model implements the
sediment diagenesis model as presented in <em><span style='font-family:"Arial",sans-serif'>Sediment
Flux Modeling </span></em>(Di Toro,2001). This model provides a more
sophisticated accounting of the decay of organic matter and remineralization in
an anaerobic sediment bed and the effects on sediment oxygen demand and
nutrient flux into the water column. The diagenesis model assumes a
depositional environment; scour of sediments is not incorporated. </p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in'>&nbsp;</p>

<p style='margin-top:0in;margin-right:0in;margin-bottom:0in;margin-left:.5in'><i>Note,
the Sand-Silt-Clay Model was not brought forward into AQUATOX DOT NET at this
time.<br>
<br>
Note, the Multi-Layer Sediment Model was not brought forward into AQUATOX DOT
NET at this time.<br>
<br>
</i></p>

<p style='margin:0in'>The simplest manner is to use&nbsp;the
AQUATOX&nbsp;&quot;default&quot; sediment bed model in which organic matter is
modeled in the sediment bed as a food source and toxicant-sorption
site&nbsp;but inorganic matter in the bed is ignored. The model will utilize
this option if none of the two approaches above have been added to a
simulation. The model will always track the remineralization of organic
material within the sediment bed and the water column.</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="Default_Sediment"></a><a
name="_Toc124172129"><span style='font-size:11.0pt'>Default Sediment Bed Model</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in;page-break-after:avoid'>The default sediment bed model
includes the following components</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>Labile Sed. Detritus (Labile Detritus in the Sediment bed)
     (g/m2 dry); </li>
 <li class=MsoNormal>Refrac. Sed. Detritus (Refractory Detritus in the Sediment
     bed) (g/m2 dry); </li>
 <li class=MsoNormal>Toxicants sorbed to both of those compartments (g/kg
     dry);</li>
</ul>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>These detrital compartments are assumed to be located in
the active layer and available&nbsp;for consumption and sorption-desorption&nbsp;of
chemical from-to the water column.&nbsp; Through long-term deposition, these
compartments may become deeply buried and join the buried detritus state
variables.&nbsp; Buried detritus is considered to be non-reactive. Nutrients
within the sediment detrital compartments are subject to release to the water
column when the organic matter decomposes.&nbsp; For more information about
this model, please see section 5.1 of the AQUATOX Technical documentation (U.S.
Environmental Protection Agency. 2018).</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Data requirements for this model are rather minimal,
primarily being the initial condition of detritus in the sediment bed.&nbsp;
Setting the initial condition for these state variables can be facilitated by
using an Excel spreadsheet (<strong><span style='font-family:"Arial",sans-serif'>AQUATOX_Sed_Bed_Inputs.xls</span></strong>)
that is included in the Studies directory.&nbsp; This spreadsheet will convert
data about the fraction organic carbon, depth,&nbsp;sediment density, and
fraction labile into initial conditions for these two state variable
compartments.</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name=TSS></a><a
name="_Toc124172130"><span style='font-size:11.0pt'>TSS</span></a></h4>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>AQUATOX allows a user to input time-varying
concentrations of TSS so that the light climate of the system being modeled can
be accurately represented.&nbsp; TSS may represent either total suspended
solids or total suspended sediments.&nbsp; A user can choose which of these
options is being specified using the radio buttons to the left of this screen
(accessed by double-clicking TSS on the state variable list). </p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>If total suspended solids are chosen, AQUATOX will
subtract phytoplankton and detritus loadings from the TSS loadings to estimate
the inorganic solids loadings,&nbsp;therefore, care should be taken to
use&nbsp;synoptic TSS and nutrient time series. </p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>When TSS is included in the model, it is a driving
variable that is not in any way linked to the sediment bed unlike the inorganic
sediments in the Sand-Silt-Clay model.</p>

<p style='margin:0in'>&nbsp;</p>

<h4 style='margin:0in;page-break-after:avoid'><a name="Sediment_Diagenesis"></a><a
name="_Toc124172131"><span style='font-size:11.0pt'>Sediment Diagenesis Model</span></a></h4>

<p style='margin:0in;page-break-after:avoid'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>Di Toros (2001) model assumes
a small aerobic layer (L1) above a larger anaerobic layer (L2).&nbsp; For this
reason, it is best to apply this optional submodel in sites where anoxic
sediments are prevalent, otherwise the assumption of a very&nbsp;thin aerobic
layer&nbsp;might not be appropriate.</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>The additional 22 state
variables added when the sediment diagenesis model is enabled are as follows:</span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Ammonia</span></strong>:
     two state variables to represent two layers (Layers 1 and 2).&nbsp; </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Nitrate:</span></strong>
     two state variables (in Layers 1 and 2).&nbsp; </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Orthophosphate</span></strong>:
     two state variables (in Layers 1 and 2).&nbsp;&nbsp; </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Methane:</span></strong>
     modeled in Layer 2 </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Sulfide</span></strong>:&nbsp;
     two state variables (in Layers 1 and 2).&nbsp; </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Bioavailable
     Silica</span></strong>:&nbsp; modeled in Layer 2.&nbsp; </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>Non-Biogenic
     Silica</span></strong>:&nbsp; two state variables (in Layers 1 and
     2).&nbsp; </li>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif;
     font-weight:normal'>Particulate Organic Matter (POM) variables</span></strong><span
     style='color:windowtext'>:</span></li>
</ul>

<p class=MsoNormal style='margin-left:.75in;text-indent:-.25in'><span
style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><strong><span style='font-family:"Arial",sans-serif;color:black'>POC
</span></strong><span style='color:black'>(Particulate Organic Carbon) in
sediment: three state variables to represent three reaction classes in the
anaerobic layer.&nbsp; </span></p>

<p class=MsoNormal style='margin-left:.75in;text-indent:-.25in'><span
style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><strong><span style='font-family:"Arial",sans-serif;color:black'>PON
</span></strong><span style='color:black'>(Particulate Organic Nitrate) in
sediment: three state variables to represent three reaction classes in the
anaerobic layer.&nbsp; </span></p>

<p class=MsoNormal style='margin-left:.75in;text-indent:-.25in'><span
style='font-size:12.0pt;font-family:"Times New Roman",serif;color:black'>-<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><strong><span style='font-family:"Arial",sans-serif;color:black'>POP
</span></strong><span style='color:black'>(Particulate Organic Phosphate) in
sediment: as with POC, three state variables to represent three reaction
classes in the anaerobic layer. </span></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal style='color:black'><strong><span style='font-family:"Arial",sans-serif'>COD</span></strong>:&nbsp;
     Driving variable for chemical oxygen demand in the water column that
     affects the flux of sulfide to the water column (optional, and to be
     differentiated from carbonaceous CBOD).&nbsp;</li>
</ul>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>The sediment diagenesis model
is relevant for only depositional regimes.&nbsp; No direct scour of sediment
from the sediment bed is modeled.</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>AQUATOX DOT NET includes the option
to <b>Model Diagenesis Layer 1 as Steady State.</b> When this option is
selected, state variables in the upper-layer (between Layer 2 and the water
column) are all modeled using the assumption of steady-state. This is the
native mode of Di Toros (2001) sediment diagenesis model. As noted in the
interface, this process significantly speeds model execution (by up-to a factor
of 10), but might have an effect on the mass balance of nutrients. However, effects
on overall model results are generally quite minor. More information about
this option is available in Chapter 7 of the Technical documentation (U.S.
Environmental Protection Agency. 2018).</span></p>

<p style='margin:0in'><b><span style='color:black'>&nbsp;</span></b></p>

<p style='margin:0in'><span style='color:black'>Data requirements for this
model are moderate.&nbsp; Initial conditions for all variables specified above
are required but rapidly become a function of organic matter deposition and can
be derived by &quot;spinning up&quot; the model (running the model with
arbitrary initial conditions and then setting the initial conditions based on
model results which are a function of organic matter deposition). Initial
conditions for the variables, except for COD, are entered on the Diagenesis
Model Setup screen.</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>Selecting the Edit Diagenesis
Parameters button on the Diagenesis Model Setup screen brings you to a large
set of &quot;<strong><span style='font-family:"Arial",sans-serif'>Diagenesis
Parameters</span></strong>&quot; that also govern the behavior of the
model.&nbsp; Most of these may be kept constant from site-to-site but Di Toro (2001)
suggests a few parameters are site-specific.&nbsp; Partitioning to phosphorus,
in particular, is subject to considerable site specificity (<em><span
style='font-family:"Arial",sans-serif'>kdPO42 </span></em>and&nbsp;<em><span
style='font-family:"Arial",sans-serif'>dKDPO41f</span></em>).&nbsp; For
reporting and archival purposes, parameters can be saved to Excel using a
button on this screen.</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>Setting the initial condition
for the organic matter sediment diagenesis&nbsp;state variables can be
facilitated by using an Excel spreadsheet (<strong><span style='font-family:
"Arial",sans-serif'>AQUATOX_Sed_Bed_Inputs.xls</span></strong>) that is
included in the Studies directory.&nbsp; This spreadsheet will convert data
about the fraction organic carbon, depth,&nbsp;sediment density, and fraction
labile and nonreactive sediment, as well as stoichiometry
assumptions,&nbsp;into initial conditions for these three state variable
compartments (POC, PON, and POP).</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<p style='margin:0in'><span style='color:black'>For more information about the
equations and concepts within the Sediment Diagenesis model, please see chapter
7 of the Technical documentation (U.S. Environmental Protection Agency. 2018).</span></p>

<p style='margin:0in'><span style='color:black'>&nbsp;</span></p>

<h3 style='margin:0in;page-break-after:avoid'><a name="_Toc124172132"><span
style='font-size:12.0pt'>Model Calibration</span></a><span style='font-size:
12.0pt'> </span></h3>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>Model calibration is the process of modifying
model&nbsp;parameters within their range of uncertainty (reasonable ranges as
defined by literature if possible) such that model results&nbsp;match observed
data or observed conditions.&nbsp; </p>

<p style='margin:0in;page-break-after:avoid'>&nbsp;</p>

<p style='margin:0in'>Parameters will have a varying degree of uncertainty
depending on how measurable a parameter is, variability in field measurements,
and uncertainty in lab equipment.&nbsp; Parameters will also have a varying
degree of sensitivity depending on how much&nbsp;model outputs change as a
function of the change in parameter value.&nbsp; </p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Calibration can be like trying to solve a puzzle.&nbsp;
It requires creative thought and a real understanding of why the model is
behaving the way it is.&nbsp; </p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>The first step in AQUATOX model calibration&nbsp;is often
to&nbsp;ensure that the food-web is stable and matches current
conditions.&nbsp; This might be quantitative if biomass data exists&nbsp;for a
site or it might be qualitative.&nbsp; For example, if the user knows that
largemouth bass exist at the modeled&nbsp;site but this category does not
persist in model results, then a re-specification of the trophic interactions
(or the lower portions of the food-web) will be required.&nbsp;&nbsp;Once the
food-web is stable and reasonable, then calibration to organic toxicant data can
be undertaken.&nbsp;</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>The user should start by trying to specify all of the
important components of the food-web simultaneously.&nbsp; Calibrating the
primary producers alone might be misleading as this model would not include
grazing pressures from higher in the food web.&nbsp; Next, produce a plot of
all the plants and animals and ensure that there are no inappropriate crashes
or expansions of biomass beyond what would be supported by observed data or,
lacking that, common sense.</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'><strong><span style='font-family:"Arial",sans-serif'>Some
notes about calibrating plants:</span></strong></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>Algae are differentiated on basis of: </li>
 <ul style='margin-top:0in' type=circle>
  <li class=MsoNormal>nutrient half-saturation values (P half-saturation and N
      half-saturation) </li>
  <li class=MsoNormal>light saturation values (Light Saturation or Adaptive
      Light option), and </li>
  <li class=MsoNormal>maximum photosynthetic rate (PMax).</li>
 </ul>
 <li class=MsoNormal>Phytoplankton sedimentation rates differ between running
     and standing water. </li>
 <li class=MsoNormal>Some parameters are fairly site-specific, probably due to
     adaptation by the algae:&nbsp; These include critical force for periphyton
     scour (FCrit) and optimum temperature (TOpt).</li>
 <li class=MsoNormal>AQUATOX biotic state variables are quite sensitive to
     temperature parameters. These parameters include optimal temperature,
     maximum temperature, and temperature response slope.</li>
 <li class=MsoNormal>Percent&nbsp;lost&nbsp;in&nbsp;slough&nbsp;event&nbsp;is&nbsp;a&nbsp;sensitive&nbsp;parameter&nbsp;for&nbsp;periphyton&nbsp;biomass.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p style='margin:0in'><strong><span style='font-family:"Arial",sans-serif'>Some
notes about calibrating animals:</span></strong></p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>The trophic interaction matrix is usually subject to
     uncertainty;&nbsp;proper specification of feeding preferences is critical.
     </li>
 <li class=MsoNormal>Simpler&nbsp;food<span style='font-family:"Cambria Math",serif'>&#8208;</span>web&nbsp;models&nbsp;are&nbsp;more&nbsp;sensitive&nbsp;to&nbsp;effects&nbsp;from&nbsp;food<span
     style='font-family:"Cambria Math",serif'>&#8208;</span>web&nbsp;interactions
     (less dietary substitution is possible resulting in more intense indirect
     effects).&nbsp;&nbsp;&nbsp;</li>
 <div>
 <li class=MsoNormal>Biomass predictions are generally sensitive to
     &quot;Maximum Consumption&quot; rate and&nbsp;&quot;Endogenous
     Respiration&quot;&nbsp;rates when not calculated based on weight
     (allometrics); when allometric formulations are used, the model is
     sensitive to those parameters.</li>
 </div>
</ul>

<div>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>Mortality is often a site-specific response and is
     therefore subject to calibration.</li>
</ul>

</div>

<div>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>The optimum temperature can have a significant effect on
     biomass computations.</li>
</ul>

</div>

<div>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>The minimum prey for feeding affects the efficiency of
     foraging behavior.</li>
</ul>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Additional information about model calibration may be found
in the following document:</p>

<p class=MsoNormal>EPA, 2009. AQUATOX Technical Note 1, A Calibrated Parameter
Set for Simulation of Algae in Shallow Rivers, EPA-823-R-09-003 February 2009</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Also see Section 2.6 on &quot;Calibration and
Validation&quot; in the AQUATOX Technical documentation (U.S. Environmental
Protection Agency. 2018) and Section 2.4 on Sensitivity Analysis.</p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

</div>

<h3 style='margin:0in'><a name="_Toc124172133"><span style='font-size:12.0pt'>Model
Validation</span></a></h3>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Model validation is a&nbsp;numerical cross-examination of
the model against new data sets.&nbsp; By definition, a calibrated
model&nbsp;is effective given the conditions it was calibrated against, but
there is uncertainty as to whether it will correctly model changing
conditions.&nbsp; If the model can be tested against a different data set than
was used for calibration, this increases confidence that the model has the
capability to effectively capture alternative scenarios.</p>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Model validation can&nbsp;consist of extrapolating the
model to one of the following:</p>

<ul style='margin-top:0in' type=disc>
 <li class=MsoNormal>New time-periods, </li>
 <li class=MsoNormal>New sites and exposure conditions, or</li>
 <li class=MsoNormal>Different chemicals. </li>
</ul>

<p style='margin:0in'>&nbsp;</p>

<p style='margin:0in'>Model validation&nbsp;exercises are&nbsp;often defined
by, or limited by, the data available for validation.&nbsp; </p>

<p style='margin:0in'>If a model validation is not successful, then the
user&nbsp;can go back to the model calibration and seek an alternate
calibration that will adapt to changing conditions more successfully.&nbsp; The
model calibration and validation data sets could be combined for a more robust
calibration, but at the expense of the loss of the validation data sets; this
combined approach should be avoided if possible.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Also see Section 2.6 on &quot;Calibration and
Validation&quot; in the AQUATOX Technical documentation (U.S. Environmental
Protection Agency. 2018).</p>

<p class=MsoNormal><span style='font-size:12.0pt;font-family:"Times New Roman",serif'>&nbsp;</span></p>

<h2><a name="Multi_Segment_Runs"></a><a name="_Toc124172134"><span
style='font-size:14.0pt;color:black'>Multi-Segment </span></a><span
style='font-size:14.0pt;color:black'>and Data-Linkage Mode</span></h2>

<p class=MsoNormal>AQUATOX DOT NET can model multiple 0-D segments linked
together. In this case, water, nutrients, and biota will flow through pre-defined
(NHDPlusV2) river and lake networks within the continental US. It can also
pull data from HAWQS/SWAT or the National Water Model for single segment
simulations of hydrologic drainage basins or lakes and reservoirs defined
within the NHDPlusV2 data set.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Modeling options are as follows:</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>Stream networks</b> based on NHDPlus flow lines linked
together. These stream networks can include lakes and reservoirs as defined as
an NHDPlusV2 waterbody. Stream segments are based on USGS COMIDs<a
href="#_ftn1" name="_ftnref1" title=""><span class=MsoFootnoteReference><span
class=MsoFootnoteReference><span style='font-size:10.0pt;font-family:"Arial",sans-serif'>[1]</span></span></span></a></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span>Water flows and time-series water volumes can be automatically
imported from the national water model.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span>Water flows, nutrient and sediments from boundary conditions and
overland flows, water temperature, chlorophyll <i>a</i>, and dissolved oxygen
can be imported using runs of the HAWQS/SWAT modeling system.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>A single HUC</b> (Hydrologic Unit polygon boundaries) may be
modeled as a segment.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span>Water flows, nutrient and sediments from boundary conditions and
overland flows, water temperature, chlorophyll <i>a</i>, and dissolved oxygen
can be imported using runs of the HAWQS/SWAT modeling system.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><b>A single lake or reservoir</b> (NHDPlusV2 waterbody) </p>

<p class=MsoListParagraphCxSpLast style='margin-left:1.0in;text-indent:-.25in'><span
style='font-family:"Courier New"'>o<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;
</span></span>Water flows and water volumes can be automatically imported from
the national water model<br>
<br>
</p>

<p class=MsoNormal><img border=0 width=578 height=200
src="AQUATOX.NET_1.0_UMAN_files/image004.gif"></p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><br>
When working within the multi-segment window, all model files are saved in a
single working directory. To start a new simulation, select <b>New Project. </b>This
will bring up the <u><span
style='color:#1F497D'>New Simulation Window</span></u>. After the spatial and
temporal domain of the simulation has been set up, and the base simulation
selected, clicking OK will exit this screen. AQUATOX will then ask for a directory
in which the new simulation will be stored. Either a new folder can be created
and selected, or an existing folder where there is no simulation, or a folder in
which the user does not mind overwriting the existing AQUATOX simulation.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>You can also toggle between a list of the most recent
project directories using the Recent Projects pull-down menu.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Once a new project has been created a map of the simulation
will be shown. A user can toggle between a log console and the map using the
radio buttons below the map. Once the model has been run, a summary of model
results across segments can be selected with the Show Graph button. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>When the map is displayed, a user can zoom and pan by using
the mouse wheel or dragging across the map with the mouse cursor. The Reset
Map button will re-center and zoom the map to show the selected model domain.
Different layers may be selected for view selecting the layers icon at the
upper right of the map that looks like stacked papers. Other mapping options
include:</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Show HUC14sthis displays the smallest size drainage basin
included in the model. HUC14s are the native size for a HAWQS/SWAT simulation
that will be linked to stream-segment results.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Show Boundary Conditions  icons will be displayed that show
boundary condition stream inputs. Green icons represent upstream boundary
conditions and a single red icon represents the pour-point for the simulation.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Labels  when selected this will display the COMIDs on the map.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'><span
style='font-family:Symbol'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span>Show Non-Run Segments  when there is a lake or reservoir within
a simulation some COMID stream segments are superseded by the larger
lake/reservoir polygon. These segments are plotted when show non-run
segments is selected.</p>

<p class=MsoListParagraphCxSpLast>&nbsp;</p>

<p class=MsoNormal>Once the domain has been selected, the user can then follow
the following steps to execute a simulation. In general, the workflow moves
from the upper left of the screen to the lower left:</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span><b>Base
JSON: </b>Choose a template 0-D AQUATOX segment to form the basis for the
multi-segment simulation. <b>This is usually completed through the New
Simulation Window, but can be modified here if that is desired.</b><br>
<br>
</p>

<p class=MsoListParagraphCxSpMiddle>This template segment should contain all
the state variables that the user wishes to model (in every segment of the
simulation). State variables should be added or removed from the 0-D segment
prior to specifying the JSON in the multi-segment interface. The state
variable list must be the same in all modeled segments to simplify the passage
of data. (Biotic state variables that are irrelevant in one segment may be set
to zero in that segment, however.)<br>
<br>
</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span><b>Choose
Template </b>(optional)<b>:</b> this button brings up the <u><span style='color:#365F91'>Select Surrogate</span></u><u><span style='color:#365F91'>
Simulation</span></u> interface that shows most of the relevant AQUATOX
simulations that can form the basis of this new model. <br>
<br>
</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span><b>Master
Setup: </b>This brings up the setup record indicating the start date, end
date, and integration options for the linked system. This setup record will be
read from and used each time the model is executed. <br>
<br>
Note: When water flow and velocity data are read from the National Water Model,
the dates specified in the setup record will be used to read time series
inputs. Therefore, if the setup record dates change after create linked
inputs is selected, the create linked inputs step will need to be repeated
to query the correct time period. The interface creates a warning in this
case.<br>
<br>
</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span><b>Choose
your data source: </b></p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b>HAWQS Simulation: </b>This will enable
the user to set up and run a HAWQS/SWAT simulation to provide data about water
flows, nutrient and sediments from boundary conditions and overland flows,
water temperature, chlorophyll <i>a</i>, and dissolved oxygen</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle><b>NWM (flows only): </b>This will import
flows and water volumes from the national water volume, but the user must
specify boundary conditions </p>

<p class=MsoListParagraphCxSpLast>&nbsp;</p>

<p class=MsoNormal style='margin-left:.5in'>Note, when modeling 0-D HUC
simulations, NWM data are not spatially aligned and therefore not relevant.
Similarly, when modeling 0-D lakes and reservoirs, HAWQS/SWAT data are not
spatially compatible, so NWM data is the only option.</p>

<p class=MsoListParagraph><b>&nbsp;</b></p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'>4a. (NWM chosen)<b>
Create Linked Inputs: </b>This button reads water flows and water velocities
from the National Water Model (NWM) and also saves a separate JSON input
simulation for each COMID in the stream network in the base directory (named
AQT_Input_[COMID].JSON). </p>

<p class=MsoListParagraphCxSpFirst><br>
Water flows from one reach to another are imported based on NWM flow data. <b>Note:
</b>This process may take several minutes depending on the number of segments
in the model and the number of days in which water-flow data needs to be read
from the web service.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>For lake segments, volume, inflows, and
outflows are read from the National Water Model.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>For stream segments, a time-series volume
for each COMID is specified based on the length of the COMID and the NWM flow
and velocity.<br>
<br>
More information about model linkage may be found in the section on <a
href="#DataLinkage">Data Linkage Assumptions</a> below.</p>

<p class=MsoListParagraphCxSpMiddle><br>
After linked inputs are created, parameters for individual segments may be
edited by clicking on the waterbody on the map. Initial conditions,
point-source, and non-point-source loadings may be edited within the interface
before executing the network in step five below. </p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>At the inflow boundary of the stream
network, boundary condition of nutrients or other state variables should be
specified. (Inflow loadings for downstream segments will be overwritten when
the stream-network is executed as these data will be passed from the upstream
segment.)</p>

<p class=MsoListParagraphCxSpLast>&nbsp;</p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'><a
name="Overland_Flows"></a>5a. (NWM Chosen) <b>Overland Flows: </b>Optional,
and stream-network only. This button allows the user to specify non-point
source loadings for each segment in a constant g/d for nutrients and organic
matter through a matrix input. If specified, these inputs will overwrite any
other non-point source loadings in a segment before model execution. <br>
<br>
</p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'>4b. (HAQS
Simulation Chosen) <b>Run HAWQS: </b>This button will first identify the
relevant domain for the HAWQS simulation and will then display a dialog that
shows the HAWQS API parameters as documented <a
href="https://dev-api.hawqs.tamu.edu/#/">here</a>. For the most part, these
parameters do not need to be changed and should only be changed by users
familiar with the HAWQS/SWAT modeling system. The user must also input a HAWQS
API Key that will be required to run the HAWQS model. <span style='background:
yellow'>To get an API key, contact TBA</span>. Following this, the status of
the HAWQS run will be shown on a progress bar in the Model Execution window.
If the HAWQS run completes successfully, model results will be saved in the
project directory and the model is ready for linkage.<br>
<br>
</p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'>5b. (HAWQS
Simulation Chosen) <b>Link HAWQS Data: </b>This button reads all of the HAWQS
linkage data for water flows, nutrients, and sediment from the completed HAWQS
simulation. This button will saves a separate JSON input simulation for each
COMID in the stream network in the base directory (named AQT_Input_[COMID].JSON).
</p>

<p class=MsoListParagraphCxSpFirst><br>
For lake segments, volume, inflows, and outflows are read from the National
Water Model.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>More information about model linkage may be
found in the section on <a href="#DataLinkage">Data Linkage Assumptions</a>
below.</p>

<p class=MsoListParagraphCxSpMiddle>&nbsp;</p>

<p class=MsoListParagraphCxSpMiddle>After linked inputs are created, parameters
for individual segments may be edited by clicking on the waterbody on the map.
Initial conditions, point-source, and non-point-source loadings may be edited
within the interface before executing the network in step five below. </p>

<p class=MsoListParagraphCxSpMiddle><b>&nbsp;</b></p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>6. <b> Execute
Network: </b>This button will start the simulations execution. <br>
<br>
For stream networks, each AQUATOX simulation will be run in the order specified
in the stream network filemultiple segments will be run in parallel when
possible. The water-quality components from upstream segments will
automatically be added as loadings to downstream segments. <br>
<br>
After the simulation is run, model results will be saved in new JSONS in the
output directory labeled AQT_RUN_{COMID}.JSON. These segments may be examined,
run, and output graphs may be produced in the main AQUATOX interface. </p>

<p class=MsoListParagraphCxSpLast>&nbsp;</p>

<p class=MsoNormal style='margin-left:.5in;text-indent:-.25in'>7.<b>  View
Outputs: </b>By selecting a state variable from the SV Index drop down box,
graphs of all segments over time will be produced. The CSV button will export
a comma-separated value matrix of model results that may be exported into
spreadsheet software for further analysis.</p>

<p class=MsoNormal><b>&nbsp;</b></p>

<p class=MsoNormal>Once a run has been completed, if Click to Outputs is
selected, clicking on a stream line or waterbody will bring you to the <a
href="#OutputScreen">output window</a> that pertains to the segment selected. </p>

<p class=MsoNormal>&nbsp;</p>

<h3 style='page-break-after:avoid'><a name=MergeSegments>Merging Segments<o:p></o:p></a></h3>

<span style='mso-bookmark:MergeSegments'></span>

<p class=MsoNormal>This capability is relevant to models using NHD+ stream
networks.<span style='mso-spacerun:yes'> </span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal>Linked AQUATOX DOT NET stream segments are based on flow
lines defined using USGS NHD+ COMIDs.<span style='mso-spacerun:yes'>
</span>Within NHD+ river networks, <span style='mso-spacerun:yes'></span>relatively
short flowlines may be present.<span style='mso-spacerun:yes'> </span>These
segments, sometimes only a few hundred meters in length, are necessary to
connect adjacent flowlines or waterbodies.<span style='mso-spacerun:yes'>
</span></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal>Because of the very short travel time of these segments, they
will often act as bottlenecks for a linked-model simulation.<span
style='mso-spacerun:yes'> </span>A model solution for a segment will generally
require time steps that are shorter than the travel time for that segment.<span
style='mso-spacerun:yes'> </span>So if a segment has a travel time of 0.01
days, at least 100 time-steps per day will be required for its simulation.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal>To help with this problem, extremely short segments may be merged
with adjacent larger segments.<span style='mso-spacerun:yes'> </span>This can
be achieved using the <b>Merge Segments</b> button.<span
style='mso-spacerun:yes'> </span>When this button is pressed, the user is
provided with a dialog that gives statistics about the estimated travel time in
the selected stream network.<span style='mso-spacerun:yes'> </span>The average
travel time for the network and the fifth-percentile travel time are shown.<span
style='mso-spacerun:yes'> </span>The user may then select a threshold for
travel time and the number of segments that will be merged is updated.<span
style='mso-spacerun:yes'> </span>If the user selects OK from the dialog, segments
shorter than that threshold will be merged with an adjacent downstream segment.<br>
<br>
Here are the steps taken when segments are merged together:</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in;mso-list:l30 level1 lfo39'><![if !supportLists]><span
style='font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:
Symbol'><span style='mso-list:Ignore'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]>The stream network (<span class=SpellE>StreamNetwork.JSON</span>)
is backed up prior to modification.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in;mso-list:l30 level1 lfo39'><![if !supportLists]><span
style='font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:
Symbol'><span style='mso-list:Ignore'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]>The modified stream network merges together
small segments, adding them to an adjacent larger segment and updating length
and travel time information.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in;mso-list:l30 level1 lfo39'><![if !supportLists]><span
style='font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:
Symbol'><span style='mso-list:Ignore'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]>The network structure is updated and the order
of model solution is updated in the stream network.</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in;mso-list:l30 level1 lfo39'><![if !supportLists]><span
style='font-family:Symbol;mso-fareast-font-family:Symbol;mso-bidi-font-family:
Symbol'><span style='mso-list:Ignore'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span></span><![endif]>The shape for the larger segment is updated,
adding the smaller segment, and the original <span class=SpellE>GeoJSON</span>
file is backed up.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>

<p class=MsoNormal>Merged segments can be created for either an NWM-based or a
HAWQS-based simulation. The merging of segments should be performed prior to step
4 (NWM chosen) or step 5 (HAWQS chosen) above.<span style='mso-spacerun:yes'>
</span>This will ensure that the set of linked model inputs will be different
for the simplified stream network than it would be for an un-merged network.<span
style='mso-spacerun:yes'> </span>After the Link HAQWS data or Create Linked
Inputs has been completed, the simplified network will be ready to run and
should produce faster run times.</p>

<p class=MsoNormal><o:p>&nbsp;</o:p></p>



<h3 style='page-break-after:avoid'><a name=DataLinkage>Data Linkage Assumptions
and Procedures</a></h3>

<p class=MsoNormal>The data linkage button performs different tasks depending
on the data source chosen. </p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>First, for every segment in the model simulation, the base
JSON is copied into a unique simulation that will be used for model execution
(file name <b>AQT_Input_</b><i>[segment_ID]</i><b>.JSON</b>). Following
linkage and prior to model execution, inputs for each segment may be modified
to some degree, but state variables cannot be added or removed as all segments
must have the same list of state variables.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>If the <b>National Water Model (NWM)</b> is the source for
linkage data, the following steps are taken:</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>For every
NHD+ waterbody in the segment (lake or reservoir)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>A web
service request for NWM water volumes and water flows is sent to the EPA streamflow
HMS web service. (note, this call may take up to a minute, depending on the
number of dates in the simulation.) </p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Time-series
water volumes from NWM are copied into the AQUATOX segment and the water-volume
initial condition is set. <i>(As AQUATOX cannot work with zero water volumes,
the minimum volume for each segment is set to a cross-sectional area of 1m<sup>2
</sup>multiplied by the site length in meters)</i></p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>c.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Water
volumes are calculated in AQUATOX using the known volume algorithm and the
water-volume time series from NWM. Inflows are also imported from NWM and are
converted from m<sup>3</sup>/s to m<sup>3</sup>/d by multiplying by 86400 s/d.
</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>d.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>The mean
depth is estimated based on the initial-condition water volume and the sites
surface area. The maximum depth is set to two times the mean depth as a rough
estimate for now, but this can be modified by the user prior to simulation
execution.</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>For every
stream segment in the simulation (that is not superseded by an NHD+ waterbody)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>A web
service request for NWM water flows and water velocities is sent to the EPA
streamflow HMS web service. (Note, this call may take up to a minute,
depending on the number of dates in the simulation.)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>A
time-series set of volumes for each segment is estimated on the basis of NWM
data on flow and velocity. <br>
<br>
volume (m<sup>3</sup>) = flow(m<sup>3</sup>/s) / velocity(m/s) * sitelength (km)
* 0.001 (m/km)<br>
<br>
<i>(As AQUATOX cannot work with zero water volumes, the minimum volume for each
segment is set to a cross-sectional area of 1m<sup>2 </sup>multiplied by the
site length in meters.)<br>
<br>
</i></p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Note that
boundary-condition nutrients and organic matter are not available from the
National Water Model so these must be input by the user by editing the
AQT_Input_JSONs. This can be done by clicking on the stream segment the user
wishes to edit following model linkage. At that point, the single-segment
interface is available for the user to edit the simulations inputs and
parameters.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>If a <b>HAWQS simulation</b> is the source for linkage data,
the following steps are taken:</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>1.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Data from
the completed HAWQS model is read from a local data file. Data are passed from
the following HAWQS outputs:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>SED or
sediment concentrations in tons,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>NO3 Nitrate
in kg,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>c.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>NH4 Ammonium
in kg,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>d.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>MINP Mineral
phosphorus in kg, </p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>e.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>CHLA, Chlorophyll-a
in kg,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>f.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span>CBOD
Carbonaceous biochemical oxygen demand in kg,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>g.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>DISOX Dissolved
oxygen in kg,</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>h.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>WTMP
Water temperature in deg. C.<br>
<br>
</p>

<p class=MsoListParagraphCxSpLast style='text-indent:-.25in'>2.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>As most of
the data above are in units of kg/d, to convert inflows to mg/L the following
equation is used</p>

<p class=MsoNormal>&nbsp;</p>

<p class="MsoNormal" style="text-align: center;">
    <span style='font-size:10.0pt;font-family:"Arial",sans-serif'>
        <img width="450" src="AQUATOX.NET_1.0_UMAN_files/image005.gif">
    </span>
</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal style='margin-left:.5in'><span style='font-family:"Cambria",serif'>where:
</span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:5.0pt;
margin-left:153.35pt;text-indent:-99.35pt'><i><span style='font-family:"Cambria",serif'>CONC<sub>COMP,COMID
 </sub></span></i><span style='font-family:"Cambria",serif'>=  </span>concentration
of the component passing into the segment from the upstream segment (mg/L);<span
style='font-family:"Cambria",serif'> </span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:5.0pt;
margin-left:153.35pt;text-indent:-99.35pt'><i><span style='font-family:"Cambria",serif'>COMP<sub>IN,COMID </sub></span></i><span
style='font-family:"Cambria",serif'>=  </span>mass of the component passing
into the COMID (kg/d, or metric ton/d for sediment);</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:5.0pt;
margin-left:153.35pt;text-indent:-99.35pt'><i><span style='font-family:"Cambria",serif'>FLOW<sub>IN,COMID </sub></span></i><span
style='font-family:"Cambria",serif'>=  </span>flow of water passing into the
COMID (m3/s)</p>

<p class=MsoNormal style='margin-top:0in;margin-right:0in;margin-bottom:5.0pt;
margin-left:153.35pt;text-indent:-99.35pt'><i><span style='font-family:"Cambria",serif'>UnitsConversion = </span></i>usually
0.0115740 (mg/kg)(day/sec)(m3/L), but x1000 for sediment.</p>

<p class=MsoNormal style='margin-left:.25in'>Chlorophyll a data is assigned to
the first phytoplankton state variable in the simulation list (if it exists).
Data are converted to organic matter based on the c-to-chlorophyll a
parameter for that plant and the assumption that there are 1.90 grams of
organic matter for every gram of carbon.<br>
<br>
</p>

<p class=MsoListParagraphCxSpFirst style='text-indent:-.25in'>3.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>For every
NHD+ waterbody in the simulation (lake or reservoir, if relevant)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Water
volumes are read from NHD+ waterbody as shown in NWM Steps 1a-1d above.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Inflow and
overland flow data from HAWQS simulations are passed for each NHD+ COMID
flowline leading into or within the NHD+ waterbody. Data are aggregated and
added to the segment:</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>i.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Overland
flows are input as non-point source loadings. All loadings to COMIDs located
in the waterbody are summed for each date.</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>ii.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Inflow
loadings are input as boundary-condition inflow loadings. All boundary
condition loadings to the water body are summed on each date.<br>
<br>
</p>

<p class=MsoListParagraphCxSpMiddle style='text-indent:-.25in'>4.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>For every
stream segment in the simulation</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>a.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Stream
geometry and site-specific parameters are read from NHD Plus Catchment SQLite
database. Parameters added to the segment are </p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>i.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Length
(km)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>ii.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Slope
(m/m)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>iii.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Latitude
(degrees)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>iv.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Site
Elevation (m)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>v.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Channel
Width (m, bankfull)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>vi.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Channel
Depth (m, bankfull)</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>b.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Daily segment
volumes are estimated based on HAWQS water flows, the geometry data gathered
above, and mannings equation, assuming a mannings N of 0.040 s/m<sup>1/3</sup>
(the AQUATOX natural channel default).</p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>c.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>If the
segment is a boundary condition for the modeled system, HAWQS boundary
conditions for sediment, nutrients, gasses, and organic matter are added as
shown in steps 1 and 2 above. </p>

<p class=MsoListParagraphCxSpMiddle style='margin-left:1.0in;text-indent:-.25in'>d.<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp; </span>Overland
flows associated with the stream segment are added to the model inputs as time-series
non-point source loadings.</p>

<p class=MsoListParagraphCxSpLast style='margin-left:1.5in;text-indent:-1.5in'><span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span>i.<span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span>Overland
flows are calculated based on the HAWQS COMPONENT_IN model result for the
segment which is the sum of overland flows and boundary condition flows. To
estimate the overland flows alone, the inflow mass of the component from the
up-river segment is subtracted from the COMPONENT_IN quantity.</p>

<p class=MsoNormal>&nbsp;</p>

<h3 style='page-break-after:avoid'><a name="New_Simulation"></a><a
name="_Toc124172135">New Simulation Window</a></h3>

<p class=MsoNormal style='page-break-after:avoid'>Three types of simulations
can be specified via the new simulation window. </p>

<p class=MsoNormal style='page-break-after:avoid'>&nbsp;</p>

<p class=MsoNormal style='page-break-after:avoid'><b>Stream Network (with
lakes)</b>: A stream network based on NHDPlusV2 flow lines that optionally
includes lakes or reservoirs can be simulated. In this case, water flows and
volumes can be specified from the National Water Model or the HAWQS/SWAT model
linkage in which case nutrient, sediment, and other simulation data will also
be available. Because HAWQS/SWAT does not explicitly model lakes and
reservoirs, if a lake/reservoir is in the model domain the national water model
will be used to estimate those water flows and volume calculations.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>A <b>0-D </b>(zero dimensional) <b>lake or reservoir</b> may
be simulated in which case inflows, outflows, surface area, and water volumes
for the waterbody will be available from the national water model. (HAWQS/SWAT
data are not available for this simulation type.)</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal><b>One HUC</b>: A single drainage basin (HUC or hydrologic
unit code) of varying sizes. HUC8 is the largest size and HUC14 is the
smallest. HAWQS/SWAT data will be used to estimate average conditions within
the primary reach of the basin.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>First, select which type of simulation you wish to model
using the radio buttons at the top of the screen. Then you will select the
geographic domain for the model.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>If you are working with a <b>stream network model</b>, zoom
the map close enough that the NHDPlusV2 stream segments start to render. These
shapes are loaded to the interface via web service so it may take a moment for
all of the shapes to be loaded. Click on the pour point (downstream) COMID to
be modeled or enter the COMID number (if known) into the COMID entry box in
Parameters to Read Network. You must then specify an up-stream COMID, for a
single stream segment, or a number of kilometers upstream, to capture all
tributaries to the selected pour point (within that number of river
kilometers). Then click Read Network. If valid inputs have been specified,
the selected stream network for the model will be rendered as red lines. The
total number of stream segments in the model will be displayed under Selected
Model Domain.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For a <b>lake or reservoir model</b>, use the map to zoom
into the waterbody of interest. You may use the mouse wheel or the plus minus
icon at the top left of the map to change the zoom level. All waterbodies
available in NWM are shown by default. Hovering the mouse cursor over one of
the waterbodies will show its name or COMID. To select a lake or reservoir for
your model, click on one of the waterbodies. It will turn red and the map will
zoom into that shape. The simulation name may then be edited. The surface
area of the shape is displayed and will be read into AQUATOX parameters once
the simulation is set up.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>For a <b>HUC simulation</b>, select the HUC size to display
using the radio buttons above the map. HUC14 segments are smallest so you must
zoom in to see those rendered. To select a single HUC for your model, click on
one of the polygons. It will change color and the map will zoom into that
shape. The simulation name may then be edited.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Once the domain for either type of model has been selected,
the user can specify a template segment for the simulation and also the start
and end date for the simulation. Clicking on the <u><span style='color:#365F91'>Choose
Template</span></u> button will show all of the relevant AQUATOX simulations
that can form the basis of this new model. If the user wishes to select a
different existing AQUATOX simulation file (JSON) than is contained in the
matrix, the <b>Browse </b>button next to the choose template button may be
chosen to select the file.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>Following the template selection, the <b>Start Date</b> and <b>End
Date</b> of the simulation may be edited as well using the calendar inputs at
the lower left of this window.</p>

<p class=MsoNormal>If you click OK you will be prompted for a directory in
which to store your simulation data.</p>

<p class=MsoNormal>&nbsp;</p>

<h3 style='page-break-after:avoid'><a name="Select_Surrogate"></a><a
name="_Toc124172136">Select Surrogate Simulation</a></h3>

<p class=MsoNormal style='page-break-after:avoid'><b>&nbsp;</b></p>

<p class=MsoNormal style='page-break-after:avoid'>Clicking Choose Template
brings up a matrix that shows all of the available lake, reservoir, and stream
segment files that can serve as a template for the multi-segment simulation
being created, (or the 0-D lake/reservoir simulation). At this time, all state
variables in each segment must be the same, so a single 0-D AQUATOX simulation
will form the basis of this simulation.</p>

<p class=MsoNormal>&nbsp;</p>

<p class=MsoNormal>This matrix shows the name, location, type of waterbody, and
summaries of plants, and animals in each simulation. Clicking on the column
headers will sort the matrix by that column. Clicking on one of the rows of
the matrix will highlight that row and that simulation will form the basis of
the multi-segment simulation that is being created, once the user selects OK.</p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124172137"><span style='font-size:14.0pt'>Tutorial</span></a></h2>

<h3><a name="Deleting_and_Adding_a_Plant"></a><a name="_Toc124172138"></a><a
name="_Toc475968400"><span style='font-size:12.0pt;color:black'>Simple Tutorial</span></a><span
style='font-size:12.0pt;color:black'> for a 0-D simulation</span></h3>

<p class=MsoNormal>This tutorial introduces basic concepts and gets a user started
with the AQUATOX interface.</p>

<h4><a name="_Toc475968401"></a><a name="_Toc124172139"><span style='font-size:
11.0pt'>Tutorial-- Step 1:&nbsp; Deleting and Adding a Plant</span></a></h4>

<p class=MsoNormal><span style='color:black'>Open the file <strong><span
style='font-family:"Arial",sans-serif'>Farm Pond MO Esfenval</span></strong>.&nbsp;
(See </span><a href="#LoadingAStudy">Loading a Simulation</a><span
style='color:black'>) &nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>This study represents a generic
pond built to USDA specifications. Esfenvalerate loadings are the worst-case
scenario using runoff from an adjacent corn field.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>First remove the macrophyte
compartment (<b>Macrophyte1: [Myriophyllum])</b> by highlighting it in the
list, and clicking on the <b>Delete</b> button.&nbsp; The variable will
disappear from the state variable list.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Likewise, state variables can be
added by clicking on the <b>Add</b> button and choosing from the list.&nbsp;
Lets add macrophytes back to the list of state variables.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Click on the Add Button and scroll
down the list until you see the <b>Macrophyte1</b> to <b>Macrophyte6</b>
compartments.&nbsp; AQUATOX has the capacity to model up to six unique
macrophyte species.&nbsp; Select <b>Macrophyte1</b> and click on OK.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>You will then be prompted as to
which database you will load parameters from. Select the default database
which is called <b>PlantLib.JSON</b>. You will then see another dialog box allowing
you to select from a list of plant-parameter datasets. This relates to the fact
that the names of the taxonomic groups and ecologic guilds on the state
variables list (on the main window) are followed by the names of the specific
groups in brackets.&nbsp; We therefore have to specify the type of macrophyte
we are adding.&nbsp; Select <i>Myriophyllum</i> on the list at the left and
click on <b>OK</b>.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Next, Setting an Initial Condition</span></p>

<h4><a name="_Toc475968402"></a><a name="_Toc124172140"><span style='font-size:
11.0pt'>Tutorial-- Step 2:&nbsp; Setting an Initial Condition</span></a></h4>

<p class=MsoNormal><span style='color:black'>To continue with our macrophyte
example, we should enter a value for the biomass of macrophytes present at the
beginning of the simulation; if the value is left as 0 and there is no loading,
then macrophytes would not be simulated.&nbsp; The initial condition will
depend on when the simulation starts (which is specified in <b>Setup</b>).&nbsp;
In this example, we will enter a value of 0.1 g/m<sup>2</sup>, which is
appropriate for <i>Myriophyllum</i> in a temperate pond at the beginning of the
growing season.&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Double click on the state variable
list on Macrophytes1: [Myriophyllum].</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The Loadings Form is then
displayed. There are three things we are looking at in this screen. First,
the initial condition for the state variable is shown at the top left of the
window. Ongoing plant loadings from up-river are shown in the panel below
that. The drop-down list specifying Loading Type can be changed to view
chemical concentrations within the macrophyte. For now, leave the Loading Type
as In inflow Water.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Enter 0.1 into the initial
condition (in g/m2 dry weight). </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=265 height=64 id="Picture 1"
src="AQUATOX.NET_1.0_UMAN_files/image006.jpg"><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>You may also enter a loading in
inflow water for macrophytes in this simulation.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=363 height=113 id="Picture 13"
src="AQUATOX.NET_1.0_UMAN_files/image007.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Ordinarily we think of loadings as
pertaining only to chemicals and freely moving organisms such as plankton and
fish.&nbsp; However, it is&nbsp;usually&nbsp;desirable to enter a small
constant loading to serve as a seed if the population or group is killed off
entirely by a toxicant or adverse environmental conditions. The seed will
allow the biota to recover or recolonize, if and when conditions become
favorable.&nbsp; This is especially important for macrophytes that suffer
winter die-back.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>To add this seed loading add the
value of&nbsp;1E-5 g/m<sup>2</sup> as a constant loading.&nbsp; This is small
enough that it will not affect the results during the growing season, but it is
large enough to prevent extinction.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>See Also: </span><a
href="#ICandLoadings">Initial Conditions and Loadings</a><span
class=MsoHyperlink>.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Next, Viewing Parameters in a
Simulation</span></p>

<h4><a name="_Toc124172141"></a><a name="_Toc475968403"><span style='font-size:
11.0pt'>Tutorial-- Step 3:&nbsp; Viewing Parameters</span></a><span
style='font-size:11.0pt'> in a Simulation</span></h4>

<p class=MsoNormal><span style='color:black'>In the following examples, we will
examine the parameters associated with the animals, plants, and chemicals in
this simulation. Parameters can be loaded from databases of parameters when
adding a state variable as we did above (when we loaded <i>Myriophyllum</i>).
Records can also be loaded from a <b>database</b> by choosing <b>Read from DB</b>
when looking at parameters associated with the animal, plant, and chemical
state variables. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>We will examine first the plants
parameters associated with <i>Myriophyllum</i>. From the loadings form, select
the <b>Parameters</b> button at top of the screen.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=110 height=40 id="Picture 15"
src="AQUATOX.NET_1.0_UMAN_files/image008.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Plant parameters for <i>Myriophyllum</i>
will then appear. Two fields near the top of the screen require
explanation.&nbsp; If you click on the drop-down menu to the right of <b>Plant
type</b>, you will be given a choice. The choice of <b>Plant type </b>is
important because different types have different physical or biological
processes that apply to them.&nbsp; For instance, phytoplankton are subject to
sinking, but not periphyton, which are attached to a surface.&nbsp; Conversely
periphyton are limited somewhat by very slow current velocity; but not
phytoplankton, which are adapted to still water.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Less obvious is the <b>Toxicity
Record</b>; again, clicking on the arrow to the right of the field will give
you several choices.&nbsp; The intent is to associate this modeled organism
record with one of the limited number of&nbsp;organisms that have some toxicity
data or procedures for estimating toxicity.&nbsp; In this instance, if you
choose <b>Macrophytes</b> the model will utilize the toxicity data (e.g., EC50)
for Esfenvalerate that is labeled as Macrophytes. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Parameter values within the
models databases are provided to get you started; if you have more appropriate
or locally measured values, you should use them. &nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>The primary parameters are shown
on the input window by default, but the screen can be modified to display other
subsets of parameters by selecting the Expand button next to each subheading.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=576 height=154 id="Picture 39"
src="AQUATOX.NET_1.0_UMAN_files/image009.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Next, we will locate the parameters
for the Chironomid (DepFeeder1) within the simulation. Hit <b>Cancel</b> to
leave the plant parameters screen and then select OK to return to the main
window and save your changes to the organisms initial condition and loadings.
</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Double click on the Chironomid
about 2/3 of the way down the state variable list and again, select Parameters.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Click on <b>Animal Type</b> to see
the pull-down menu.&nbsp; Chironomids have aquatic larvae, so <b>Benthic insect</b>
is chosen; this is important because emergence is simulated by AQUATOX for
insects as a loss term but does not apply to other animals.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=441 height=149 id="Picture 40"
src="AQUATOX.NET_1.0_UMAN_files/image010.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Finally, we will examine the parameters
pertaining to organic matter for this simulation.&nbsp; To do this, return to
the main window,&nbsp;and select the <b>Org. Matter<strong><span
style='font-family:"Arial",sans-serif'> </span></strong></b>button. Many of the
parameters on this screen are global and there is little need to change them
for a site, unless the organic material is quite different or there is some
reason that the microflora might have adapted to abnormal conditions, such as a
thermal spring or acid mine drainage.&nbsp; If you have site-specific nutrient
decomposition data, or wish to alter the sinking rate for suspended organic
matter, those model parameters would be found here. For more information about
the three different types of Remineralization records included with AQUATOX see
the section titled </span><a href="#Remineralization">Remineralization</a><b><span
style='color:black'>.</span></b></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=576 height=121 id="Picture 42"
src="AQUATOX.NET_1.0_UMAN_files/image011.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>After you have viewed the organic
matter parameters, select OK or Cancel to return to the main simulation
window. At the upper right of the main simulation window are buttons that will
take you to matrices that show the available databases site, animal, organic
matter (remin.), chemical, and plant parameters that are included with the
AQUATOX model. Unless these parameters are loaded into a simulation, as
discussed above, they will have no impact on a models result. (See also </span><a
href="#_Databases_vs._Parameters">Parameters within Database Files vs.
Parameters within in a Simulation</a><span class=MsoHyperlink>.</span><span
style='color:black'>)</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Next, Viewing Toxicant Loadings</span></p>

<h4 style='page-break-after:avoid'><a name="_Toc475968404"></a><a
name="_Toc124172142"><span style='font-size:11.0pt'>Tutorial-- Step 4:&nbsp;
Viewing Toxicant Loadings</span></a></h4>

<p class=MsoNormal style='page-break-after:avoid'><span style='color:black'>Following
the tutorial to this point, we should still have <strong><span
style='font-family:"Arial",sans-serif'>FarmPond MO Esfenval.aps</span></strong>
loaded from back in<b> Step 1</b>.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>In the State and Driving
Variables list, double-click on <b>Dissolved org. tox 1</b> to bring up the
loadings form for the chemical, and to examine the options for loadings to the
system.&nbsp; Pollutant loadings can be entered as constant or dynamic loadings
in several different forms. The pollutant can be entered as a concentration in
the dissolved phase or loosely bound to suspended sediment; the water inflow
and the site volume are then used by the model to compute the loading per unit
volume.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Click on the Loading Type drop
down box to toggle between loadings types.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span><img border=0
width=357 height=262 id="Picture 49"
src="AQUATOX.NET_1.0_UMAN_files/image012.jpg"></p>

<p class=MsoNormal align=center style='text-align:center'><span
style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Point-source loadings are mass per
day (g/d) for the entire site; they are divided by the site volume to obtain
the loading per unit volume.&nbsp; In this example, dynamic loadings from a
point discharge as calculated by the PRZM model are entered.&nbsp; Note that
the dynamic loadings are interpolated, so if the intent is to represent a spike
such as from storm runoff on a particular day, then the loadings should be
bracketed by 0&quot; loadings.&nbsp; The model assumes that the loadings wrap
around with an annual cycle and that the last loading can be interpolated to
the first loading as if it were in the succeeding year.&nbsp; Exercise caution
when modeling multiple years with loadings data from only one or a few
years.&nbsp; Sporadic loadings, which could be expected only in one particular
year might inappropriately be repeated in successive years.&nbsp; If you do not
wish loadings to be repeated, enter values (0&quot; or otherwise) for the
first and last days of the simulation.&nbsp; The dynamic loadings in this
example were entered by hand; an excellent alternative is to download or
prepare a file external to the model and import it into the simulation using
the <b>Import</b> button.&nbsp; See also </span><a
href="#Important_Note_about_Dynamic_Loadings">Importing Loading</a><span
style='color:black'>.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Another potential pollution
loading source is from direct precipitation.&nbsp; These loads are given as
g/m2 day because AQUATOX does not explicitly model the addition of water to a
system via precipitation.&nbsp; </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Next, Running the Simulation</span></p>

<h4><a name="_Toc475968405"></a><a name="_Toc124172143"><span style='font-size:
11.0pt'>Tutorial-- Step 5:&nbsp; Running the Simulation</span></a></h4>

<p class=MsoNormal><span style='color:black'>To change the time period of a
simulation or other integration options, click the <b>Edit Setup</b> button to
the upper left of the main window. The most important inputs from this screen
are the start and end of model simulation, which sets the simulation time.
(This should be compatible with the time period of any time-series model
loadings you have entered previously.) For other options, see the </span><a
href="#SetupWindow">Setup Parameters</a><span style='color:black'> section of
this manual.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>To run the simulation, select the <b>Run</b>
button to the left of the main window.&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>You will be asked for an optional
text string to define the run. Multiple runs can be saved within each AQUATOX
JSON and then can be compared graphically or have their results compared within
the output window. The date of each simulation will be also saved.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=402 height=102 id="Picture 50"
src="AQUATOX.NET_1.0_UMAN_files/image013.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>A progress bar will appear showing
the status of the model run. This run should take a minute or two.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Next, Viewing Output</span></p>

<h4><a name="_Toc475968406"></a><a name="_Toc124172144"><span style='font-size:
11.0pt'>Tutorial-- Step 6:&nbsp; Viewing Output</span></a></h4>

<p class=MsoNormal><span style='color:black'>When a simulation has been
executed, results for various state variables are stored in memory. These
results will be saved as an updated JSON if the simulation is saved.&nbsp; After
the simulation has completed the <b>Output</b> window will pop up. (To view
the output at another time, select the Output button on the main window.)</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>By selecting <b>Save CSV</b>,
model results may be exported to a comma-separated text-file format for import
into spreadsheet or database software. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>By selecting <b>New Graph</b>, a
set of model results can be created. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoListParagraph style='text-indent:-.25in'><span style='font-family:
Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>To see the chemical concentration in
water, name the graph Chemical in Water, and then use the arrow buttons to
move Dissolved org tox 1: to the box on the right.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><img border=0 width=576 height=183 id="Picture 3"
src="AQUATOX.NET_1.0_UMAN_files/image014.jpg"></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>After selecting OK you will see a
graph of predicted water-column concentrations of Esfenvalerate given the PRZM
based inputs.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoListParagraph style='text-indent:-.25in'><span style='font-family:
Symbol;color:black'><span style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
</span></span><span style='color:black'>To see results for all the animals in a
simulation, click New Graph again, name the graph Animals, click on the
Animals check box on top of the output list, and use the double right arrow
to move all of the animals onto the graph.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>Clicking on any of the results in
the graph will identify the state variable and provide a snapshot of its
concentration in time. Also, items on the graph may be toggled by clicking the
colored rectangles within the legend. </span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>For more information about the
output window see: </span><a href="#OutputScreen">Output Window</a><span
style='color:black'>.</span></p>

<p class=MsoNormal><span style='color:black'>&nbsp;</span></p>

<p class=MsoNormal><span style='color:black'>You have now seen the basic
building blocks for an AQUATOX model. You should have an initial exposure to
state variables, parameters, external loadings, available databases of
parameters, running a model and viewing model output. </span></p>

<p class=MsoNormal>&nbsp;</p>

<h2><a name="_Toc124172145">References</a></h2>

<div style="text-indent: -36px; padding-left: 36px;">
<p class=MsoNormal><span style='color:black'>Ambrose, J., Robert B. , J. L.
Martin, and T. A. Wool. 2006. WASP7 Benthic Algae - Model Theory and User's
Guide, Supplement to Water Quality Analysis Simulation Program (WASP) User
Documentation Page 32. U.S. Environmental Protection Agency, Washington, DC </span></p>

<p><span style='color:black'>Barber, M. C. 2003. A Review and Comparison of
Models for Predicting Dynamic Chemical Bioconcentration in Fish. <i>Environmental
Toxicology and Chemistry,</i> 22(9):1963-1992.</span></p>

<p>Di Toro, D. M. 2001. <i>Sediment Flux Modeling.</i> Wiley-Interscience, New
York.</p>

<p class=MsoNormal>Iman, R. L., and W. J. Conover. 1982. A distribution-free
approach to inducing rank correlation among input variables. <em><span
style='font-family:"Arial",sans-serif'>Communications in Statistics</span></em>
B11: 311-334. </p> </p>

<p class=MsoNormal>Park, R.A., R.V. O'Neill, J.A. Bloomfield, H.H. Shugart,
Jr., R.S. Booth, J.F. Koonce, M.S. Adams, L.S. Clesceri, E.M. Colon, E.H.
Dettman, R.A. Goldstein, J.A. Hoopes, D.D. Huff, S. Katz, J.F. Kitchell, R.C.
Kohberger, E.J. LaRow, D.C, McNaught, J.L. Peterson, D. Scavia, J.E. Titus,
P.R. Weiler, J.W. Wilkinson, and C.S. Zahorcak. 1974. A Generalized Model for
Simulating Lake Ecosystems. Simulation, 23(2):30-50. Reprinted in Benchmark
Papers in Ecology.</p> </p>

<p class=MsoNormal>Park, R.A., D. Scavia, and N.L. Clesceri. 1975. CLEANER,
The Lake George Model. In Ecological Modeling in a Management Context.
Resources for the Future, Inc., Washington, D.C.</p></p>

<p class=MsoNormal>Park, R.A. 1978. A Model for Simulating Lake Ecosystems.
Center for Ecological Modeling Report No. 3, Rensselaer Polytechnic Institute,
Troy, New York, 19 pp.</p></p>

<p class=MsoNormal>Park, R.A., T.W. Groden, and C.J. Desormeau. 1979. Modifications
to the Model CLEANER Requiring Further Research. In Perspectives on Lake
Ecosystem Modeling, edited by D. Scavia and A. Robertson. Ann Arbor Science
Publishers, Inc., 22 pp.</p></p>

<p class=MsoNormal>Park, R.A., C.D. Collins, C.I. Connolly, J.R. Albanese, and
B.B. MacLeod. 1980. Documentation of the Aquatic Ecosystem Model MS.CLEANER, A
Final Report for Grant No. R80504701, U.S. Environmental Protection Agency,
Environmental Research Laboratory, Athens, Georgia. 112 pp.</p></p>

<p class=MsoNormal>Park, R.A., C.I. Connolly, J.R. Albanese, L.S. Clesceri,
G.W. Heitzman, H.H. Herbrandson, B.H. Indyke, J.R. Loehe, S. Ross, D.D. Sharma,
and W.W. Shuster. 1980. Modeling Transport and Behavior of Pesticides and
Other Toxic Organic Materials in Aquatic Environments. Center for Ecological
Modeling Report No. 7. Rensselaer Polytechnic Institute, Troy, New York. 163
pp.</p></p>

<p class=MsoNormal>Park, R.A., C.I. Connolly, J.R. Albanese, L.S. Clesceri,
G.W. Heitzman, H.H. Herbrandson, B.H. Indyke, J.R. Loehe, S. Ross, D.D. Sharma,
and W.W. Shuster. 1982. Modeling the Fate of Toxic Organic Materials in
Aquatic Environments. U.S. Environmental Protection Agency Rept.
EPA-6OO/S3-82-028, Athens, Georgia.</p></p>

<p class=MsoNormal>Park, R.A. 1984. TOXTRACE: A Model to Simulate the Fate
and Transport of Toxic Chemicals in Terrestrial and Aquatic Environments.
Acqua e Aria, No. 6, p. 599-607 (in Italian).</p></p>

<p class=MsoNormal>Park, R.A. 1990. AQUATOX, a Modular Toxic Effects Model for
Aquatic Ecosystems. Final Report, EPA-026-87; U.S. Environmental Protection
Agency, Corvallis, Oregon.</p></p>

<p>Raimondo, S., D.N. Vivian, and M.G. Barron.
2010. <i>Web-based Interspecies Correlation Estimation (Web-ICE) for Acute
Toxicity: User Manual. Version 3.1. </i>EPA/600/R-10/004. Office of Research
and Development, U. S. Environmental Protection Agency. Gulf Breeze, FL.</p></p>

<p class=MsoNormal>Saltelli, A. 2001. Sensitivity Analysis for Importance
Assessment. Paper read at Sensitivity Analysis Methods, June 11-12, 2001, at
North Carolina State University, Raleigh NC</p></p>

<p class=MsoNormal>Scavia, D., and R.A. Park. 1976. Documentation of Selected
Constructs and Parameter Values in the Aquatic Model CLEANER. Ecological
Modelling 2(1):33-58.</p></p>

<p class=MsoNormal>Surez, L.A., and M.C. Barber. 1992. PIRANHA Version 2.0,
FGETS Version 3.0-11 User's Manual, In PIRANHA Pesticide and Industrial
Chemical Risk Analysis and Hazard Assessment. Athens, Georgia: U.S.
Environmental Protection Agency.</p></p>

<p class=MsoNormal>U.S. Environmental Protection Agency. 1997. Guiding
Principles for Monte Carlo Analysis. Risk Assessment Forum. Washington, DC:
U.S. Environmental Protection Agency</p></p>

<p class=MsoNormal>U.S. Environmental Protection Agency. 2009. AQUATOX
Technical Note 1, A Calibrated Parameter Set for Simulation of Algae in Shallow
Rivers, EPA-823-R-09-003 February 2009</p></p>

<p class=MsoNormal>U.S. Environmental Protection Agency. 2018. AQUATOX (Release
3.2) Modeling Environmental Fate and Ecological Effects in Aquatic Ecosystems
Volume 2: Technical Documentation. EPA/600/B-18/241. August 2018. <a
href="https://www.epa.gov/sites/default/files/2018-09/documents/aquatox_tech_doc_3.2.pdf">https://www.epa.gov/sites/default/files/2018-09/documents/aquatox_tech_doc_3.2.pdf</a>
</p>

<p class=MsoNormal>&nbsp;</p>

</div>
</div>

<div><br clear=all>

<hr align=left size=1 width="33%">

<div id=ftn1>

<p class=MsoFootnoteText><a href="#_ftnref1" name="_ftn1" title=""><span
class=MsoFootnoteReference><span class=MsoFootnoteReference><span
style='font-size:10.0pt;font-family:"Arial",sans-serif'>[1]</span></span></span></a>
COMIDs are Common Identifiers from NHDPlus data model version 2.1</p>

</div>

</div>

</body>

</html>
